# aaSEA

<details>

* Version: 1.1.0
* Source code: https://github.com/cran/aaSEA
* Date/Publication: 2019-11-09 16:20:02 UTC
* Number of recursive dependencies: 114

Run `revdep_details(,"aaSEA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# ADAMgui

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/ADAMgui
* URL: TBA
* BugReports: https://github.com/jrybarczyk/ADAMgui/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 157

Run `revdep_details(,"ADAMgui")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3 marked UTF-8 strings
    ```

# adapr

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/adapr
* BugReports: https://github.com/gelfondjal/adapr/issues
* Date/Publication: 2017-11-30 00:19:12 UTC
* Number of recursive dependencies: 127

Run `revdep_details(,"adapr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘plotly’ ‘shinydashboard’
      All declared Imports should be used.
    Missing or unexported object: ‘devtools::clean_source’
    ```

# adegenet

<details>

* Version: 2.1.3
* Source code: https://github.com/cran/adegenet
* URL: https://github.com/thibautjombart/adegenet
* Date/Publication: 2020-05-10 22:00:02 UTC
* Number of recursive dependencies: 106

Run `revdep_details(,"adegenet")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R       2.0Mb
        data    1.3Mb
        files   1.7Mb
    ```

# adepro

<details>

* Version: 3.0.0
* Source code: https://github.com/cran/adepro
* Date/Publication: 2020-03-26 15:40:02 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"adepro")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘jsonlite’
      All declared Imports should be used.
    ```

# AdhereRViz

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/AdhereRViz
* URL: https://github.com/ddediu/AdhereR
* Date/Publication: 2020-05-16 09:30:14 UTC
* Number of recursive dependencies: 110

Run `revdep_details(,"AdhereRViz")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘lubridate’ ‘parallel’ ‘rsvg’ ‘scales’
      All declared Imports should be used.
    ```

# alevinQC

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/alevinQC
* URL: https://github.com/csoneson/alevinQC
* BugReports: https://github.com/csoneson/alevinQC/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 87

Run `revdep_details(,"alevinQC")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.7Mb
      sub-directories of 1Mb or more:
        extdata  11.6Mb
    ```

# AMPLE

<details>

* Version: 0.0.2
* Source code: https://github.com/cran/AMPLE
* Date/Publication: 2019-09-16 04:10:03 UTC
* Number of recursive dependencies: 66

Run `revdep_details(,"AMPLE")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyjs’
      All declared Imports should be used.
    ```

# analysisPipelines

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/analysisPipelines
* URL: https://github.com/Mu-Sigma/analysis-pipelines
* BugReports: https://github.com/Mu-Sigma/analysis-pipelines/issues
* Date/Publication: 2020-06-12 08:00:02 UTC
* Number of recursive dependencies: 128

Run `revdep_details(,"analysisPipelines")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘SparkR’
    ```

# animalcules

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/animalcules
* URL: https://github.com/compbiomed/animalcules
* BugReports: https://github.com/compbiomed/animalcules/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 189

Run `revdep_details(,"animalcules")` for more info

</details>

## Newly fixed

*   checking examples ... ERROR
    ```
    Running examples in ‘animalcules-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: find_taxonomy_300
    > ### Title: Find the taxonomy for maximum 300 tids
    > ### Aliases: find_taxonomy_300
    > 
    > ### ** Examples
    > 
    > taxonLevels <- find_taxonomy_300(tids=1200)
    Error: HTTP failure: 429
    {"error":"API rate limit exceeded","api-key":"100.36.143.38","count":"4","limit":"3"}
    Execution halted
    ```

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.7Mb
      sub-directories of 1Mb or more:
        doc     4.4Mb
        shiny   1.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘reactable’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    alpha_div_boxplot: no visible binding for global variable ‘richness’
    differential_abundance: no visible binding for global variable ‘padj’
    differential_abundance: no visible binding for global variable ‘pValue’
    differential_abundance: no visible binding for global variable
      ‘log2FoldChange’
    diversities_help: no visible binding for global variable ‘x’
    find_biomarker: no visible binding for global variable ‘rowname’
    find_biomarker: no visible binding for global variable ‘importance’
    find_biomarker: no visible binding for global variable ‘.’
    find_biomarker: no visible binding for global variable ‘Overall’
    find_biomarker: no visible binding for global variable ‘y’
    find_biomarker: no visible binding for global variable ‘m’
    find_biomarker: no visible binding for global variable ‘d’
    relabu_barplot: no visible binding for global variable ‘.’
    relabu_boxplot: no visible binding for global variable ‘.’
    relabu_heatmap: no visible binding for global variable ‘.’
    upsample_counts: no visible binding for global variable ‘.’
    Undefined global functions or variables:
      . Overall d importance log2FoldChange m pValue padj richness rowname
      x y
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘devtools’
    ```

# animaltracker

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/animaltracker
* Date/Publication: 2020-03-25 11:50:02 UTC
* Number of recursive dependencies: 105

Run `revdep_details(,"animaltracker")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘V8’
      All declared Imports should be used.
    ```

# animint2

<details>

* Version: 2019.7.3
* Source code: https://github.com/cran/animint2
* URL: https://github.com/tdhock/animint2
* BugReports: https://github.com/tdhock/animint2/issues
* Date/Publication: 2019-07-18 06:36:12 UTC
* Number of recursive dependencies: 134

Run `revdep_details(,"animint2")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘animint2-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: aes
    > ### Title: Define aesthetic mappings.
    > ### Aliases: aes
    > 
    > ### ** Examples
    > 
    > aes(x = mpg, y = wt)
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 732 | SKIPPED: 7 | WARNINGS: 14 | FAILED: 19 ]
      1. Failure: grid: facet order follows default data frame order (@test-compiler-facet-locate.r#118) 
      2. Failure: grid: facet order follows default data frame order (@test-compiler-facet-locate.r#119) 
      3. Failure: grid: facet order follows default data frame order (@test-compiler-facet-locate.r#126) 
      4. Failure: grid: facet order follows default data frame order (@test-compiler-facet-locate.r#127) 
      5. Failure: grid: facet order follows default data frame order (@test-compiler-facet-locate.r#134) 
      6. Failure: grid: facet order follows default data frame order (@test-compiler-facet-locate.r#135) 
      7. Failure: wrap: facet order follows default data frame order (@test-compiler-facet-locate.r#148) 
      8. Failure: wrap: facet order follows default data frame order (@test-compiler-facet-locate.r#154) 
      9. Failure: wrap: facet order follows default data frame order (@test-compiler-facet-locate.r#160) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘lazyeval’ ‘tibble’
      All declared Imports should be used.
    ```

# ANOVAIREVA

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/ANOVAIREVA
* Date/Publication: 2019-10-16 12:40:02 UTC
* Number of recursive dependencies: 112

Run `revdep_details(,"ANOVAIREVA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘car’ ‘datasets’ ‘dplyr’ ‘ggplot2’ ‘plotly’ ‘shiny’
      All declared Imports should be used.
    ```

# ANOVAShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/ANOVAShiny
* Date/Publication: 2019-05-01 07:00:03 UTC
* Number of recursive dependencies: 107

Run `revdep_details(,"ANOVAShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DescTools’ ‘HH’ ‘datasets’ ‘dplyr’ ‘rhandsontable’ ‘shiny’
      All declared Imports should be used.
    ```

# antitrust

<details>

* Version: 0.99.11
* Source code: https://github.com/cran/antitrust
* Date/Publication: 2019-06-21 08:00:15 UTC
* Number of recursive dependencies: 65

Run `revdep_details(,"antitrust")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘evd’
      All declared Imports should be used.
    ```

# apexcharter

<details>

* Version: 0.1.4
* Source code: https://github.com/cran/apexcharter
* URL: https://github.com/dreamRs/apexcharter, https://dreamrs.github.io/apexcharter
* BugReports: https://github.com/dreamRs/apexcharter/issues
* Date/Publication: 2020-03-31 09:50:10 UTC
* Number of recursive dependencies: 91

Run `revdep_details(,"apexcharter")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 230 marked UTF-8 strings
    ```

# appreci8R

<details>

* Version: 1.6.0
* Source code: https://github.com/cran/appreci8R
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 133

Run `revdep_details(,"appreci8R")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      'PolyPhen.Hsapiens.dbSNP131', 'SIFT.Hsapiens.dbSNP137'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ArchaeoPhases

<details>

* Version: 1.4.5
* Source code: https://github.com/cran/ArchaeoPhases
* BugReports: https://github.com/ArchaeoStat/ArchaeoPhases/issues
* Date/Publication: 2020-05-29 15:10:02 UTC
* Number of recursive dependencies: 107

Run `revdep_details(,"ArchaeoPhases")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# arena2r

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/arena2r
* URL: https://github.com/pedroliman/arena2r
* BugReports: https://github.com/pedroliman/arena2r/issues
* Date/Publication: 2018-10-19 15:30:03 UTC
* Number of recursive dependencies: 83

Run `revdep_details(,"arena2r")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘purrr’ ‘shinyBS’ ‘shinydashboard’ ‘shinyjs’
      All declared Imports should be used.
    ```

# ASSOCShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/ASSOCShiny
* Date/Publication: 2019-05-14 12:50:03 UTC
* Number of recursive dependencies: 111

Run `revdep_details(,"ASSOCShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘arules’ ‘arulesViz’ ‘dplyr’ ‘methods’ ‘plotly’ ‘shiny’ ‘shinyMatrix’
      All declared Imports should be used.
    ```

# AUCell

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/AUCell
* URL: http://scenic.aertslab.org
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 168

Run `revdep_details(,"AUCell")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘doMC’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘S4Vectors’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    AUCell_createViewerApp : <anonymous>: no visible global function
      definition for ‘%>%’
    AUCell_createViewerApp : <anonymous>: no visible binding for global
      variable ‘tsne1’
    AUCell_createViewerApp : <anonymous>: no visible binding for global
      variable ‘tsne2’
    AUCell_createViewerApp : <anonymous>: no visible binding for global
      variable ‘cell’
    Undefined global functions or variables:
      %>% cell tsne1 tsne2
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘SingleCellExperiment’
    ```

# autoTS

<details>

* Version: 0.9.11
* Source code: https://github.com/cran/autoTS
* URL: https://github.com/vivienroussez/autoTS
* BugReports: https://github.com/vivienroussez/autots/issues
* Date/Publication: 2020-06-05 12:20:06 UTC
* Number of recursive dependencies: 112

Run `revdep_details(,"autoTS")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘plotly’ ‘shinycssloaders’
      All declared Imports should be used.
    ```

# BARIS

<details>

* Version: 1.1.1
* Source code: https://github.com/cran/BARIS
* URL: https://github.com/feddelegrand7/BARIS
* BugReports: https://github.com/feddelegrand7/BARIS/issues
* Date/Publication: 2020-05-25 04:10:02 UTC
* Number of recursive dependencies: 85

Run `revdep_details(,"BARIS")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstudioapi’
      All declared Imports should be used.
    ```

# BatchQC

<details>

* Version: 1.16.0
* Source code: https://github.com/cran/BatchQC
* URL: https://github.com/mani2012/BatchQC
* BugReports: https://github.com/mani2012/BatchQC/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 124

Run `revdep_details(,"BatchQC")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      fbatch3      -0.1891     0.3385  -0.559   0.5787  
      fcond1        0.5066     0.2764   1.833   0.0721 .
      ---
      Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
      
      Residual standard error: 1.071 on 56 degrees of freedom
      Multiple R-squared:  0.06516,	Adjusted R-squared:  0.01508 
      F-statistic: 1.301 on 3 and 56 DF,  p-value: 0.2831
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 15 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 1 ]
      1. Error: batchQC (@test_batchQC.R#25) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    gene_plot: no visible global function definition for ‘boxplot’
    gene_plot: no visible binding for global variable ‘batch’
    gls.series.C: no visible global function definition for ‘lm.fit’
    sample_plot: no visible global function definition for ‘boxplot’
    sample_plot: no visible binding for global variable ‘batch’
    Undefined global functions or variables:
      batch boxplot lm.fit
    Consider adding
      importFrom("graphics", "boxplot")
      importFrom("stats", "lm.fit")
    to your NAMESPACE file.
    ```

# BayesBD

<details>

* Version: 1.2
* Source code: https://github.com/cran/BayesBD
* Date/Publication: 2017-08-21 17:32:06 UTC
* Number of recursive dependencies: 40

Run `revdep_details(,"BayesBD")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘mritc’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BayesianNetwork

<details>

* Version: 0.1.5
* Source code: https://github.com/cran/BayesianNetwork
* URL: https://github.com/paulgovan/bayesiannetwork
* BugReports: https://github.com/paulgovan/bayesiannetwork/issues
* Date/Publication: 2018-12-02 05:10:03 UTC
* Number of recursive dependencies: 122

Run `revdep_details(,"BayesianNetwork")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘shinytest’ ‘testthat’
      All declared Imports should be used.
    ```

# BayesNetBP

<details>

* Version: 1.5.2
* Source code: https://github.com/cran/BayesNetBP
* BugReports: https://github.com/hyu-ub/BayesNetBP/issues
* Date/Publication: 2020-04-14 09:00:09 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"BayesNetBP")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘cyjShiny’
    ```

# BBEST

<details>

* Version: 0.1-6
* Source code: https://github.com/cran/BBEST
* Date/Publication: 2016-03-09 18:55:20
* Number of recursive dependencies: 64

Run `revdep_details(,"BBEST")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘wmtsa’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BCEA

<details>

* Version: 2.3-1.1
* Source code: https://github.com/cran/BCEA
* URL: http://www.statistica.it/gianluca/BCEA, http://www.statistica.it/gianluca
* Date/Publication: 2019-08-26 07:55:24 UTC
* Number of recursive dependencies: 96

Run `revdep_details(,"BCEA")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

# bdchecks

<details>

* Version: 0.1.7
* Source code: https://github.com/cran/bdchecks
* URL: https://github.com/bd-R/bdchecks
* BugReports: https://github.com/bd-R/bdchecks/issues
* Date/Publication: 2019-02-18 15:00:07 UTC
* Number of recursive dependencies: 119

Run `revdep_details(,"bdchecks")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 72 marked UTF-8 strings
    ```

# bdclean

<details>

* Version: 0.1.15
* Source code: https://github.com/cran/bdclean
* URL: https://github.com/bd-R/bdclean, https://bd-r.github.io/The-bdverse/index.html
* BugReports: https://github.com/bd-R/bdclean/issues
* Date/Publication: 2019-04-11 14:45:17 UTC
* Number of recursive dependencies: 129

Run `revdep_details(,"bdclean")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# bde

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/bde
* Date/Publication: 2015-02-16 13:48:05
* Number of recursive dependencies: 58

Run `revdep_details(,"bde")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    rsample,BoundedDensity: no visible global function definition for
      ‘runif’
    setNormalizationConst,MacroBetaChen99Kernel: no visible global function
      definition for ‘integrate’
    setNormalizationConst,MacroBetaHirukawaTSKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstant,MacroBetaHirukawaJLNKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstants,MicroBetaChen99Kernel :
      density.kernelFunction: no visible global function definition for
      ‘dbeta’
    setNormalizationConstants,MicroBetaChen99Kernel : <anonymous>: no
      visible global function definition for ‘integrate’
    show,BoundedDensity: no visible global function definition for ‘str’
    show,KernelDensity: no visible global function definition for ‘str’
    Undefined global functions or variables:
      dbeta integrate runif str
    Consider adding
      importFrom("stats", "dbeta", "integrate", "runif")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# bea.R

<details>

* Version: 1.0.6
* Source code: https://github.com/cran/bea.R
* URL: https://github.com/us-bea/bea.R
* Date/Publication: 2018-02-23 19:30:19 UTC
* Number of recursive dependencies: 75

Run `revdep_details(,"bea.R")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘chron’ ‘colorspace’ ‘gtable’ ‘htmltools’ ‘htmlwidgets’
      ‘httpuv’ ‘magrittr’ ‘munsell’ ‘plyr’ ‘scales’ ‘stringi’ ‘xtable’
      ‘yaml’
      All declared Imports should be used.
    ```

# beanz

<details>

* Version: 2.4
* Source code: https://github.com/cran/beanz
* Date/Publication: 2018-11-05 21:30:03 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"beanz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        libs   7.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstantools’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# bestSDP

<details>

* Version: 0.1.2
* Source code: https://github.com/cran/bestSDP
* Date/Publication: 2019-12-13 13:10:06 UTC
* Number of recursive dependencies: 83

Run `revdep_details(,"bestSDP")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘dplyr’ ‘ggplot2’ ‘readxl’ ‘rlist’ ‘shinyBS’ ‘shinyWidgets’
      ‘shinydashboard’ ‘shinyjs’ ‘shinythemes’ ‘stringr’ ‘tidyr’
      All declared Imports should be used.
    ```

# BETS

<details>

* Version: 0.4.9
* Source code: https://github.com/cran/BETS
* URL: https://github.com/nmecsys/BETS
* BugReports: https://github.com/nmecsys/BETS/issues
* Date/Publication: 2018-09-28 17:10:03 UTC
* Number of recursive dependencies: 141

Run `revdep_details(,"BETS")` for more info

</details>

## In both

*   checking whether the package can be unloaded cleanly ... WARNING
    ```
    Error: package or namespace load failed for ‘BETS’ in dyn.load(file, DLLpath = DLLpath, ...):
     unable to load shared object '/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so':
      dlopen(/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so, 6): Symbol not found: _GSS_C_NT_USER_NAME
      Referenced from: /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so
      Expected in: flat namespace
     in /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so
    Execution halted
    ```

*   checking whether the namespace can be loaded with stated dependencies ... WARNING
    ```
    Error in dyn.load(file, DLLpath = DLLpath, ...) : 
      unable to load shared object '/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so':
      dlopen(/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so, 6): Symbol not found: _GSS_C_NT_USER_NAME
      Referenced from: /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so
      Expected in: flat namespace
     in /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so
    Calls: <Anonymous> ... namespaceImport -> loadNamespace -> library.dynam -> dyn.load
    Execution halted
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Error: package or namespace load failed for ‘BETS’ in dyn.load(file, DLLpath = DLLpath, ...):
     unable to load shared object '/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so':
      dlopen(/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so, 6): Symbol not found: _GSS_C_NT_USER_NAME
      Referenced from: /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so
      Expected in: flat namespace
     in /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/BETS/RMySQL/libs/RMySQL.so
    Call sequence:
    6: stop(msg, call. = FALSE, domain = NA)
    5: value[[3L]](cond)
    4: tryCatchOne(expr, names, parentenv, handlers[[1L]])
    3: tryCatchList(expr, classes, parentenv, handlers)
    2: tryCatch({
           attr(package, "LibPath") <- which.lib.loc
           ns <- loadNamespace(package, lib.loc)
           env <- attachNamespace(ns, pos = pos, deps, exclude, include.only)
       },
    Execution halted
    ```

# bibliometrix

<details>

* Version: 3.0.2
* Source code: https://github.com/cran/bibliometrix
* URL: https://www.bibliometrix.org, https://github.com/massimoaria/bibliometrix
* BugReports: https://github.com/massimoaria/bibliometrix/issues
* Date/Publication: 2020-06-17 10:00:03 UTC
* Number of recursive dependencies: 150

Run `revdep_details(,"bibliometrix")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        data   2.0Mb
        doc    2.7Mb
    ```

# bigPint

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/bigPint
* URL: https://github.com/lindsayrutter/bigPint
* BugReports: https://github.com/lindsayrutter/bigPint/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 165

Run `revdep_details(,"bigPint")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.1Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              2.7Mb
        shiny-examples   3.0Mb
    ```

# bigQueryR

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/bigQueryR
* URL: http://code.markedmondson.me/bigQueryR/
* BugReports: https://github.com/cloudyr/bigQueryR/issues
* Date/Publication: 2019-10-09 12:40:03 UTC
* Number of recursive dependencies: 59

Run `revdep_details(,"bigQueryR")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘bigrquery’
    ```

# billboarder

<details>

* Version: 0.2.8
* Source code: https://github.com/cran/billboarder
* URL: https://github.com/dreamRs/billboarder
* BugReports: https://github.com/dreamRs/billboarder/issues
* Date/Publication: 2020-01-09 18:30:26 UTC
* Number of recursive dependencies: 76

Run `revdep_details(,"billboarder")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc   3.4Mb
    ```

# binovisualfields

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/binovisualfields
* URL: https://people.eng.unimelb.edu.au/aturpin/opi/index.html
* Date/Publication: 2020-03-10 10:20:02 UTC
* Number of recursive dependencies: 48

Run `revdep_details(,"binovisualfields")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gtools’
      All declared Imports should be used.
    ```

# bioCancer

<details>

* Version: 1.16.0
* Source code: https://github.com/cran/bioCancer
* URL: http://kmezhoud.github.io/bioCancer
* BugReports: https://github.com/kmezhoud/bioCancer/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 204

Run `revdep_details(,"bioCancer")` for more info

</details>

## In both

*   checking Rd files ... WARNING
    ```
    checkRd: (5) bioCancer.Rd:0-20: Must have a \description
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.3Mb
      sub-directories of 1Mb or more:
        app       3.3Mb
        doc       2.8Mb
        extdata   1.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘AlgDesign’ ‘import’ ‘methods’ ‘shinythemes’
      All declared Imports should be used.
    ```

# BiocOncoTK

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/BiocOncoTK
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 205

Run `revdep_details(,"BiocOncoTK")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Execution halted
      ── 3. Error: rbind utilities work (@test_dockstore_scripts.R#29)  ──────────────
      cannot open the connection
      Backtrace:
       1. base::readRDS("demo.rds")
       2. base::gzfile(file, "rb")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 0 | SKIPPED: 0 | WARNINGS: 2 | FAILED: 3 ]
      1. Failure: exprByMSI_csv produces expected CSV (@test_dockstore_scripts.R#10) 
      2. Error: exprByMSI_csv produces expected CSV (@test_dockstore_scripts.R#11) 
      3. Error: rbind utilities work (@test_dockstore_scripts.R#29) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.6Mb
      sub-directories of 1Mb or more:
        data        6.1Mb
        doc         3.6Mb
        pamphlets   4.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ggscat_av: no visible binding for global variable ‘symbol’
    ggscat_av: no visible binding for global variable ‘msival’
    ggscat_av: no visible binding for global variable ‘tmsi’
    ggscat_av: no visible binding for global variable ‘log2exa’
    mc3toGR : <anonymous>: no visible binding for global variable
      ‘Consequence’
    multiviz: no visible binding for global variable ‘acronym’
    multiviz: no visible binding for global variable ‘msival’
    prc: no visible binding for global variable ‘acronym’
    prc: no visible global function definition for ‘right_join’
    rainfall: no visible global function definition for ‘genome’
    tumNorSet : <anonymous>: no visible global function definition for
      ‘pancan_SE’
    Undefined global functions or variables:
      BiocFileCache Consequence acronym genome log2ex log2exa mapIds
      map_tcga_ncit msicode msival new pancan_SE project_short_name
      right_join seqlengths symbol tfstart tmsi v1 v2
    Consider adding
      importFrom("methods", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 46 marked UTF-8 strings
    ```

# BioInstaller

<details>

* Version: 0.3.7
* Source code: https://github.com/cran/BioInstaller
* URL: https://github.com/JhuangLab/BioInstaller
* BugReports: https://github.com/JhuangLab/BioInstaller/issues
* Date/Publication: 2018-11-20 15:50:09 UTC
* Number of recursive dependencies: 105

Run `revdep_details(,"BioInstaller")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   8.1Mb
    ```

# BioNetStat

<details>

* Version: 1.8.4
* Source code: https://github.com/cran/BioNetStat
* URL: http://github.com/jardimViniciusC/BioNetStat
* BugReports: http://github.com/jardimViniciusC/BioNetStat/issues
* Date/Publication: 2020-06-01
* Number of recursive dependencies: 129

Run `revdep_details(,"BioNetStat")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        extdata   1.9Mb
        shiny     1.9Mb
    ```

*   checking package subdirectories ... NOTE
    ```
    Found the following CITATION file in a non-standard place:
      CITATION
    Most likely ‘inst/CITATION’ should be used instead.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BiocParallel’ ‘RColorBrewer’ ‘RJSONIO’ ‘ggplot2’ ‘knitr’ ‘pheatmap’
      ‘plyr’ ‘stats’ ‘utils’ ‘whisker’ ‘yaml’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘DT’ ‘shiny’ ‘shinyBS’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for ‘bplapply’
    gaussianDensity: no visible global function definition for ‘bw.nrd0’
    gaussianDensity: no visible global function definition for ‘density’
    pathPlot: no visible binding for global variable ‘median’
    readVarFile: no visible global function definition for ‘read.table’
    resInt : <anonymous>: no visible global function definition for ‘dist’
    retEdgesTable: no visible global function definition for ‘p.adjust’
    retTable: no visible global function definition for ‘p.adjust’
    spectralDistributionTest: no visible global function definition for
      ‘bplapply’
    spectralEntropyTest: no visible global function definition for
      ‘bplapply’
    var.list: no visible global function definition for ‘aggregate’
    Undefined global functions or variables:
      adjacencyMatrix1 aggregate bplapply bw.nrd0 cor density dist expr
      median p.adjust read.csv read.table
    Consider adding
      importFrom("stats", "aggregate", "bw.nrd0", "cor", "density", "dist",
                 "median", "p.adjust")
      importFrom("utils", "read.csv", "read.table")
    to your NAMESPACE file.
    ```

# BIRDS

<details>

* Version: 0.1
* Source code: https://github.com/cran/BIRDS
* URL: https://github.com/greensway/BIRDS
* BugReports: https://github.com/Greensway/BIRDS/issues
* Date/Publication: 2020-03-20 14:00:02 UTC
* Number of recursive dependencies: 192

Run `revdep_details(,"BIRDS")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘esquisse’ ‘rgdal’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 27976 marked UTF-8 strings
    ```

# biva

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/biva
* Date/Publication: 2018-10-30 19:20:06 UTC
* Number of recursive dependencies: 81

Run `revdep_details(,"biva")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘corrgram’ ‘rpivotTable’ ‘shinyAce’
      All declared Imports should be used.
    ```

# blkbox

<details>

* Version: 1.0
* Source code: https://github.com/cran/blkbox
* Date/Publication: 2016-08-08 02:37:07
* Number of recursive dependencies: 153

Run `revdep_details(,"blkbox")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘bigrf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘glmnet’ ‘gtools’ ‘knitr’ ‘nnet’ ‘parallel’ ‘rJava’ ‘reshape’
      ‘rmarkdown’ ‘shinyjs’
      All declared Imports should be used.
    Missing or unexported object: ‘xgboost::predict’
    ```

# blockCV

<details>

* Version: 2.1.1
* Source code: https://github.com/cran/blockCV
* URL: https://github.com/rvalavi/blockCV
* Date/Publication: 2020-02-23 20:00:02 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"blockCV")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘biomod2’
    ```

# BLRShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/BLRShiny
* Date/Publication: 2019-05-09 15:50:10 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"BLRShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘caret’ ‘datasets’ ‘dplyr’ ‘e1071’ ‘ggplot2’ ‘rhandsontable’ ‘shiny’
      All declared Imports should be used.
    ```

# bsplus

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/bsplus
* URL: https://github.com/ijlyttle/bsplus
* BugReports: https://github.com/ijlyttle/bsplus/issues
* Date/Publication: 2018-04-05 18:39:46 UTC
* Number of recursive dependencies: 57

Run `revdep_details(,"bsplus")` for more info

</details>

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      y[1]: "col-sm-3 col-sm-offset-1"
      
      ── 2. Failure: constructor with different column-specs (@test-accordion_sidebar.
      tagGetAttribute(acc_sidebar_main, "class") not identical to "col-sm-7 col-sm-offset-1".
      1/1 mismatches
      x[1]: "col-sm-7 offset-md-1 col-sm-offset-1"
      y[1]: "col-sm-7 col-sm-offset-1"
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 102 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 2 ]
      1. Failure: constructor with different column-specs (@test-accordion_sidebar.R#57) 
      2. Failure: constructor with different column-specs (@test-accordion_sidebar.R#59) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# caffsim

<details>

* Version: 0.2.2
* Source code: https://github.com/cran/caffsim
* URL: https://github.com/asancpt/caffsim
* BugReports: https://github.com/asancpt/caffsim/issues
* Date/Publication: 2017-08-28 03:36:21 UTC
* Number of recursive dependencies: 66

Run `revdep_details(,"caffsim")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘markdown’
      All declared Imports should be used.
    ```

# canvasXpress

<details>

* Version: 1.27.6-1
* Source code: https://github.com/cran/canvasXpress
* URL: https://github.com/neuhausi/canvasXpress.git
* BugReports: https://github.com/neuhausi/canvasXpress/issues
* Date/Publication: 2020-04-11 16:20:02 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"canvasXpress")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        htmlwidgets   3.2Mb
    ```

# CaPO4Sim

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CaPO4Sim
* Date/Publication: 2019-04-11 14:55:39 UTC
* Number of recursive dependencies: 85

Run `revdep_details(,"CaPO4Sim")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# cartools

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/cartools
* Date/Publication: 2018-08-20 09:00:03 UTC
* Number of recursive dependencies: 131

Run `revdep_details(,"cartools")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘animation’ ‘devtools’ ‘gapminder’ ‘knitr’ ‘rlist’ ‘rmarkdown’
      ‘roxygen2’ ‘sde’ ‘shiny’ ‘tidyverse’ ‘usethis’ ‘utils’
      All declared Imports should be used.
    ```

# ChAMP

<details>

* Version: 2.18.2
* Source code: https://github.com/cran/ChAMP
* Date/Publication: 2020-05-14
* Number of recursive dependencies: 240

Run `revdep_details(,"ChAMP")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘IlluminaHumanMethylation450kmanifest’
    
    Depends: includes the non-default packages:
      'minfi', 'ChAMPdata', 'FEM', 'DMRcate',
      'Illumina450ProbeVariants.db',
      'IlluminaHumanMethylationEPICmanifest', 'DT'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CHETAH

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/CHETAH
* URL: https://github.com/jdekanter/CHETAH
* BugReports: https://github.com/jdekanter/CHETAH
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 142

Run `revdep_details(,"CHETAH")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.1Mb
      sub-directories of 1Mb or more:
        data  11.0Mb
        doc    1.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘SingleCellExperiment’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# chipPCR

<details>

* Version: 0.0.8-10
* Source code: https://github.com/cran/chipPCR
* URL: https://github.com/michbur/chipPCR
* BugReports: https://github.com/michbur/chipPCR/issues
* Date/Publication: 2015-04-10 01:10:56
* Number of recursive dependencies: 122

Run `revdep_details(,"chipPCR")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot,refMFI-refMFI: no visible global function definition for ‘mtext’
    plot,refMFI-refMFI: no visible global function definition for ‘abline’
    plot,refMFI-refMFI: no visible global function definition for ‘legend’
    plot,refMFI-refMFI: no visible global function definition for ‘lines’
    qqline,refMFI: no visible binding for global variable ‘qnorm’
    Undefined global functions or variables:
      abline adjustcolor arrows axis coef cor head layout legend lines lm
      mad matplot median mtext na.omit par points polygon predict
      predict.lm qnorm quantile rect rnorm rstudent rug sd shapiro.test
      smooth.spline tail text wilcox.test
    Consider adding
      importFrom("grDevices", "adjustcolor")
      importFrom("graphics", "abline", "arrows", "axis", "layout", "legend",
                 "lines", "matplot", "mtext", "par", "points", "polygon",
                 "rect", "rug", "text")
      importFrom("stats", "coef", "cor", "lm", "mad", "median", "na.omit",
                 "predict", "predict.lm", "qnorm", "quantile", "rnorm",
                 "rstudent", "sd", "shapiro.test", "smooth.spline",
                 "wilcox.test")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

# chromoMap

<details>

* Version: 0.2
* Source code: https://github.com/cran/chromoMap
* Date/Publication: 2019-04-10 19:27:11 UTC
* Number of recursive dependencies: 30

Run `revdep_details(,"chromoMap")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# cicerone

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/cicerone
* URL: https://cicerone.john-coene.com/
* BugReports: https://github.com/JohnCoene/cicerone/issues
* Date/Publication: 2020-02-29 12:40:02 UTC
* Number of recursive dependencies: 19

Run `revdep_details(,"cicerone")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# cjoint

<details>

* Version: 2.1.0
* Source code: https://github.com/cran/cjoint
* Date/Publication: 2018-10-19 08:20:02 UTC
* Number of recursive dependencies: 74

Run `revdep_details(,"cjoint")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyBS’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 29 marked UTF-8 strings
    ```

# clustDRM

<details>

* Version: 0.1-0
* Source code: https://github.com/cran/clustDRM
* Date/Publication: 2019-03-15 16:53:44 UTC
* Number of recursive dependencies: 118

Run `revdep_details(,"clustDRM")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# CLUSTShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CLUSTShiny
* Date/Publication: 2019-04-28 07:10:06 UTC
* Number of recursive dependencies: 99

Run `revdep_details(,"CLUSTShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘cluster’ ‘datasets’ ‘dplyr’ ‘klaR’ ‘psycho’ ‘shiny’
      All declared Imports should be used.
    ```

# CMShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CMShiny
* Date/Publication: 2019-10-24 17:20:05 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"CMShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘caret’ ‘e1071’ ‘epitools’ ‘shiny’ ‘shinyMatrix’
      All declared Imports should be used.
    ```

# CNVPanelizer

<details>

* Version: 1.20.0
* Source code: https://github.com/cran/CNVPanelizer
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 103

Run `revdep_details(,"CNVPanelizer")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘utils:::format.object_size’
      See the note in ?`:::` about the use of this operator.
    ```

# cocktailApp

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/cocktailApp
* URL: https://github.com/shabbychef/cocktailApp
* BugReports: https://github.com/shabbychef/cocktailApp/issues
* Date/Publication: 2019-07-02 08:30:10 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"cocktailApp")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 14661 marked UTF-8 strings
    ```

# codebook

<details>

* Version: 0.9.2
* Source code: https://github.com/cran/codebook
* URL: https://github.com/rubenarslan/codebook
* BugReports: https://github.com/rubenarslan/codebook/issues
* Date/Publication: 2020-06-06 23:40:03 UTC
* Number of recursive dependencies: 178

Run `revdep_details(,"codebook")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘graphics’ ‘jsonlite’ ‘rlang’ ‘tidyselect’ ‘vctrs’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 65 marked UTF-8 strings
    ```

# cognitoR

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/cognitoR
* BugReports: https://github.com/chi2labs/cognitoR/issues
* Date/Publication: 2020-04-15 05:20:02 UTC
* Number of recursive dependencies: 50

Run `revdep_details(,"cognitoR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘yaml’
      All declared Imports should be used.
    ```

# COMPASS

<details>

* Version: 1.26.0
* Source code: https://github.com/cran/COMPASS
* BugReports: https://github.com/RGLab/COMPASS/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 132

Run `revdep_details(,"COMPASS")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .github
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BiocStyle’ ‘rmarkdown’
      All declared Imports should be used.
    ':::' call which should be '::': ‘flowWorkspace:::.getNodeInd’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .CellCounts: no visible binding for global variable
      ‘_COMPASS_CellCounts’
    COMPASSfitToCountsTable: no visible binding for global variable
      ‘population’
    COMPASSfitToCountsTable: no visible binding for global variable ‘Count’
    COMPASSfitToCountsTable: no visible binding for global variable ‘id’
    CellCounts_character: no visible binding for global variable
      ‘_COMPASS_CellCounts_character’
    GetThresholdedIntensities : <anonymous>: no visible global function
      definition for ‘gh_pop_is_bool_gate’
    Undefined global functions or variables:
      Count _COMPASS_CellCounts _COMPASS_CellCounts_character
      gh_pop_is_bool_gate id population
    ```

# competitiontoolbox

<details>

* Version: 0.1.2
* Source code: https://github.com/cran/competitiontoolbox
* Date/Publication: 2019-09-12 11:30:02 UTC
* Number of recursive dependencies: 67

Run `revdep_details(,"competitiontoolbox")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# conflr

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/conflr
* URL: https://line.github.io/conflr/, https://github.com/line/conflr
* BugReports: https://github.com/line/conflr/issues
* Date/Publication: 2020-04-08 12:50:02 UTC
* Number of recursive dependencies: 56

Run `revdep_details(,"conflr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# CoRegNet

<details>

* Version: 1.26.0
* Source code: https://github.com/cran/CoRegNet
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 44

Run `revdep_details(,"CoRegNet")` for more info

</details>

## In both

*   checking for unstated dependencies in ‘tests’ ... WARNING
    ```
    '::' or ':::' import not declared from: ‘BiocGenerics’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘RColorBrewer’ ‘gplots’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    refine,coregnet: no visible global function definition for ‘glm’
    refine,coregnet: no visible global function definition for ‘abline’
    regulatorInfluence,coregnet : <anonymous> : <anonymous>: no visible
      global function definition for ‘t.test’
    regulators,coregnet: no visible global function definition for
      ‘na.omit’
    repressors,coregnet: no visible global function definition for
      ‘na.omit’
    targets,coregnet: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      abline arrows as.dist axis coef colorRampPalette cor fisher.test glm
      hclust heatmap.2 lines lm mtext na.omit p.adjust par pchisq pnorm
      quantile rainbow sd segments symbols t.test text vcount wilcox.test
    Consider adding
      importFrom("grDevices", "colorRampPalette", "rainbow")
      importFrom("graphics", "abline", "arrows", "axis", "lines", "mtext",
                 "par", "segments", "symbols", "text")
      importFrom("stats", "as.dist", "coef", "cor", "fisher.test", "glm",
                 "hclust", "lm", "na.omit", "p.adjust", "pchisq", "pnorm",
                 "quantile", "sd", "t.test", "wilcox.test")
    to your NAMESPACE file.
    ```

# corporaexplorer

<details>

* Version: 0.8.2
* Source code: https://github.com/cran/corporaexplorer
* URL: https://kgjerde.github.io/corporaexplorer, https://github.com/kgjerde/corporaexplorer
* BugReports: https://github.com/kgjerde/corporaexplorer/issues
* Date/Publication: 2020-03-07 19:40:02 UTC
* Number of recursive dependencies: 96

Run `revdep_details(,"corporaexplorer")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘ggplot2’ ‘rmarkdown’ ‘shinyWidgets’ ‘shinydashboard’
      ‘shinyjs’
      All declared Imports should be used.
    ```

# cosinor

<details>

* Version: 1.1
* Source code: https://github.com/cran/cosinor
* URL: http://github.com/sachsmc/cosinor
* Date/Publication: 2014-07-28 20:04:18
* Number of recursive dependencies: 65

Run `revdep_details(,"cosinor")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    cosinor_analyzer : <anonymous>: no visible global function definition
      for ‘as.formula’
    ggplot.cosinor.lm: no visible global function definition for ‘predict’
    predict.cosinor.lm: no visible global function definition for ‘fitted’
    predict.cosinor.lm: no visible global function definition for ‘predict’
    simulate_cosinor: no visible global function definition for ‘runif’
    simulate_cosinor: no visible global function definition for ‘rbinom’
    simulate_cosinor: no visible global function definition for ‘rnorm’
    summary.cosinor.lm: no visible global function definition for ‘vcov’
    summary.cosinor.lm: no visible global function definition for ‘qnorm’
    summary.cosinor.lm: no visible global function definition for ‘pnorm’
    test_cosinor: no visible global function definition for ‘pchisq’
    test_cosinor: no visible global function definition for ‘pnorm’
    Undefined global functions or variables:
      as.formula fitted lm na.omit pchisq pnorm predict qnorm rbinom rnorm
      runif terms vcov vitamind
    Consider adding
      importFrom("stats", "as.formula", "fitted", "lm", "na.omit", "pchisq",
                 "pnorm", "predict", "qnorm", "rbinom", "rnorm", "runif",
                 "terms", "vcov")
    to your NAMESPACE file.
    ```

# coveffectsplot

<details>

* Version: 0.0.9
* Source code: https://github.com/cran/coveffectsplot
* URL: https://github.com/smouksassi/interactiveforestplot
* BugReports: https://github.com/smouksassi/interactiveforestplot/issues
* Date/Publication: 2020-04-03 14:20:02 UTC
* Number of recursive dependencies: 103

Run `revdep_details(,"coveffectsplot")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colourpicker’ ‘data.table’ ‘dplyr’ ‘markdown’ ‘shinyjs’ ‘table1’
      ‘tidyr’
      All declared Imports should be used.
    ```

# CRANsearcher

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/CRANsearcher
* URL: https://github.com/RhoInc/CRANsearcher
* BugReports: https://github.com/RhoInc/CRANsearcher/issues
* Date/Publication: 2017-06-26 23:13:21 UTC
* Number of recursive dependencies: 44

Run `revdep_details(,"CRANsearcher")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 11 marked Latin-1 strings
      Note: found 57 marked UTF-8 strings
    ```

# crawl

<details>

* Version: 2.2.1
* Source code: https://github.com/cran/crawl
* Date/Publication: 2018-09-14 19:20:10 UTC
* Number of recursive dependencies: 110

Run `revdep_details(,"crawl")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gdistance’ ‘raster’
      All declared Imports should be used.
    ```

# crispRdesignR

<details>

* Version: 1.1.5
* Source code: https://github.com/cran/crispRdesignR
* URL: <https://github.com/dylanbeeber/crispRdesignR>
* Date/Publication: 2020-05-26 10:00:03 UTC
* Number of recursive dependencies: 62

Run `revdep_details(,"crispRdesignR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘gbm’
      All declared Imports should be used.
    ```

# crisprseekplus

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/crisprseekplus
* URL: https://github.com/UMMS-Biocore/crisprseekplus
* BugReports: https://github.com/UMMS-Biocore/crisprseekplus/issues/new
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 139

Run `revdep_details(,"crisprseekplus")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    installpack: no visible global function definition for
      'install.packages'
    Undefined global functions or variables:
      install.packages
    Consider adding
      importFrom("utils", "install.packages")
    to your NAMESPACE file.
    ```

# CrossICC

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/CrossICC
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 91

Run `revdep_details(,"CrossICC")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    .mergemodel: no visible global function definition for ‘coxph’
    CrossICC: no visible global function definition for ‘.’
    FAIME: no visible binding for global variable ‘SampleExp’
    get.max.var.rows: no visible binding for global variable ‘V1’
    predictor: no visible global function definition for ‘assay’
    runFAIME: no visible global function definition for ‘assay’
    Undefined global functions or variables:
      . SampleExp V1 assay coxph
    ```

# crossmeta

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/crossmeta
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 193

Run `revdep_details(,"crossmeta")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: ‘shinyBS’
    Unexported object imported by a ':::' call: ‘GEOquery:::parseGSEMatrix’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘getDirListing’ ‘getGEO’ ‘getGEOSuppFiles’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘download.file’
    getDirListing: no visible global function definition for ‘read.table’
    getGEOSuppFiles: no visible global function definition for
      ‘download.file’
    get_dfs: no visible global function definition for ‘head’
    load_agil_plat : <anonymous>: no visible global function definition for
      ‘capture.output’
    load_agil_plat: no visible global function definition for ‘as’
    load_illum_plat: no visible global function definition for ‘as’
    padog: no visible global function definition for ‘is’
    symbol_annot: no visible binding for global variable ‘SYMBOL_9606’
    which_max_iqr: no visible binding for global variable ‘iqrange’
    Undefined global functions or variables:
      SYMBOL_9606 as capture.output download.file head iqrange is
      read.table
    Consider adding
      importFrom("methods", "as", "is")
      importFrom("utils", "capture.output", "download.file", "head",
                 "read.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘PADOG’, ‘GeneMeta’
    ```

# CSUV

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CSUV
* BugReports: https://github.com/christineyuen/CSUV/issues
* Date/Publication: 2020-05-04 14:10:02 UTC
* Number of recursive dependencies: 108

Run `revdep_details(,"CSUV")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘parcor’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# CTAShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CTAShiny
* Date/Publication: 2019-04-13 08:22:55 UTC
* Number of recursive dependencies: 23

Run `revdep_details(,"CTAShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘epitools’ ‘rpivotTable’ ‘shinyMatrix’
      All declared Imports should be used.
    ```

# ctsem

<details>

* Version: 3.2.1
* Source code: https://github.com/cran/ctsem
* URL: https://github.com/cdriveraus/ctsem
* Date/Publication: 2020-04-26 01:00:07 UTC
* Number of recursive dependencies: 129

Run `revdep_details(,"ctsem")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.6Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        libs   6.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘numDeriv’ ‘pkgbuild’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# CTShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/CTShiny
* Date/Publication: 2019-05-03 12:51:01 UTC
* Number of recursive dependencies: 110

Run `revdep_details(,"CTShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘caret’ ‘datasets’ ‘dplyr’ ‘e1071’ ‘party’ ‘rhandsontable’ ‘rpart’
      ‘rpart.plot’ ‘shiny’
      All declared Imports should be used.
    ```

# CVE

<details>

* Version: 1.11.2
* Source code: https://github.com/cran/CVE
* Date/Publication: 2019-06-07
* Number of recursive dependencies: 183

Run `revdep_details(,"CVE")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘CVE-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: get.oncotator.anno
    > ### Title: Open Cancer Variant Explorer (CVE) Shiny app
    > ### Aliases: get.oncotator.anno
    > 
    > ### ** Examples
    > 
    > exCase <- data.frame(chr = rep(10, 3),
    +     start = c("100894110", "100985376", "101137905"),
    +     end = c("100894110", "100985376", "101137905"),
    +     ref_allele = c("T", "C", "G"),
    +     obs_allele = c("G", "A", "A"))
    > get.oncotator.anno(exCase)
    Error in open.connection(con, "rb") : HTTP error 502.
    Calls: get.oncotator.anno ... parse_and_simplify -> parseJSON -> parse_con -> open -> open.connection
    Execution halted
    ```

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                                    old_size new_size compress
      WGCNAmelanoma_extension.RData    2.4Mb    2.1Mb       xz
      crcCase.RData                    1.1Mb    678Kb       xz
      melanomaCase.RData               654Kb    473Kb       xz
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   4.4Mb
        doc    1.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RTCGAToolbox’
      All declared Imports should be used.
    Package in Depends field not imported from: ‘tidyverse’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# d3Tree

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/d3Tree
* URL: https://github.com/metrumresearchgroup/d3Tree
* BugReports: https://github.com/metrumresearchgroup/d3Tree/issues
* Date/Publication: 2017-06-13 20:35:10 UTC
* Number of recursive dependencies: 47

Run `revdep_details(,"d3Tree")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# datacheck

<details>

* Version: 1.2.2
* Source code: https://github.com/cran/datacheck
* URL: https://github.com/c5sire/datacheck
* BugReports: https://github.com/c5sire/datacheck/issues
* Date/Publication: 2015-04-18 08:56:10
* Number of recursive dependencies: 82

Run `revdep_details(,"datacheck")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    heatmap_quality: no visible global function definition for ‘heatmap’
    is.oneOf: no visible global function definition for ‘read.csv’
    is_one_of: no visible global function definition for ‘read.csv’
    pkg.version: no visible global function definition for ‘citation’
    pkg_version: no visible global function definition for ‘citation’
    ruleCoverage: no visible global function definition for ‘dotchart’
    ruleCoverage: no visible global function definition for ‘abline’
    rule_coverage: no visible global function definition for ‘dotchart’
    rule_coverage: no visible global function definition for ‘abline’
    shortSummary: no visible global function definition for ‘sd’
    short_summary: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      abline citation colorRampPalette dotchart heatmap read.csv sd
      setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "dotchart")
      importFrom("stats", "heatmap", "sd")
      importFrom("utils", "citation", "read.csv", "setTxtProgressBar",
                 "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    Package has no Sweave vignette sources and no VignetteBuilder field.
    ```

# datadigest

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/datadigest
* URL: https://github.com/RhoInc/datadigest
* BugReports: https://github.com/RhoInc/datadigest/issues
* Date/Publication: 2018-09-03 11:10:06 UTC
* Number of recursive dependencies: 92

Run `revdep_details(,"datadigest")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘miniUI’
      All declared Imports should be used.
    ```

# dataesgobr

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/dataesgobr
* Date/Publication: 2019-06-18 14:50:03 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"dataesgobr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘shinyFiles’ ‘shinycssloaders’ ‘shinyjs’ ‘shinythemes’
      All declared Imports should be used.
    ```

# debrowser

<details>

* Version: 1.16.1
* Source code: https://github.com/cran/debrowser
* URL: https://github.com/UMMS-Biocore/debrowser
* BugReports: https://github.com/UMMS-Biocore/debrowser/issues/new
* Date/Publication: 2020-05-14
* Number of recursive dependencies: 198

Run `revdep_details(,"debrowser")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    checkCountData : <anonymous>: no visible binding for global variable
      ‘err’
    debrowserdataload: no visible binding for global variable ‘demodata’
    debrowserhistogram: no visible global function definition for ‘config’
    getBarMainPlot: no visible global function definition for ‘config’
    getBoxMainPlot: no visible global function definition for ‘config’
    getDensityPlot: no visible binding for global variable ‘samples’
    getDensityPlot: no visible global function definition for ‘config’
    getIQRPlot: no visible global function definition for ‘config’
    mainScatterNew: no visible global function definition for ‘config’
    plot_pca: no visible binding for global variable ‘x’
    plot_pca: no visible binding for global variable ‘y’
    plot_pca: no visible binding for global variable ‘samples’
    plot_pca: no visible global function definition for ‘config’
    runHeatmap: no visible global function definition for ‘config’
    Undefined global functions or variables:
      config demodata err samples x y
    ```

# delayed

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/delayed
* URL: https://tlverse.org/delayed
* BugReports: https://github.com/tlverse/delayed/issues
* Date/Publication: 2020-02-28 11:40:02 UTC
* Number of recursive dependencies: 64

Run `revdep_details(,"delayed")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘assertthat’
      All declared Imports should be used.
    ```

# DEP

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/DEP
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 148

Run `revdep_details(,"DEP")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   1.4Mb
        doc    3.0Mb
    ```

# detzrcr

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/detzrcr
* URL: https://github.com/magnuskristoffersen/detzrcr
* Date/Publication: 2020-01-09 15:30:09 UTC
* Number of recursive dependencies: 71

Run `revdep_details(,"detzrcr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘utils’
      All declared Imports should be used.
    ```

# DiscoRhythm

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/DiscoRhythm
* URL: https://github.com/matthewcarlucci/DiscoRhythm
* BugReports: https://github.com/matthewcarlucci/DiscoRhythm/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 152

Run `revdep_details(,"DiscoRhythm")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    discoApp: no visible binding for global variable ‘.discorhythm_ncores’
    discoBatch: no visible binding for global variable ‘discoODAres’
    Undefined global functions or variables:
      .discorhythm_ncores discoODAres
    ```

# diveRsity

<details>

* Version: 1.9.90
* Source code: https://github.com/cran/diveRsity
* URL: http://diversityinlife.weebly.com/
* Date/Publication: 2017-04-04 10:59:38 UTC
* Number of recursive dependencies: 125

Run `revdep_details(,"diveRsity")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'sendplot', 'HWxtest'
    ```

# DLMtool

<details>

* Version: 5.4.5
* Source code: https://github.com/cran/DLMtool
* URL: http://www.datalimitedtoolkit.org/
* BugReports: https://github.com/DLMtool/DLMtool/issues
* Date/Publication: 2020-06-16 14:10:28 UTC
* Number of recursive dependencies: 127

Run `revdep_details(,"DLMtool")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   3.3Mb
        libs   1.1Mb
    ```

# Doscheda

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/Doscheda
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 157

Run `revdep_details(,"Doscheda")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              1.6Mb
        shiny-examples   2.0Mb
    ```

# dosedesignR

<details>

* Version: 0.2.4
* Source code: https://github.com/cran/dosedesignR
* Date/Publication: 2020-04-23 17:10:02 UTC
* Number of recursive dependencies: 77

Run `revdep_details(,"dosedesignR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘DoseFinding’ ‘ggplot2’ ‘lattice’ ‘latticeExtra’ ‘shinyjs’
      All declared Imports should be used.
    ```

# dplyrAssist

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/dplyrAssist
* URL: https://github.com/cardiomoon/dplyrAssist
* BugReports: https://github.com/cardiomoon/dplyrAssist/issues
* Date/Publication: 2017-09-04 17:46:22 UTC
* Number of recursive dependencies: 109

Run `revdep_details(,"dplyrAssist")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘tidyr’ ‘tidyverse’
      All declared Imports should be used.
    ```

# dragon

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/dragon
* URL: https://github.com/spielmanlab/dragon
* BugReports: https://github.com/spielmanlab/dragon/issues
* Date/Publication: 2019-10-20 07:10:06 UTC
* Number of recursive dependencies: 123

Run `revdep_details(,"dragon")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘RColorBrewer’ ‘broom’ ‘colorspace’ ‘colourpicker’ ‘cowplot’
      ‘igraph’ ‘magrittr’ ‘shinyBS’ ‘shinyWidgets’ ‘shinydashboard’
      ‘shinythemes’ ‘tidyverse’ ‘visNetwork’
      All declared Imports should be used.
    ```

# dropR

<details>

* Version: 0.1
* Source code: https://github.com/cran/dropR
* Date/Publication: 2015-01-02 06:52:56
* Number of recursive dependencies: 19

Run `revdep_details(,"dropR")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    compute_xsq: no visible global function definition for ‘chisq.test’
    Undefined global functions or variables:
      chisq.test
    Consider adding
      importFrom("stats", "chisq.test")
    to your NAMESPACE file.
    ```

# dtwclust

<details>

* Version: 5.5.6
* Source code: https://github.com/cran/dtwclust
* URL: https://github.com/asardaes/dtwclust
* BugReports: https://github.com/asardaes/dtwclust/issues
* Date/Publication: 2019-12-11 08:20:02 UTC
* Number of recursive dependencies: 94

Run `revdep_details(,"dtwclust")` for more info

</details>

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# DVHmetrics

<details>

* Version: 0.3.10
* Source code: https://github.com/cran/DVHmetrics
* URL: https://github.com/dwoll/DVHmetrics/
* Date/Publication: 2020-03-19 22:10:03 UTC
* Number of recursive dependencies: 72

Run `revdep_details(,"DVHmetrics")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# dynamichazard

<details>

* Version: 0.6.6
* Source code: https://github.com/cran/dynamichazard
* URL: https://github.com/boennecd/dynamichazard
* BugReports: https://github.com/boennecd/dynamichazard/issues
* Date/Publication: 2019-10-14 08:50:02 UTC
* Number of recursive dependencies: 79

Run `revdep_details(,"dynamichazard")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.9Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
        doc    1.7Mb
        libs   2.7Mb
    ```

# EBImage

<details>

* Version: 4.30.0
* Source code: https://github.com/cran/EBImage
* URL: https://github.com/aoles/EBImage
* BugReports: https://github.com/aoles/EBImage/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 43

Run `revdep_details(,"EBImage")` for more info

</details>

## In both

*   checking whether package ‘EBImage’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      morphology.cpp:163:14: warning: expression result unused [-Wunused-value]
      morphology.cpp:166:14: warning: expression result unused [-Wunused-value]
      morphology.cpp:170:14: warning: expression result unused [-Wunused-value]
      morphology.cpp:196:14: warning: expression result unused [-Wunused-value]
      morphology.cpp:199:14: warning: expression result unused [-Wunused-value]
      morphology.cpp:203:14: warning: expression result unused [-Wunused-value]
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex/EBImage/new/EBImage.Rcheck/00install.out’ for details.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        doc      5.1Mb
        images   1.7Mb
    ```

# effectR

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/effectR
* Date/Publication: 2018-09-30 21:30:03 UTC
* Number of recursive dependencies: 76

Run `revdep_details(,"effectR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rmarkdown’
      All declared Imports should be used.
    ```

# elementR

<details>

* Version: 1.3.6
* Source code: https://github.com/cran/elementR
* URL: https://github.com/charlottesirot/elementR
* BugReports: https://github.com/charlottesirot/elementR/issues
* Date/Publication: 2018-05-03 14:08:24 UTC
* Number of recursive dependencies: 121

Run `revdep_details(,"elementR")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘elementR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: elementR_project
    > ### Title: Object elementR_project
    > ### Aliases: elementR_project
    > 
    > ### ** Examples
    > 
    > ## create a new elementR_repStandard object based on the "filePath" 
    > ## from a folder containing sample replicate
    > 
    > filePath <- system.file("Example_Session", package="elementR")
    > 
    > exampleProject <- elementR_project$new(filePath)
    Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
      [tcl] invalid command name "toplevel".
    Calls: <Anonymous> ... tktoplevel -> tkwidget -> tcl -> .Tcl.objv -> structure
    Execution halted
    ```

# embryogrowth

<details>

* Version: 7.6
* Source code: https://github.com/cran/embryogrowth
* Date/Publication: 2019-08-30 07:10:02 UTC
* Number of recursive dependencies: 81

Run `revdep_details(,"embryogrowth")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 5 marked UTF-8 strings
    ```

# EML

<details>

* Version: 2.0.2
* Source code: https://github.com/cran/EML
* URL: https://docs.ropensci.org/EML, https://github.com/ropensci/EML
* BugReports: https://github.com/ropensci/EML/issues
* Date/Publication: 2020-02-08 06:10:15 UTC
* Number of recursive dependencies: 91

Run `revdep_details(,"EML")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘EML’
    ```

# ENCODExplorer

<details>

* Version: 2.14.0
* Source code: https://github.com/cran/ENCODExplorer
* BugReports: https://github.com/CharlesJB/ENCODExplorer/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 98

Run `revdep_details(,"ENCODExplorer")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    createDesign: no visible binding for global variable ‘file_format’
    createDesign: no visible binding for global variable ‘status’
    createDesign: no visible binding for global variable ‘accession’
    createDesign: no visible binding for global variable ‘controls’
    createDesign : get_ctrl_design: no visible binding for global variable
      ‘accession’
    createDesign : get_ctrl_design: no visible binding for global variable
      ‘href’
    createDesign: no visible global function definition for ‘.’
    createDesign: no visible binding for global variable ‘href’
    createDesign: no visible binding for global variable ‘Experiment’
    createDesign: no visible binding for global variable ‘Value’
    downloadEncode: no visible binding for global variable ‘file_accession’
    downloadEncode: no visible binding for global variable ‘file_format’
    downloadEncode: no visible binding for global variable ‘accession’
    shinyEncode: no visible binding for global variable ‘ui’
    shinyEncode: no visible binding for global variable ‘server’
    Undefined global functions or variables:
      . Experiment Value accession controls file_accession file_format href
      server status ui
    ```

# enviGCMS

<details>

* Version: 0.6.6
* Source code: https://github.com/cran/enviGCMS
* URL: https://github.com/yufree/enviGCMS
* Date/Publication: 2020-06-04 15:20:06 UTC
* Number of recursive dependencies: 138

Run `revdep_details(,"enviGCMS")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘xcms’
    ```

# epicontacts

<details>

* Version: 1.1.0
* Source code: https://github.com/cran/epicontacts
* URL: http://www.repidemicsconsortium.org/epicontacts/
* BugReports: http://github.com/reconhub/epicontacts/issues
* Date/Publication: 2017-11-21 08:24:53 UTC
* Number of recursive dependencies: 76

Run `revdep_details(,"epicontacts")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘colorspace’
      All declared Imports should be used.
    ```

# EpiSignalDetection

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/EpiSignalDetection
* URL: http://atlas.ecdc.europa.eu/public/index.aspx
* Date/Publication: 2018-11-15 22:00:04 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"EpiSignalDetection")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘ggplot2’ ‘knitr’ ‘pander’
      All declared Imports should be used.
    ```

# epivizrChart

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/epivizrChart
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 172

Run `revdep_details(,"epivizrChart")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 25.1Mb
      sub-directories of 1Mb or more:
        data   3.3Mb
        doc   17.3Mb
        www    3.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'epivizrServer'
      All declared Imports should be used.
    ```

# erma

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/erma
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 139

Run `revdep_details(,"erma")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 218.3Mb
      sub-directories of 1Mb or more:
        bed_tabix  178.2Mb
        data        37.1Mb
        doc          2.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    geneTxRange: no visible global function definition for ‘select’
    genemodelOLD: no visible binding for global variable ‘exonsBy’
    stateProfile: no visible binding for global variable ‘states_25’
    stateProfile: no visible binding for global variable ‘short_celltype’
    stateProfile: no visible binding for global variable ‘name’
    statesByRange: no visible binding for global variable ‘mod’
    statesByRange: no visible binding for global variable ‘upstream’
    statesByRange: no visible binding for global variable ‘downstream’
    statesByRange : <anonymous>: no visible binding for global variable
      ‘tss’
    statesByRange: no visible binding for global variable ‘states_25’
    Undefined global functions or variables:
      downstream exonsBy mod name select short_celltype states_25 tss
      upstream
    ```

# EventStudy

<details>

* Version: 0.36
* Source code: https://github.com/cran/EventStudy
* URL: http://eventstudytools.com
* BugReports: https://github.com/EventStudyTools/api-wrapper.r/issues
* Date/Publication: 2019-03-14 12:13:31 UTC
* Number of recursive dependencies: 132

Run `revdep_details(,"EventStudy")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        doc   5.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘curl’ ‘openxlsx’ ‘stringr’
      All declared Imports should be used.
    ```

# ezplot

<details>

* Version: 0.6.1
* Source code: https://github.com/cran/ezplot
* Date/Publication: 2020-06-13 14:30:02 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"ezplot")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tsibble’
      All declared Imports should be used.
    ```

# farrell

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/farrell
* URL: https://github.com/feddelegrand7/farrell
* BugReports: https://github.com/feddelegrand7/farrell/issues
* Date/Publication: 2020-06-03 05:20:14 UTC
* Number of recursive dependencies: 57

Run `revdep_details(,"farrell")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rlang’
      All declared Imports should be used.
    ```

# FastqCleaner

<details>

* Version: 1.6.0
* Source code: https://github.com/cran/FastqCleaner
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 86

Run `revdep_details(,"FastqCleaner")` for more info

</details>

## In both

*   checking for portable file names ... NOTE
    ```
    Found the following non-portable file paths:
      FastqCleaner/inst/application/www/help/docs/articles/Overview_files/figure-html/unnamed-chunk-11-1.png
      FastqCleaner/inst/application/www/help/docs/articles/Overview_files/figure-html/unnamed-chunk-12-1.png
      FastqCleaner/inst/application/www/help/docs/articles/Overview_files/figure-html/unnamed-chunk-13-1.png
      FastqCleaner/inst/application/www/help/docs/articles/Overview_files/figure-html/unnamed-chunk-14-1.png
      FastqCleaner/inst/application/www/help/docs/articles/Overview_files/figure-html/unnamed-chunk-7-1.png
      FastqCleaner/inst/application/www/help/docs/articles/Overview_files/figure-html/unnamed-chunk-8-1.png
    
    Tarballs are only required to store paths of up to 100 bytes and cannot
    store those of more than 256 bytes, with restrictions including to 100
    bytes for the final component.
    See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        application   4.2Mb
        doc           1.1Mb
    ```

# FELLA

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/FELLA
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 101

Run `revdep_details(,"FELLA")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘musmusculus.Rmd’ using ‘UTF-8’... OK
      ‘quickstart.Rmd’ using ‘UTF-8’... OK
      ‘zebrafish.Rmd’ using ‘UTF-8’... OK
      ‘FELLA.Rnw’ using ‘UTF-8’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘FELLA.Rnw’
      ...
    
    > g.go <- addGOToGraph(graph = g, GOterm = "GO:0005739", 
    +     godata.options = list(OrgDb = "org.Hs.eg.db", ont = "CC"), 
    +     mart.options = list( .... [TRUNCATED] 
    Ensembl site unresponsive, trying useast mirror
    Ensembl site unresponsive, trying uswest mirror
    
      When sourcing ‘FELLA.R’:
    Error: Multiple cache results found
    Execution halted
    ```

*   checking R code for possible problems ... NOTE
    ```
    buildGraphFromKEGGREST: no visible global function definition for
      ‘capture.output’
    Undefined global functions or variables:
      capture.output
    Consider adding
      importFrom("utils", "capture.output")
    to your NAMESPACE file.
    ```

# firebase

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/firebase
* URL: https://firebase.john-coene.com/, https://github.com/JohnCoene/firebase
* BugReports: https://github.com/JohnCoene/firebase/issues
* Date/Publication: 2020-03-30 15:50:05 UTC
* Number of recursive dependencies: 22

Run `revdep_details(,"firebase")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# fitteR

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/fitteR
* Date/Publication: 2017-12-11 14:27:45 UTC
* Number of recursive dependencies: 199

Run `revdep_details(,"fitteR")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ExtDist’
    ```

# fitur

<details>

* Version: 0.6.1
* Source code: https://github.com/cran/fitur
* URL: https://github.com/tomroh/fitur
* BugReports: https://github.com/tomroh/fitur/issues
* Date/Publication: 2018-09-01 19:20:03 UTC
* Number of recursive dependencies: 79

Run `revdep_details(,"fitur")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstudioapi’
      All declared Imports should be used.
    ```

# flair

<details>

* Version: 0.0.2
* Source code: https://github.com/cran/flair
* URL: https://github.com/kbodwin/flair, https://kbodwin.github.io/flair/index.html
* BugReports: https://github.com/kbodwin/flair/issues
* Date/Publication: 2020-04-23 18:00:05 UTC
* Number of recursive dependencies: 76

Run `revdep_details(,"flair")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘evaluate’ ‘ggplot2’
      All declared Imports should be used.
    ```

# flora

<details>

* Version: 0.3.4
* Source code: https://github.com/cran/flora
* URL: http://www.github.com/gustavobio/flora
* BugReports: http://www.github.com/gustavobio/flora/issues
* Date/Publication: 2020-04-28 05:00:03 UTC
* Number of recursive dependencies: 53

Run `revdep_details(,"flora")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        R   7.1Mb
    ```

# flowAI

<details>

* Version: 1.18.2
* Source code: https://github.com/cran/flowAI
* Date/Publication: 2020-06-16
* Number of recursive dependencies: 80

Run `revdep_details(,"flowAI")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

# flowcatchR

<details>

* Version: 1.22.0
* Source code: https://github.com/cran/flowcatchR
* URL: https://github.com/federicomarini/flowcatchR, https://federicomarini.github.io/flowcatchR/
* BugReports: https://github.com/federicomarini/flowcatchR/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 103

Run `revdep_details(,"flowcatchR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.0Mb
      sub-directories of 1Mb or more:
        data   1.9Mb
        doc    9.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘EBImage’ which was already attached by Depends.
      Please remove these calls from your code.
    ```

# frailtypack

<details>

* Version: 3.3.0
* Source code: https://github.com/cran/frailtypack
* URL: https://virginie1rondeau.wixsite.com/virginierondeau/software-frailtypack https://frailtypack-pkg.shinyapps.io/shiny_frailtypack
* Date/Publication: 2020-06-15 06:00:12 UTC
* Number of recursive dependencies: 82

Run `revdep_details(,"frailtypack")` for more info

</details>

## In both

*   checking whether package ‘frailtypack’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/frailtypack/new/frailtypack.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘frailtypack’ ...
** package ‘frailtypack’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  Adonnees.f90 -o Adonnees.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  AparamMultive.f90 -o AparamMultive.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  Aparameters.f90 -o Aparameters.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  aGhermite.f90 -o aGhermite.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  ahrmsym.f90 -o ahrmsym.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  funcpajsplinesIndiv.f90 -o funcpajsplinesIndiv.o
clang -mmacosx-version-min=10.13 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include   -fPIC  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -c init.c -o init.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  risque.f90 -o risque.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  somme.f90 -o somme.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  test.f90 -o test.o
funcpajsplinesIndiv.f90:6:8:

     use comon,only:mm,mm3,mm2,mm1,& !m3m3,m2m2,m1m1,mmm,m3m2,m3m1,m3m,m2m1,m2m,m1m
        1
Fatal Error: Can't open module file ‘comon.mod’ for reading at (1): No such file or directory
compilation terminated.
make: *** [funcpajsplinesIndiv.o] Error 1
make: *** Waiting for unfinished jobs....
somme.f90:7:47:

     integer:: i,num_thread,num_proc,nb_thread,d
                                               1
Warning: Unused variable ‘d’ declared at (1) [-Wunused-variable]
somme.f90:7:45:

     integer:: i,num_thread,num_proc,nb_thread,d
                                             1
Warning: Unused variable ‘nb_thread’ declared at (1) [-Wunused-variable]
somme.f90:1:31:

 subroutine somme(ab,s,nboot,nbr,vOut)
                               1
Warning: Unused dummy argument ‘nbr’ at (1) [-Wunused-dummy-argument]
somme.f90:7:35:

     integer:: i,num_thread,num_proc,nb_thread,d
                                   1
Warning: Unused variable ‘num_proc’ declared at (1) [-Wunused-variable]
ERROR: compilation failed for package ‘frailtypack’
* removing ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/frailtypack/new/frailtypack.Rcheck/frailtypack’

```
### CRAN

```
* installing *source* package ‘frailtypack’ ...
** package ‘frailtypack’ successfully unpacked and MD5 sums checked
** using staged installation
** libs
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  Adonnees.f90 -o Adonnees.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  AparamMultive.f90 -o AparamMultive.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  Aparameters.f90 -o Aparameters.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  aGhermite.f90 -o aGhermite.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  ahrmsym.f90 -o ahrmsym.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  funcpajsplinesIndiv.f90 -o funcpajsplinesIndiv.o
clang -mmacosx-version-min=10.13 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -I/usr/local/include   -fPIC  -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk -c init.c -o init.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  risque.f90 -o risque.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  somme.f90 -o somme.o
gfortran -mmacosx-version-min=10.13 -fno-optimize-sibling-calls   -fPIC  -Wall -g -O2  -c  test.f90 -o test.o
funcpajsplinesIndiv.f90:4:8:

     use tailles
        1
Fatal Error: Can't open module file ‘tailles.mod’ for reading at (1): No such file or directory
compilation terminated.
somme.f90:7:47:

     integer:: i,num_thread,num_proc,nb_thread,d
                                               1
Warning: Unused variable ‘d’ declared at (1) [-Wunused-variable]
somme.f90:7:45:

     integer:: i,num_thread,num_proc,nb_thread,d
                                             1
Warning: Unused variable ‘nb_thread’ declared at (1) [-Wunused-variable]
somme.f90:1:31:

 subroutine somme(ab,s,nboot,nbr,vOut)
                               1
Warning: Unused dummy argument ‘nbr’ at (1) [-Wunused-dummy-argument]
somme.f90:7:35:

     integer:: i,num_thread,num_proc,nb_thread,d
                                   1
Warning: Unused variable ‘num_proc’ declared at (1) [-Wunused-variable]
make: *** [funcpajsplinesIndiv.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘frailtypack’
* removing ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/frailtypack/old/frailtypack.Rcheck/frailtypack’

```
# FSK2R

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/FSK2R
* Date/Publication: 2020-05-13 15:00:02 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"FSK2R")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: 'googlesheets', 'readtext'
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# FunChIP

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/FunChIP
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 50

Run `revdep_details(,"FunChIP")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘fda’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# g3viz

<details>

* Version: 1.1.2
* Source code: https://github.com/cran/g3viz
* URL: https://github.com/G3viz/g3viz
* BugReports: https://github.com/G3viz/g3viz/issues
* Date/Publication: 2019-05-29 21:30:10 UTC
* Number of recursive dependencies: 67

Run `revdep_details(,"g3viz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.4Mb
      sub-directories of 1Mb or more:
        doc   9.3Mb
    ```

# GA4GHshiny

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/GA4GHshiny
* URL: https://github.com/labbcb/GA4GHshiny
* BugReports: https://github.com/labbcb/GA4GHshiny/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 121

Run `revdep_details(,"GA4GHshiny")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 4 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 12 ]
      1. Error: app works (@test-app.R#5) 
      2. Error: initializeReferences works (@test-initializeReferences.R#6) 
      3. Error: initializeVariantSet works (@test-initializeVariantSet.R#6) 
      4. Error: searchVariantsByGeneSymbol works (@test-searchVariantsByGeneSymbol.R#9) 
      5. Error: searchVariantsByGeneSymbol feature=transcripts works (@test-searchVariantsByGeneSymbol.R#22) 
      6. Error: searchVariantsByGeneSymbol feature=exons works (@test-searchVariantsByGeneSymbol.R#35) 
      7. Error: searchVariantsByGeneSymbol feature=cds works (@test-searchVariantsByGeneSymbol.R#48) 
      8. Error: searchVariantsByGeneSymbol feature=promoters works (@test-searchVariantsByGeneSymbol.R#60) 
      9. Error: searchVariantsByGeneSymbol multiple genes works (@test-searchVariantsByGeneSymbol.R#72) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# gamesGA

<details>

* Version: 1.1.3.7
* Source code: https://github.com/cran/gamesGA
* URL: https://bradduthie.github.io/gamesGA/
* BugReports: https://github.com/bradduthie/gamesGA/issues
* Date/Publication: 2020-03-01 21:00:02 UTC
* Number of recursive dependencies: 37

Run `revdep_details(,"gamesGA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# gastempt

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/gastempt
* URL: http://github.com/dmenne/gastempt
* BugReports: http://github.com/dmenne/gastempt/issues
* Date/Publication: 2020-05-01 10:10:02 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"gastempt")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        libs   4.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# GDCRNATools

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/GDCRNATools
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 214

Run `revdep_details(,"GDCRNATools")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.1Mb
      sub-directories of 1Mb or more:
        R      3.2Mb
        data   1.1Mb
        doc    2.6Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    gdcRNAMerge: no visible global function definition for ‘read.table’
    gdcRNAMerge : <anonymous>: no visible global function definition for
      ‘read.delim’
    gdcRNAMerge: no visible global function definition for ‘read.delim’
    hyperTestFun: no visible global function definition for ‘phyper’
    kmTestFun: no visible global function definition for ‘pchisq’
    kmTestFun: no visible global function definition for ‘qnorm’
    manifestDownloadFun: no visible global function definition for
      ‘read.table’
    mirCorTestFun: no visible global function definition for ‘cor.test’
    multiRegFun: no visible global function definition for ‘cor.test’
    Undefined global functions or variables:
      Category Counts FDR Regulation Terms URLencode aggregate cor.test
      download.file foldEnrichment model.matrix p.adjust pchisq phyper
      qnorm read.delim read.table unzip write.table
    Consider adding
      importFrom("stats", "aggregate", "cor.test", "model.matrix",
                 "p.adjust", "pchisq", "phyper", "qnorm")
      importFrom("utils", "URLencode", "download.file", "read.delim",
                 "read.table", "unzip", "write.table")
    to your NAMESPACE file.
    ```

# genBart

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/genBart
* Date/Publication: 2018-03-13 22:13:14 UTC
* Number of recursive dependencies: 146

Run `revdep_details(,"genBart")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘NMF’ ‘RColorBrewer’ ‘VennDiagram’ ‘clValid’ ‘data.table’ ‘grid’
      ‘gtools’ ‘pca3d’ ‘rmarkdown’ ‘scales’ ‘shinydashboard’ ‘shinyjs’
      ‘statmod’ ‘stringr’
      All declared Imports should be used.
    ```

# GeneNetworkBuilder

<details>

* Version: 1.30.0
* Source code: https://github.com/cran/GeneNetworkBuilder
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 73

Run `revdep_details(,"GeneNetworkBuilder")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.6Mb
      sub-directories of 1Mb or more:
        data          2.8Mb
        doc           6.5Mb
        htmlwidgets   1.8Mb
    ```

# GenEst

<details>

* Version: 1.4.4
* Source code: https://github.com/cran/GenEst
* Date/Publication: 2020-06-08 18:20:03 UTC
* Number of recursive dependencies: 83

Run `revdep_details(,"GenEst")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmlwidgets’
      All declared Imports should be used.
    ```

# GeneTonic

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/GeneTonic
* URL: https://github.com/federicomarini/GeneTonic
* BugReports: https://github.com/federicomarini/GeneTonic/issues
* Date/Publication: 2020-05-28
* Number of recursive dependencies: 176

Run `revdep_details(,"GeneTonic")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.9Mb
      sub-directories of 1Mb or more:
        doc  12.7Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘pcaExplorer’
    ```

# GenomicScores

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/GenomicScores
* URL: https://github.com/rcastelo/GenomicScores
* BugReports: https://github.com/rcastelo/GenomicScores/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 142

Run `revdep_details(,"GenomicScores")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    +   score(gsco, gr1)
    +   gscores(gsco, gr2)
    +   populations(gsco)
    +   gscores(gsco, gr2, pop="DP2")
    + }
    Loading required package: phastCons100way.UCSC.hg19
    Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
      there is no package called ‘phastCons100way.UCSC.hg19’
    > 
    > ## supporting AnnotationHub resources
    > ## Not run: 
    > ##D availableGScores()
    > ##D gsco <- getGScores("phastCons100way.UCSC.hg19")
    > ##D gsco
    > ##D gscores(gsco, gr1)
    > ## End(Not run)
    > 
    > ## metadata from a GScores object
    > name(gsco)
    Error in name(gsco) : object 'gsco' not found
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'phastCons100way.UCSC.hg19', 'MafDb.1Kgenomes.phase1.hs37d5'
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/gscores-template/inst/extdata/.gitkeep
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        doc       1.5Mb
        extdata   2.8Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘phastCons100way.UCSC.hg19’, ‘MafDb.1Kgenomes.phase1.hs37d5’
    ```

# GerminaR

<details>

* Version: 1.4.2
* Source code: https://github.com/cran/GerminaR
* URL: https://flavjack.github.io/germinaquant/
* BugReports: https://github.com/flavjack/germinar/issues
* Date/Publication: 2020-03-28 11:30:05 UTC
* Number of recursive dependencies: 111

Run `revdep_details(,"GerminaR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘assertthat’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# ggedit

<details>

* Version: 0.3.1
* Source code: https://github.com/cran/ggedit
* URL: https://github.com/yonicd/ggedit
* BugReports: https://github.com/yonicd/ggedit/issues
* Date/Publication: 2020-06-02 11:50:06 UTC
* Number of recursive dependencies: 80

Run `revdep_details(,"ggedit")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# ggExtra

<details>

* Version: 0.9
* Source code: https://github.com/cran/ggExtra
* URL: https://github.com/daattali/ggExtra
* BugReports: https://github.com/daattali/ggExtra/issues
* Date/Publication: 2019-08-27 04:30:02 UTC
* Number of recursive dependencies: 110

Run `revdep_details(,"ggExtra")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘grDevices’ ‘scales’ ‘utils’
      All declared Imports should be used.
    ```

# ggplotAssist

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/ggplotAssist
* URL: https://github.com/cardiomoon/ggplotAssist
* BugReports: https://github.com/cardiomoon/ggplotAssist/issues
* Date/Publication: 2017-11-12 12:24:46 UTC
* Number of recursive dependencies: 117

Run `revdep_details(,"ggplotAssist")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gcookbook’ ‘ggthemes’ ‘moonBook’ ‘tidyverse’
      All declared Imports should be used.
    ```

# ggquickeda

<details>

* Version: 0.1.6
* Source code: https://github.com/cran/ggquickeda
* URL: https://github.com/smouksassi/ggquickeda
* BugReports: https://github.com/smouksassi/ggquickeda/issues
* Date/Publication: 2020-04-17 14:00:03 UTC
* Number of recursive dependencies: 157

Run `revdep_details(,"ggquickeda")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘Formula’ ‘GGally’ ‘Hmisc’ ‘colourpicker’ ‘dplyr’ ‘ggpmisc’
      ‘ggpubr’ ‘ggrepel’ ‘ggstance’ ‘grDevices’ ‘grid’ ‘gridExtra’
      ‘lazyeval’ ‘markdown’ ‘plotly’ ‘quantreg’ ‘rlang’ ‘shinyjqui’
      ‘shinyjs’ ‘survival’ ‘survminer’ ‘table1’ ‘tidyr’
      All declared Imports should be used.
    ```

# gmDatabase

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/gmDatabase
* URL: http://www.r-project.org, http://www.stat.boogaart.de
* Date/Publication: 2016-06-16 12:32:05
* Number of recursive dependencies: 24

Run `revdep_details(,"gmDatabase")` for more info

</details>

## In both

*   checking whether the package can be unloaded cleanly ... WARNING
    ```
    Error: package ‘RMySQL’ could not be loaded
    Execution halted
    ```

*   checking whether the namespace can be loaded with stated dependencies ... WARNING
    ```
    Error in dyn.load(file, DLLpath = DLLpath, ...) : 
      unable to load shared object '/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/gmDatabase/RMySQL/libs/RMySQL.so':
      dlopen(/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/gmDatabase/RMySQL/libs/RMySQL.so, 6): Symbol not found: _GSS_C_NT_USER_NAME
      Referenced from: /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/gmDatabase/RMySQL/libs/RMySQL.so
      Expected in: flat namespace
     in /Users/barret/Documents/git/rstudio/shiny/shiny/revdep/library.noindex/gmDatabase/RMySQL/libs/RMySQL.so
    Calls: <Anonymous> ... namespaceImport -> loadNamespace -> library.dynam -> dyn.load
    Execution halted
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Error: package ‘RMySQL’ could not be loaded
    Call sequence:
    3: stop(gettextf("package %s could not be loaded", sQuote(pkg)), 
           call. = FALSE, domain = NA)
    2: .getRequiredPackages2(pkgInfo, quietly = quietly)
    1: library(package, lib.loc = lib.loc, character.only = TRUE, verbose = FALSE)
    Execution halted
    ```

# GmicR

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/GmicR
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 133

Run `revdep_details(,"GmicR")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components with restrictions not permitted:
      GPL-2 + file LICENSE
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    GSEAGO_Builder: no visible binding for global variable ‘ex’
    Undefined global functions or variables:
      ex
    ```

# GOFShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/GOFShiny
* Date/Publication: 2019-07-31 09:30:03 UTC
* Number of recursive dependencies: 31

Run `revdep_details(,"GOFShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rhandsontable’ ‘shiny’ ‘stats’
      All declared Imports should be used.
    ```

# GPA

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/GPA
* URL: http://dongjunchung.github.io/GPA/
* BugReports: https://github.com/dongjunchung/GPA/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 70

Run `revdep_details(,"GPA")` for more info

</details>

## In both

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# gQTLstats

<details>

* Version: 1.20.0
* Source code: https://github.com/cran/gQTLstats
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 192

Run `revdep_details(,"gQTLstats")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 65.7Mb
      sub-directories of 1Mb or more:
        data        11.0Mb
        doc          1.1Mb
        registries  18.8Mb
        vcf         33.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘permScore_3’
    storeToMaxAssocBySNP: no visible global function definition for ‘nth’
    storeToMaxAssocBySNP: no visible binding for global variable ‘MAF’
    storeToMaxAssocBySNP: no visible binding for global variable ‘probeid’
    storeToMaxAssocBySNP: no visible binding for global variable ‘mindist’
    tqbrowser: no visible global function definition for ‘experiments’
    tqbrowser : server: no visible global function definition for
      ‘experiments’
    tqbrowser : server: no visible global function definition for
      ‘TabixFile’
    tqbrowser : server: no visible binding for global variable ‘assoc’
    tqbrowser : server: no visible binding for global variable ‘stateid’
    tqbrowser : server: no visible binding for global variable ‘state’
    transTable: no visible binding for global variable ‘i’
    tsByRank_sing: no visible binding for global variable ‘i’
    tsByRank_sing : <anonymous>: no visible binding for global variable ‘i’
    boxswarm,SnpToGeneQTL: no visible binding for global variable ‘g1’
    Undefined global functions or variables:
      DNAStringSetList MAF TabixFile TxDb assoc calls ch chisq criterion ex
      exonsBy experiments g1 geom_boxplot gt i id maf mindist ml10fdr nth
      permScore_1 permScore_2 permScore_3 probeid snp state stateid value x
    ```

*   checking contents of ‘data’ directory ... NOTE
    ```
    Output for data("bigfiltFDR", package = "gQTLstats"):
      
      
    Output for data("bigrawFDR", package = "gQTLstats"):
      
      
    Output for data("filtFDR", package = "gQTLstats"):
      
      
    Output for data("rawFDR", package = "gQTLstats"):
      
      
    Output for data("sensByProbe", package = "gQTLstats"):
      
      
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 8 marked Latin-1 strings
      Note: found 12 marked UTF-8 strings
    ```

# gravitas

<details>

* Version: 0.1.2
* Source code: https://github.com/cran/gravitas
* URL: https://github.com/Sayani07/gravitas/
* BugReports: https://github.com/Sayani07/gravitas/issues
* Date/Publication: 2020-02-17 09:20:02 UTC
* Number of recursive dependencies: 93

Run `revdep_details(,"gravitas")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    > library(dplyr)
    
    Attaching package: ‘dplyr’
    
    The following objects are masked from ‘package:stats’:
    
        filter, lag
    
    The following objects are masked from ‘package:base’:
    
        intersect, setdiff, setequal, union
    
    > library(ggplot2)
    > library(lvplot)
    > # Search for granularities
    > smart_meter10 %>%
    +   search_gran(highest_unit = "week")
    Error in if (tsibble::is_regular(.data)) { : argument is of length zero
    Calls: %>% ... freduce -> withVisible -> <Anonymous> -> search_gran
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > 
      > test_check("gravitas")
      ── 1. Error: norun check runs in create_gran (@test-harmony.R#48)  ─────────────
      argument is of length zero
      Backtrace:
        1. testthat::expect_equal(...)
       10. gravitas::harmony(., ugran = "day", filter_in = "wknd_wday")
       13. gravitas::search_gran(...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 126 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 1 ]
      1. Error: norun check runs in create_gran (@test-harmony.R#48) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# gridsampler

<details>

* Version: 0.6
* Source code: https://github.com/cran/gridsampler
* URL: https://github.com/markheckmann/gridsampler
* BugReports: https://github.com/markheckmann/gridsampler/issues
* Date/Publication: 2016-11-23 17:24:13
* Number of recursive dependencies: 72

Run `revdep_details(,"gridsampler")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiasedUrn’
      All declared Imports should be used.
    ```

# GSCA

<details>

* Version: 2.17.0
* Source code: https://github.com/cran/GSCA
* Date/Publication: 2019-10-29
* Number of recursive dependencies: 75

Run `revdep_details(,"GSCA")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      'shiny', 'sp', 'gplots', 'ggplot2', 'reshape2', 'RColorBrewer',
      'rhdf5'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Affyhgu133A2Expr’ ‘Affyhgu133Plus2Expr’ ‘Affyhgu133aExpr’
      ‘Affymoe4302Expr’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    GSCAplot: no visible global function definition for ‘title’
    GSCAplot: no visible global function definition for ‘dev.off’
    annotatePeaks: no visible binding for global variable ‘allreffile’
    tabSearch: no visible global function definition for ‘data’
    tabSearch: no visible binding for global variable ‘Affyhgu133aExprtab’
    tabSearch: no visible binding for global variable ‘Affymoe4302Exprtab’
    tabSearch: no visible binding for global variable ‘Affyhgu133A2Exprtab’
    tabSearch: no visible binding for global variable
      ‘Affyhgu133Plus2Exprtab’
    Undefined global functions or variables:
      Affyhgu133A2Exprtab Affyhgu133Plus2Exprtab Affyhgu133aExprtab
      Affymoe4302Exprtab P.value SampleType Var1 Var2 allreffile
      colorRampPalette data dev.off fisher.test geneid hist par pdf qnorm
      quantile sd str t.stat t.test title value variable write.csv
      write.table
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "pdf")
      importFrom("graphics", "hist", "par", "title")
      importFrom("stats", "fisher.test", "qnorm", "quantile", "sd", "t.test")
      importFrom("utils", "data", "str", "write.csv", "write.table")
    to your NAMESPACE file.
    ```

# gt

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/gt
* URL: https://github.com/rstudio/gt
* BugReports: https://github.com/rstudio/gt/issues
* Date/Publication: 2020-05-23 07:00:02 UTC
* Number of recursive dependencies: 91

Run `revdep_details(,"gt")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        help   5.5Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 116 marked UTF-8 strings
    ```

# gwdegree

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/gwdegree
* URL: https://github.com/michaellevy/gwdegree, https://michaellevy.shinyapps.io/gwdegree/
* BugReports: https://github.com/michaellevy/gwdegree/issues
* Date/Publication: 2016-07-09 10:46:45
* Number of recursive dependencies: 75

Run `revdep_details(,"gwdegree")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    simCCCent : <anonymous>: no visible global function definition for
      ‘simulate.formula’
    Undefined global functions or variables:
      simulate.formula
    ```

# HaDeX

<details>

* Version: 1.2.1
* Source code: https://github.com/cran/HaDeX
* Date/Publication: 2020-06-09 11:20:06 UTC
* Number of recursive dependencies: 129

Run `revdep_details(,"HaDeX")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        HaDeX   4.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘gsubfn’ ‘stringr’
      All declared Imports should be used.
    ```

# hchinamap

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/hchinamap
* URL: https://github.com/czxa/hchinamap
* BugReports: https://github.com/czxa/hchinamap/issues
* Date/Publication: 2019-08-23 08:50:02 UTC
* Number of recursive dependencies: 75

Run `revdep_details(,"hchinamap")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        doc           2.7Mb
        htmlwidgets   2.8Mb
    ```

# highcharter

<details>

* Version: 0.7.0
* Source code: https://github.com/cran/highcharter
* URL: http://jkunst.com/highcharter
* BugReports: https://github.com/jbkunst/highcharter/issues
* Date/Publication: 2019-01-15 16:50:03 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"highcharter")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        doc           3.7Mb
        htmlwidgets   4.0Mb
    ```

# HumanTranscriptomeCompendium

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/HumanTranscriptomeCompendium
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 98

Run `revdep_details(,"HumanTranscriptomeCompendium")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘HumanTranscriptomeCompendium-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: htx_query_by_study_accession
    > ### Title: retrieve 'restfulSE' SummarizedExperiment instance for selected
    > ###   studies in htx compendium
    > ### Aliases: htx_query_by_study_accession
    > 
    > ### ** Examples
    > 
    > htx_query_by_study_accession("ERP011411")
    acquiring base restfulSE...
    adding RDS to local cache, future invocations will use local image
    adding rname 'https://s3.amazonaws.com/bcfound-bigrna/rangedHtxGeneSE.rds'
    Loading required package: rhdf5client
    Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
      there is no package called ‘rhdf5client’
    Error in .requirePackage(package) : 
      unable to find required package ‘rhdf5client’
    Calls: htx_query_by_study_accession ... .findInheritedMethods -> getClass -> getClassDef -> .requirePackage
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
       12. S4Vectors::lapply(X, FUN, ...)
       14. base::lapply(as.list(X), match.fun(FUN), ...)
       15. SummarizedExperiment:::FUN(X[[i]], ...)
       16. methods:::dim(a)
       18. methods:::.findInheritedMethods(classes, fdef, mtable)
       19. methods::getClass(cl1, where = where, .Force = TRUE)
       20. methods::getClassDef(Class, where)
       21. methods:::.requirePackage(package)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 6 | SKIPPED: 0 | WARNINGS: 1 | FAILED: 1 ]
      1. Error: query resolution works (@test_htc.R#42) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

# hwordcloud

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/hwordcloud
* URL: https://github.com/czxa/hwordcloud
* BugReports: https://github.com/czxa/hwordcloud/issues
* Date/Publication: 2019-08-07 04:50:02 UTC
* Number of recursive dependencies: 72

Run `revdep_details(,"hwordcloud")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colourpicker’ ‘shiny’ ‘wordcloud2’
      All declared Imports should be used.
    ```

# ideal

<details>

* Version: 1.12.1
* Source code: https://github.com/cran/ideal
* URL: https://github.com/federicomarini/ideal, https://federicomarini.github.io/ideal/
* BugReports: https://github.com/federicomarini/ideal/issues
* Date/Publication: 2020-05-26
* Number of recursive dependencies: 198

Run `revdep_details(,"ideal")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ideal: no visible binding for '<<-' assignment to ‘ideal_env’
    ideal : <anonymous>: no visible binding for global variable ‘airway’
    ideal : <anonymous>: no visible binding for global variable ‘pvalue’
    ideal : <anonymous>: no visible binding for global variable ‘baseMean’
    ideal : <anonymous>: no visible binding for global variable ‘ideal_env’
    Undefined global functions or variables:
      airway baseMean ideal_env pvalue
    ```

# idefix

<details>

* Version: 0.4.3
* Source code: https://github.com/cran/idefix
* URL: https://github.com/traets/idefix
* Date/Publication: 2020-04-07 14:40:05 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"idefix")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘ChoiceModelR’, ‘RSGHB’
    ```

# idem

<details>

* Version: 5.0
* Source code: https://github.com/cran/idem
* Date/Publication: 2020-05-23 23:50:02 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"idem")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstantools’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# immunarch

<details>

* Version: 0.6.5
* Source code: https://github.com/cran/immunarch
* URL: https://immunarch.com/, https://github.com/immunomind/immunarch
* BugReports: https://github.com/immunomind/immunarch/issues
* Date/Publication: 2020-06-14 17:40:06 UTC
* Number of recursive dependencies: 164

Run `revdep_details(,"immunarch")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.3Mb
      sub-directories of 1Mb or more:
        data   5.0Mb
        doc    1.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dbplyr’
      All declared Imports should be used.
    ```

# IMWatson

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/IMWatson
* Date/Publication: 2019-03-29 16:40:03 UTC
* Number of recursive dependencies: 52

Run `revdep_details(,"IMWatson")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘V8’ ‘assertive’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# IncucyteDRC

<details>

* Version: 0.5.4
* Source code: https://github.com/cran/IncucyteDRC
* URL: https://github.com/chapmandu2/IncucyteDRC
* BugReports: https://github.com/chapmandu2/IncucyteDRC/issues
* Date/Publication: 2016-04-23 14:21:03
* Number of recursive dependencies: 119

Run `revdep_details(,"IncucyteDRC")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    ✖ could not find function "row_number"
    ℹ Input `idx` is `row_number()`.
    ℹ The error occured in group 1: sampleid = "PDD00017273", conc = 0.3703704.
    Backtrace:
         █
      1. └─IncucyteDRC::fitDoseResponseCurve(test_idrc_set)
      2.   └─IncucyteDRC::exportDRCDataToDataFrame(idrc_set, include_control)
      3.     └─`%>%`(...)
      4.       ├─base::withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
      5.       └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
      6.         └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
      7.           └─IncucyteDRC:::`_fseq`(`_lhs`)
      8.             └─magrittr::freduce(value, `_function_list`)
      9.               └─function_list[[i]](value)
     10.                 ├─dplyr::mutate(., idx = row_number())
     11.                 └─dplyr:::mutate.data.frame(., idx = row_number())
     12.                   └─dplyr:::mutate_cols(.data, ...)
    <parent: error/simpleError>
    Backtrace:
    █
    Execution halted
    ```

# INDperform

<details>

* Version: 0.2.2
* Source code: https://github.com/cran/INDperform
* URL: https://github.com/saskiaotto/INDperform
* BugReports: https://github.com/SaskiaAOtto/INDperform/issues
* Date/Publication: 2020-01-09 12:30:14 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"INDperform")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘INDperform-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: merge_models
    > ### Title: Merging two model output tibbles.
    > ### Aliases: merge_models
    > 
    > ### ** Examples
    > 
    > # Using some models of the Baltic Sea demo data:
    > # Merging GAM and GAMM tibbles
    > test_ids <- 47:50 # choose subset
    > gam_tbl <- model_gam_ex[test_ids,]
    > gamm_tbl <- model_gamm(ind_init_ex[test_ids,], filter= gam_tbl$tac)
    Error in model_gamm(ind_init_ex[test_ids, ], filter = gam_tbl$tac) : 
      No IND~pressure GAMM could be fitted! Check if you chose the correct error distribution (default is gaussian()).
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
       4. tibble:::tbl_subassign_row(xj, i, value, value_arg)
       5. base::tryCatch(...)
       6. base:::tryCatchList(expr, classes, parentenv, handlers)
       7. base:::tryCatchOne(expr, names, parentenv, handlers[[1L]])
       8. value[[3L]](cond)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 475 | SKIPPED: 0 | WARNINGS: 10 | FAILED: 4 ]
      1. Error: (unknown) (@test_calc_deriv.R#6) 
      2. Error: (unknown) (@test_cond_boot.R#112) 
      3. Error: (unknown) (@test_model_gamm.R#4) 
      4. Error: (unknown) (@test_scoring.R#15) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   3.6Mb
        help   1.6Mb
    ```

# inlabru

<details>

* Version: 2.1.13
* Source code: https://github.com/cran/inlabru
* URL: http://www.inlabru.org,
* BugReports: https://github.com/fbachl/inlabru/issues
* Date/Publication: 2020-02-16 22:10:09 UTC
* Number of recursive dependencies: 109

Run `revdep_details(,"inlabru")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘INLA’
    ```

# INSPEcT

<details>

* Version: 1.18.0
* Source code: https://github.com/cran/INSPEcT
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 136

Run `revdep_details(,"INSPEcT")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 10.9Mb
      sub-directories of 1Mb or more:
        R      2.1Mb
        data   2.8Mb
        doc    3.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    output_pars: no visible global function definition for ‘plot.new’
    Undefined global functions or variables:
      plot.new
    Consider adding
      importFrom("graphics", "plot.new")
    to your NAMESPACE file.
    ```

# interactiveDisplay

<details>

* Version: 1.26.0
* Source code: https://github.com/cran/interactiveDisplay
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 181

Run `revdep_details(,"interactiveDisplay")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: ‘.interactiveDisplayBase’
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘rstudio’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for global variable ‘Var1’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘Var2’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘value’
    Undefined global functions or variables:
      AnnotationTrack GO.db GRanges GRangesList GenomeAxisTrack IRanges
      IdeogramTrack MRcounts Var1 Var2 as.dendrogram assays cim
      colorRampPalette coord_flip cutree dendrapply dev.off dist
      elementMetadata experimentData exprs fData hclust installed.packages
      is.leaf layout_circle legend libSize mcols pData plotFeature plotOrd
      plotTracks png rainbow ranges rowRanges seqlengths seqlengths<-
      seqlevels<- seqnames ucscGenomes value
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png",
                 "rainbow")
      importFrom("graphics", "legend")
      importFrom("stats", "as.dendrogram", "cutree", "dendrapply", "dist",
                 "hclust", "is.leaf")
      importFrom("utils", "installed.packages")
    to your NAMESPACE file.
    ```

# interAdapt

<details>

* Version: 0.1
* Source code: https://github.com/cran/interAdapt
* Date/Publication: 2014-08-03 08:03:00
* Number of recursive dependencies: 42

Run `revdep_details(,"interAdapt")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    compute_design_performance : expected_duration_plot: no visible global
      function definition for ‘legend’
    compute_design_performance : boundary_adapt_plot: no visible global
      function definition for ‘matplot’
    compute_design_performance : boundary_adapt_plot: no visible global
      function definition for ‘legend’
    compute_design_performance : boundary_standard_H0C_plot: no visible
      global function definition for ‘matplot’
    compute_design_performance : boundary_standard_H0C_plot: no visible
      global function definition for ‘legend’
    compute_design_performance : boundary_standard_H01_plot: no visible
      global function definition for ‘matplot’
    compute_design_performance : boundary_standard_H01_plot: no visible
      global function definition for ‘legend’
    Undefined global functions or variables:
      legend lines matplot read.csv rnorm
    Consider adding
      importFrom("graphics", "legend", "lines", "matplot")
      importFrom("stats", "rnorm")
      importFrom("utils", "read.csv")
    to your NAMESPACE file.
    ```

# IOHanalyzer

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/IOHanalyzer
* URL: http://iohprofiler.liacs.nl, https://github.com/IOHprofiler/IOHAnalyzer
* BugReports: https://github.com/IOHprofiler/IOHAnalyzer/issues
* Date/Publication: 2020-01-10 15:30:03 UTC
* Number of recursive dependencies: 103

Run `revdep_details(,"IOHanalyzer")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘bsplus’ ‘colourpicker’ ‘kableExtra’ ‘markdown’ ‘reshape2’
      ‘shinydashboard’ ‘shinyjs’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 75 marked UTF-8 strings
    ```

# iSEE

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/iSEE
* URL: https://github.com/iSEE/iSEE
* BugReports: https://github.com/iSEE/iSEE/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 154

Run `revdep_details(,"iSEE")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    +   return(qc_colors)
    + }
    > 
    > ecm <- ExperimentColorMap(
    +     assays = list(
    +         tophat_counts = count_colors
    +     ),
    +     colData = list(
    +         passes_qc_checks_s = qc_color_fun
    +     )
    + )
    > 
    > # Example SingleCellExperiment ----
    > 
    > library(scRNAseq)
    > sce <- ReprocessedAllenData(assays="tophat_counts")
    Error: Corrupt Cache: sqlite file
      See vignette section on corrupt cache
      cache: /Users/barret/Library/Caches/ExperimentHub
      filename: experimenthub.sqlite3
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      The following objects are masked from 'package:base':
      
          aperm, apply, rowsum
      
      Loading required package: SingleCellExperiment
      > 
      > test_check("iSEE")
      Loading required package: scRNAseq
      Loading required package: scater
      Loading required package: ggplot2
      Error: Corrupt Cache: sqlite file
        See vignette section on corrupt cache
        cache: /Users/barret/Library/Caches/ExperimentHub
        filename: experimenthub.sqlite3
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 26.3Mb
      sub-directories of 1Mb or more:
        doc  24.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::selectSome’
      See the note in ?`:::` about the use of this operator.
    ```

# iSEEu

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/iSEEu
* URL: https://github.com/iSEE/iSEEu
* BugReports: https://github.com/iSEE/iSEEu/issues
* Date/Publication: 2020-05-06
* Number of recursive dependencies: 171

Run `revdep_details(,"iSEEu")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ###   .panelColor,DifferentialStatisticsTable-method
    > ###   .defineDataInterface,DifferentialStatisticsTable-method
    > ###   .hideInterface,DifferentialStatisticsTable-method
    > ###   .generateTable,DifferentialStatisticsTable-method
    > ###   .createObservers,DifferentialStatisticsTable-method
    > ###   .cacheCommonInfo,DifferentialStatisticsTable-method
    > ###   .refineParameters,DifferentialStatisticsTable-method
    > ###   .multiSelectionInvalidated,DifferentialStatisticsTable-method
    > 
    > ### ** Examples
    > 
    > library(scRNAseq)
    > library(scater)
    Loading required package: ggplot2
    > 
    > sce <- ReprocessedAllenData(assays="tophat_counts")
    Error: Corrupt Cache: sqlite file
      See vignette section on corrupt cache
      cache: /Users/barret/Library/Caches/ExperimentHub
      filename: experimenthub.sqlite3
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
          colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
      
      The following objects are masked from 'package:base':
      
          aperm, apply, rowsum
      
      Loading required package: SingleCellExperiment
      > 
      > test_check("iSEEu")
      Error: Corrupt Cache: sqlite file
        See vignette section on corrupt cache
        cache: /Users/barret/Library/Caches/ExperimentHub
        filename: experimenthub.sqlite3
      Execution halted
    ```

# iSTATS

<details>

* Version: 1.4
* Source code: https://github.com/cran/iSTATS
* Date/Publication: 2020-06-04 15:50:03 UTC
* Number of recursive dependencies: 80

Run `revdep_details(,"iSTATS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        app    4.3Mb
        data   1.6Mb
    ```

# ivygapSE

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/ivygapSE
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 151

Run `revdep_details(,"ivygapSE")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 16.7Mb
      sub-directories of 1Mb or more:
        data  13.9Mb
        doc    2.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    exprByType: no visible binding for global variable ‘ivySE’
    ivyGlimpse: no visible binding for global variable ‘ivySE’
    ivyGlimpse : server: no visible binding for global variable ‘ivySE’
    Undefined global functions or variables:
      ivySE
    ```

# jpmesh

<details>

* Version: 1.2.1
* Source code: https://github.com/cran/jpmesh
* URL: https://uribo.github.io/jpmesh
* BugReports: https://github.com/uribo/jpmesh/issues
* Date/Publication: 2020-05-06 12:20:03 UTC
* Number of recursive dependencies: 126

Run `revdep_details(,"jpmesh")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 176 marked UTF-8 strings
    ```

# jpndistrict

<details>

* Version: 0.3.7
* Source code: https://github.com/cran/jpndistrict
* URL: https://uribo.github.io/jpndistrict
* BugReports: https://github.com/uribo/jpndistrict/issues
* Date/Publication: 2020-06-12 07:30:03 UTC
* Number of recursive dependencies: 101

Run `revdep_details(,"jpndistrict")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 188 marked UTF-8 strings
    ```

# jsmodule

<details>

* Version: 1.0.8
* Source code: https://github.com/cran/jsmodule
* URL: https://github.com/jinseob2kim/jsmodule
* BugReports: https://github.com/jinseob2kim/jsmodule/issues
* Date/Publication: 2020-06-04 17:00:02 UTC
* Number of recursive dependencies: 188

Run `revdep_details(,"jsmodule")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Cairo’ ‘survC1’
      All declared Imports should be used.
    ```

# KNNShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/KNNShiny
* Date/Publication: 2019-05-09 16:00:07 UTC
* Number of recursive dependencies: 105

Run `revdep_details(,"KNNShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘FNN’ ‘caret’ ‘datasets’ ‘dplyr’ ‘e1071’ ‘psycho’ ‘rhandsontable’
      ‘shiny’
      All declared Imports should be used.
    ```

# kokudosuuchi

<details>

* Version: 0.4.2
* Source code: https://github.com/cran/kokudosuuchi
* URL: https://github.com/yutannihilation/kokudosuuchi
* BugReports: https://github.com/yutannihilation/kokudosuuchi/issues
* Date/Publication: 2018-01-26 04:21:18 UTC
* Number of recursive dependencies: 66

Run `revdep_details(,"kokudosuuchi")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 52458 marked UTF-8 strings
    ```

# koRpus

<details>

* Version: 0.11-5
* Source code: https://github.com/cran/koRpus
* URL: https://reaktanz.de/?c=hacking&s=koRpus
* BugReports: https://github.com/unDocUMeantIt/koRpus/issues
* Date/Publication: 2018-10-28 07:50:02 UTC
* Number of recursive dependencies: 60

Run `revdep_details(,"koRpus")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'koRpus.lang.de', 'koRpus.lang.es', 'koRpus.lang.fr',
      'koRpus.lang.it', 'koRpus.lang.ru'
    
    Package which this enhances but not available for checking: ‘rkward’
    ```

# leaflet.extras2

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/leaflet.extras2
* URL: https://trafficonese.github.io/leaflet.extras2, https://github.com/trafficonese/leaflet.extras2
* BugReports: https://github.com/trafficonese/leaflet.extras2/issues
* Date/Publication: 2020-05-18 15:10:06 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"leaflet.extras2")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmlwidgets’
      All declared Imports should be used.
    ```

# learnstats

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/learnstats
* Date/Publication: 2015-06-25 07:20:19
* Number of recursive dependencies: 128

Run `revdep_details(,"learnstats")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    qqsim: no visible global function definition for ‘hist’
    qqsim: no visible global function definition for ‘mtext’
    qqsim: no visible global function definition for ‘jpeg’
    qqsim: no visible global function definition for ‘dev.off’
    timeseressim: no visible global function definition for ‘rnorm’
    timeseressim: no visible global function definition for ‘par’
    timeseressim: no visible global function definition for ‘mtext’
    timeseressim: no visible global function definition for ‘jpeg’
    timeseressim: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      axis dchisq dev.off dnorm dt hist jpeg legend mtext par pchisq pf
      pnorm polygon pt qchisq qf qqline qqnorm qt rchisq rnorm rt runif
      segments title
    Consider adding
      importFrom("grDevices", "dev.off", "jpeg")
      importFrom("graphics", "axis", "hist", "legend", "mtext", "par",
                 "polygon", "segments", "title")
      importFrom("stats", "dchisq", "dnorm", "dt", "pchisq", "pf", "pnorm",
                 "pt", "qchisq", "qf", "qqline", "qqnorm", "qt", "rchisq",
                 "rnorm", "rt", "runif")
    to your NAMESPACE file.
    ```

# levi

<details>

* Version: 1.6.0
* Source code: https://github.com/cran/levi
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 84

Run `revdep_details(,"levi")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    LEVIui: no visible global function definition for ‘is’
    levi_function: no visible binding for global variable ‘x’
    Undefined global functions or variables:
      is x
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# likert

<details>

* Version: 1.3.5
* Source code: https://github.com/cran/likert
* URL: http://jason.bryer.org/likert, http://github.com/jbryer/likert
* BugReports: https://github.com/jbryer/likert/issues
* Date/Publication: 2016-12-31 01:37:00
* Number of recursive dependencies: 102

Run `revdep_details(,"likert")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 7 marked UTF-8 strings
    ```

# live

<details>

* Version: 1.5.13
* Source code: https://github.com/cran/live
* URL: https://github.com/ModelOriented/live
* BugReports: https://github.com/ModelOriented/live/issues
* Date/Publication: 2020-01-15 06:30:17 UTC
* Number of recursive dependencies: 111

Run `revdep_details(,"live")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘e1071’
      All declared Imports should be used.
    ```

# lmviz

<details>

* Version: 0.1.2
* Source code: https://github.com/cran/lmviz
* Date/Publication: 2019-05-15 14:30:03 UTC
* Number of recursive dependencies: 25

Run `revdep_details(,"lmviz")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyjs’
      All declared Imports should be used.
    ```

# mapdeck

<details>

* Version: 0.3.3
* Source code: https://github.com/cran/mapdeck
* URL: https://symbolixau.github.io/mapdeck/articles/mapdeck.html
* BugReports: https://github.com/SymbolixAU/mapdeck/issues
* Date/Publication: 2020-05-14 19:30:08 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"mapdeck")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.1Mb
      sub-directories of 1Mb or more:
        data          1.3Mb
        doc           1.6Mb
        htmlwidgets   1.9Mb
        libs          2.9Mb
    ```

# mapedit

<details>

* Version: 0.6.0
* Source code: https://github.com/cran/mapedit
* URL: https://github.com/r-spatial/mapedit
* BugReports: https://github.com/r-spatial/mapedit/issues
* Date/Publication: 2020-02-02 17:20:02 UTC
* Number of recursive dependencies: 107

Run `revdep_details(,"mapedit")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘geojsonio’
    ```

# MDSPCAShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/MDSPCAShiny
* Date/Publication: 2019-05-24 08:20:20 UTC
* Number of recursive dependencies: 49

Run `revdep_details(,"MDSPCAShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘datasets’ ‘dplyr’ ‘psych’ ‘shiny’
      All declared Imports should be used.
    ```

# memapp

<details>

* Version: 2.13
* Source code: https://github.com/cran/memapp
* URL: https://github.com/lozalojo/memapp
* BugReports: https://github.com/lozalojo/memapp/issues
* Date/Publication: 2019-06-04 06:40:03 UTC
* Number of recursive dependencies: 112

Run `revdep_details(,"memapp")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘RColorBrewer’ ‘RODBC’ ‘dplyr’ ‘foreign’ ‘formattable’ ‘ggplot2’
      ‘haven’ ‘mem’ ‘openxlsx’ ‘plotly’ ‘readxl’ ‘shinyBS’ ‘shinyWidgets’
      ‘shinydashboard’ ‘shinyjs’ ‘shinythemes’ ‘stringi’ ‘stringr’ ‘tidyr’
      All declared Imports should be used.
    ```

# memery

<details>

* Version: 0.5.2
* Source code: https://github.com/cran/memery
* URL: https://github.com/leonawicz/memery
* BugReports: https://github.com/leonawicz/memery/issues
* Date/Publication: 2019-04-19 20:51:00 UTC
* Number of recursive dependencies: 97

Run `revdep_details(,"memery")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colourpicker’ ‘shinyBS’ ‘shinycssloaders’
      All declared Imports should be used.
    ```

# meta4diag

<details>

* Version: 2.0.8
* Source code: https://github.com/cran/meta4diag
* Date/Publication: 2018-07-23 12:00:02 UTC
* Number of recursive dependencies: 24

Run `revdep_details(,"meta4diag")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

# MetCirc

<details>

* Version: 1.18.0
* Source code: https://github.com/cran/MetCirc
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 102

Run `revdep_details(,"MetCirc")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘MSnbase:::bin_Spectra’ ‘circlize:::get.sector.data’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘recordPlotFill_degreeFeatures’ ‘recordPlotHighlight’ ‘replayPlotAdd’
      ‘replayPlotOrder’ ‘select’ ‘spectraCond’ ‘typeMatch_link0’
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotSpectra: no visible binding for global variable ‘mz’
    plotSpectra: no visible binding for global variable ‘int’
    Undefined global functions or variables:
      int mz
    ```

# MethylAid

<details>

* Version: 1.22.0
* Source code: https://github.com/cran/MethylAid
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 164

Run `revdep_details(,"MethylAid")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘MethylAid-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: summarize
    > ### Title: summarization of Illumina Human DNA Methylation array data
    > ### Aliases: summarize
    > 
    > ### ** Examples
    > 
    > library(minfiData)
    Error in library(minfiData) : there is no package called ‘minfiData’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      ERROR in test_summarize: Error in library(minfiData) : there is no package called 'minfiData'
      ERROR in test_visualize: Error in library(minfiData) : there is no package called 'minfiData'
      
      Test files with failing tests
      
         test_summarize.R 
           test_summarize 
      
         test_visualize.R 
           test_visualize 
      
      
      Error in BiocGenerics:::testPackage("MethylAid") : 
        unit tests failed for package MethylAid
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'MethylAidData', 'minfiData'
    ```

# methylGSA

<details>

* Version: 1.6.1
* Source code: https://github.com/cran/methylGSA
* URL: https://github.com/reese3928/methylGSA
* BugReports: https://github.com/reese3928/methylGSA/issues
* Date/Publication: 2020-05-06
* Number of recursive dependencies: 198

Run `revdep_details(,"methylGSA")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    getAnnot: no visible global function definition for ‘getAnnotation’
    getAnnot: no visible binding for global variable
      ‘IlluminaHumanMethylation450kanno.ilmn12.hg19’
    getAnnot: no visible binding for global variable
      ‘IlluminaHumanMethylationEPICanno.ilm10b4.hg19’
    Undefined global functions or variables:
      IlluminaHumanMethylation450kanno.ilmn12.hg19
      IlluminaHumanMethylationEPICanno.ilm10b4.hg19 getAnnotation
    ```

# mgcViz

<details>

* Version: 0.1.6
* Source code: https://github.com/cran/mgcViz
* URL: https://github.com/mfasiolo/mgcViz
* BugReports: https://github.com/mfasiolo/mgcViz/issues
* Date/Publication: 2020-03-04 15:10:02 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"mgcViz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc   4.4Mb
    ```

# microhaplot

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/microhaplot
* URL: https://github.com/ngthomas/microhaplot
* BugReports: https://github.com/ngthomas/microhaplot/issues
* Date/Publication: 2019-10-03 07:40:05 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"microhaplot")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘ggiraph’ ‘ggplot2’ ‘grid’ ‘gtools’ ‘scales’ ‘shinyBS’
      ‘shinyWidgets’ ‘tidyr’
      All declared Imports should be used.
    ```

# miniMeta

<details>

* Version: 0.2
* Source code: https://github.com/cran/miniMeta
* Date/Publication: 2020-03-01 13:10:02 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"miniMeta")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘markdown’ ‘metafor’
      All declared Imports should be used.
    ```

# miRcomp

<details>

* Version: 1.18.0
* Source code: https://github.com/cran/miRcomp
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 37

Run `revdep_details(,"miRcomp")` for more info

</details>

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                      old_size new_size compress
      lifetech.rda       350Kb    276Kb       xz
      qpcRb4.rda         290Kb    243Kb       xz
      qpcRb5.rda         288Kb    242Kb       xz
      qpcRdefault.rda    290Kb    242Kb       xz
      qpcRl5.rda         287Kb    239Kb       xz
      qpcRlinexp.rda     235Kb    199Kb       xz
    ```

# MLDAShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/MLDAShiny
* Date/Publication: 2019-04-25 12:20:02 UTC
* Number of recursive dependencies: 116

Run `revdep_details(,"MLDAShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DiscriMiner’ ‘MASS’ ‘caret’ ‘datasets’ ‘dplyr’ ‘e1071’ ‘klaR’
      ‘rhandsontable’ ‘shiny’
      All declared Imports should be used.
    ```

# MLInterfaces

<details>

* Version: 1.68.0
* Source code: https://github.com/cran/MLInterfaces
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 179

Run `revdep_details(,"MLInterfaces")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘rda’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘fpc’ ‘gbm’ ‘ggvis’ ‘hwriter’ ‘mlbench’ ‘pls’ ‘shiny’ ‘stats4’
      ‘threejs’ ‘tools’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘Biobase’ ‘BiocGenerics’ ‘Rcpp’ ‘annotate’ ‘cluster’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported objects imported by ':::' calls:
      ‘MASS:::predict.lda’ ‘ada:::predict.ada’ ‘e1071:::predict.svm’
      ‘rpart:::predict.rpart’ ‘stats:::plot.hclust’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking S3 generic/method consistency ... NOTE
    ```
    Found the following apparent S3 methods exported but not registered:
      predict.RAB predict.classifierOutput predict.dlda2 predict.gbm2
      predict.knn.cv2 predict.knn2 predict.lvq predict.rdaML
      predict.rdacvML print.rdaML print.rdacvML
    See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
    manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plspinHcube : <anonymous>: no visible global function definition for
      ‘points3d’
    predict.classifierOutput: no visible global function definition for
      ‘exprs’
    predict.gbm2: no visible global function definition for ‘predict.gbm’
    rab: no visible global function definition for ‘rpart’
    getGrid,ExpressionSet: no visible global function definition for
      ‘exprs’
    planarPlot,classifierOutput-ExpressionSet-character: no visible global
      function definition for ‘exprs’
    predScore,classifierOutput: no visible global function definition for
      ‘rowMax’
    Undefined global functions or variables:
      %>% := abstract actionButton add_tooltip annotation bind_shiny br
      brewer.pal clusterboot column div em exprs featureNames fluidPage
      fluidRow gbm ggvis ggvisOutput h1 hclustCBI htmlOutput hwrite isolate
      key layer_points lookUp mainPanel mlbench.hypercube numericInput
      observe p pData plotOutput plotcp points3d predict.gbm reactive
      renderPlot renderScatterplotThree renderTable renderText rowMax rpart
      scatterplot3js scatterplotThreeOutput selectInput shinyApp
      sidebarLayout sidebarPanel stopApp strong tableOutput textOutput
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: plspinHcube.Rd:17-19: Dropping empty section \details
    ```

# mlr3shiny

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/mlr3shiny
* Date/Publication: 2020-03-31 10:30:02 UTC
* Number of recursive dependencies: 121

Run `revdep_details(,"mlr3shiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘data.table’ ‘e1071’ ‘mlr3’ ‘mlr3learners’ ‘mlr3measures’ ‘plyr’
      ‘purrr’ ‘ranger’ ‘readxl’ ‘shinyWidgets’ ‘shinyalert’
      ‘shinydashboard’ ‘shinyjs’ ‘shinythemes’ ‘stats’ ‘stringr’
      All declared Imports should be used.
    ```

# MLRShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/MLRShiny
* Date/Publication: 2019-04-22 19:20:03 UTC
* Number of recursive dependencies: 101

Run `revdep_details(,"MLRShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘QuantPsyc’ ‘corrgram’ ‘dplyr’ ‘forecast’ ‘psych’ ‘shinyAce’
      All declared Imports should be used.
    ```

# MLRShiny2

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/MLRShiny2
* Date/Publication: 2019-10-02 08:40:02 UTC
* Number of recursive dependencies: 101

Run `revdep_details(,"MLRShiny2")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘QuantPsyc’ ‘corrgram’ ‘dplyr’ ‘forecast’ ‘psych’ ‘shinyAce’
      All declared Imports should be used.
    ```

# MMDiff2

<details>

* Version: 1.16.0
* Source code: https://github.com/cran/MMDiff2
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 103

Run `revdep_details(,"MMDiff2")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BSgenome.Mmusculus.UCSC.mm9’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    DBAmmd: no visible global function definition for ‘new’
    getReads: no visible global function definition for ‘IRangesList’
    Undefined global functions or variables:
      IRangesList new
    Consider adding
      importFrom("methods", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# MNLR

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/MNLR
* Date/Publication: 2019-03-23 10:06:11 UTC
* Number of recursive dependencies: 96

Run `revdep_details(,"MNLR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘caret’ ‘datasets’ ‘e1071’ ‘nnet’ ‘shiny’ ‘stats’
      All declared Imports should be used.
    ```

# MSEtool

<details>

* Version: 1.6.0
* Source code: https://github.com/cran/MSEtool
* URL: http://www.datalimitedtoolkit.org
* BugReports: https://github.com/tcarruth/MSEtool/issues
* Date/Publication: 2020-05-05 04:30:02 UTC
* Number of recursive dependencies: 105

Run `revdep_details(,"MSEtool")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 19.5Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        doc    1.2Mb
        libs  15.9Mb
    ```

# MSGFgui

<details>

* Version: 1.22.0
* Source code: https://github.com/cran/MSGFgui
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 70

Run `revdep_details(,"MSGFgui")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘MSGFplus’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MSnbase

<details>

* Version: 2.14.2
* Source code: https://github.com/cran/MSnbase
* URL: https://lgatto.github.io/MSnbase
* BugReports: https://github.com/lgatto/MSnbase/issues
* Date/Publication: 2020-05-15
* Number of recursive dependencies: 222

Run `revdep_details(,"MSnbase")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.2Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   1.9Mb
        doc    7.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Laurent Gatto <laurent.gatto@uclouvain.be> [aut, cre] (<https://orcid.org/0000-0002-1520-2268>)
      Johannes Rainer <Johannes.Rainer@eurac.edu> [aut, cre] (<https://orcid.org/0000-0002-6977-7147>)
      Sebastian Gibb <mail@sebastiangibb.de> [aut, cre]
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': ‘S4Vectors:::makePrettyMatrixForCompactPrinting’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘Biobase:::.showAnnotatedDataFrame’ ‘MALDIquant:::.estimateNoise’
      ‘MALDIquant:::.localMaxima’ ‘MALDIquant:::.movingAverage’
      ‘MALDIquant:::.savitzkyGolay’
      ‘S4Vectors:::makeClassinfoRowForCompactPrinting’
      ‘mzR:::.hasChromatograms’ ‘mzR:::.hasSpectra’
      See the note in ?`:::` about the use of this operator.
    ```

# mstrio

<details>

* Version: 11.2.1
* Source code: https://github.com/cran/mstrio
* Date/Publication: 2020-03-26 08:00:02 UTC
* Number of recursive dependencies: 59

Run `revdep_details(,"mstrio")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘miniUI’
      All declared Imports should be used.
    ```

# MuChPoint

<details>

* Version: 0.6.1
* Source code: https://github.com/cran/MuChPoint
* Date/Publication: 2018-06-24 13:09:46 UTC
* Number of recursive dependencies: 22

Run `revdep_details(,"MuChPoint")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘capushe’ ‘shiny’
      All declared Imports should be used.
    ```

# nbc4va

<details>

* Version: 1.1
* Source code: https://github.com/cran/nbc4va
* Date/Publication: 2018-08-05 16:40:03 UTC
* Number of recursive dependencies: 47

Run `revdep_details(,"nbc4va")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ── 4. Error: nbc() can handle incorrect column data types in the correct positio
      The train data IDs are not unique (duplicates found:)
      Backtrace:
       1. testthat::expect_warning(...)
       6. nbc4va:::internalCheckNBC(trainTypes, testTypes)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 68 | SKIPPED: 0 | WARNINGS: 2 | FAILED: 4 ]
      1. Failure: (unknown) (@test_nbc4va.R#29) 
      2. Failure: (unknown) (@test_nbc4va.R#32) 
      3. Failure: nbc() sets the out$test.known to correctly indicate whether or not the test causes are known (@test_nbc4va.R#46) 
      4. Error: nbc() can handle incorrect column data types in the correct positions (@test_nbc4va.R#67) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘openVA’
    ```

# NBShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/NBShiny
* Date/Publication: 2019-05-06 08:30:03 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"NBShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘caret’ ‘datasets’ ‘dplyr’ ‘e1071’ ‘rhandsontable’ ‘shiny’
      All declared Imports should be used.
    ```

# NBShiny2

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/NBShiny2
* Date/Publication: 2019-10-25 08:00:09 UTC
* Number of recursive dependencies: 98

Run `revdep_details(,"NBShiny2")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘caret’ ‘datasets’ ‘dplyr’ ‘e1071’ ‘rhandsontable’ ‘shiny’
      All declared Imports should be used.
    ```

# NDP

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/NDP
* Date/Publication: 2019-03-30 15:00:02 UTC
* Number of recursive dependencies: 29

Run `revdep_details(,"NDP")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# neo4r

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/neo4r
* URL: https://github.com/neo4j-rstats/neo4r
* BugReports: https://github.com/neo4j-rstats/neo4r/issues
* Date/Publication: 2019-02-15 22:20:03 UTC
* Number of recursive dependencies: 46

Run `revdep_details(,"neo4r")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘igraph’ ‘rlang’ ‘tidyselect’
      All declared Imports should be used.
    ```

# nprcgenekeepr

<details>

* Version: 1.0.3
* Source code: https://github.com/cran/nprcgenekeepr
* URL: https://rmsharp.github.io/nprcgenekeepr/, https://github.com/rmsharp/nprcgenekeepr
* BugReports: https://github.com/rmsharp/nprcgenekeepr/issues
* Date/Publication: 2020-06-02 12:40:03 UTC
* Number of recursive dependencies: 114

Run `revdep_details(,"nprcgenekeepr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyBS’
      All declared Imports should be used.
    ```

# oceanis

<details>

* Version: 1.7.3
* Source code: https://github.com/cran/oceanis
* URL: https://github.com/insee-psar-at/oceanis-package/
* BugReports: https://github.com/insee-psar-at/oceanis-package/issues
* Date/Publication: 2020-06-02 10:20:02 UTC
* Number of recursive dependencies: 128

Run `revdep_details(,"oceanis")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        extdata   1.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘units’
      All declared Imports should be used.
    ```

# omicplotR

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/omicplotR
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 73

Run `revdep_details(,"omicplotR")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .github
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘ALDEx2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# oneSENSE

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/oneSENSE
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 103

Run `revdep_details(,"oneSENSE")` for more info

</details>

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'export' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

# ontoProc

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/ontoProc
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 125

Run `revdep_details(,"ontoProc")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Name: common_classes
    > ### Title: list and count samples with common ontological annotation in two
    > ###   SEs
    > ### Aliases: common_classes
    > 
    > ### ** Examples
    > 
    > if (requireNamespace("SingleR")) {
    +   imm = SingleR::ImmGenData()
    +   if ("label.ont" %in% names(colData(imm))) {
    +     cl = getCellOnto()
    +     blu = SingleR::BlueprintEncodeData()
    +     common_classes( cl, imm, blu )
    +     }
    +   }
    Loading required namespace: SingleR
    Error: Corrupt Cache: sqlite file
      See vignette section on corrupt cache
      cache: /Users/barret/Library/Caches/ExperimentHub
      filename: experimenthub.sqlite3
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 39.6Mb
      sub-directories of 1Mb or more:
        app       1.9Mb
        data      2.0Mb
        doc       1.8Mb
        ontoRda  33.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    CLfeat : prupdate: no visible binding for global variable ‘PRID’
    CLfeat : prupdate: no visible binding for global variable ‘SYMBOL’
    bind_formal_tags: no visible global function definition for ‘colData’
    bind_formal_tags: no visible global function definition for ‘colData<-’
    common_classes: no visible global function definition for ‘colData’
    connect_classes: no visible global function definition for ‘colData’
    ctmarks: no visible binding for '<<-' assignment to ‘pr’
    ctmarks: no visible binding for '<<-' assignment to ‘go’
    ctmarks : server: no visible binding for global variable ‘text’
    cyclicSigset: no visible binding for global variable ‘sigels’
    ldfToTerm: no visible binding for global variable ‘PROSYM’
    subset_descendants: no visible global function definition for ‘colData’
    sym2CellOnto: no visible binding for global variable ‘SYMBOL’
    Undefined global functions or variables:
      PRID PROSYM SYMBOL colData colData<- sigels text
    Consider adding
      importFrom("graphics", "text")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 162 marked UTF-8 strings
    ```

# oolong

<details>

* Version: 0.3.4
* Source code: https://github.com/cran/oolong
* URL: https://github.com/chainsawriot/oolong
* BugReports: https://github.com/chainsawriot/oolong/issues
* Date/Publication: 2020-03-21 17:40:06 UTC
* Number of recursive dependencies: 130

Run `revdep_details(,"oolong")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘digest’ ‘dplyr’ ‘miniUI’ ‘text2vec’
      All declared Imports should be used.
    ```

# openmetrics

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/openmetrics
* URL: https://github.com/atheriel/openmetrics
* BugReports: https://github.com/atheriel/openmetrics/issues
* Date/Publication: 2020-04-23 17:50:02 UTC
* Number of recursive dependencies: 39

Run `revdep_details(,"openmetrics")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# openPrimeRui

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/openPrimeRui
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 135

Run `revdep_details(,"openPrimeRui")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘openPrimeR’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# oppr

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/oppr
* URL: https://prioritizr.github.io/oppr, https://github.com/prioritizr/oppr
* BugReports: https://github.com/prioritizr/oppr/issues
* Date/Publication: 2020-05-20 10:10:16 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"oppr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'gurobi', 'Rsymphony'
    ```

# pairsD3

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/pairsD3
* URL: http://github.com/garthtarr/pairsD3/
* Date/Publication: 2015-04-28 06:58:01
* Number of recursive dependencies: 28

Run `revdep_details(,"pairsD3")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    pairsD3: no visible global function definition for ‘gray.colors’
    Undefined global functions or variables:
      gray.colors
    Consider adding
      importFrom("grDevices", "gray.colors")
    to your NAMESPACE file.
    ```

# PanVizGenerator

<details>

* Version: 1.16.0
* Source code: https://github.com/cran/PanVizGenerator
* URL: https://github.com/thomasp85/PanVizGenerator
* BugReports: https://github.com/thomasp85/PanVizGenerator/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 100

Run `revdep_details(,"PanVizGenerator")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# PathoStat

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/PathoStat
* URL: https://github.com/mani2012/PathoStat
* BugReports: https://github.com/mani2012/PathoStat/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 184

Run `revdep_details(,"PathoStat")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ComplexHeatmap’ ‘RColorBrewer’
      All declared Imports should be used.
    ```

# pcaExplorer

<details>

* Version: 2.14.2
* Source code: https://github.com/cran/pcaExplorer
* URL: https://github.com/federicomarini/pcaExplorer, https://federicomarini.github.io/pcaExplorer/
* BugReports: https://github.com/federicomarini/pcaExplorer/issues
* Date/Publication: 2020-06-06
* Number of recursive dependencies: 177

Run `revdep_details(,"pcaExplorer")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 21.3Mb
      sub-directories of 1Mb or more:
        doc  20.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    pcaExplorer: no visible binding for '<<-' assignment to
      ‘pcaexplorer_env’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘airway’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘pcaexplorer_env’
    Undefined global functions or variables:
      airway pcaexplorer_env
    ```

# pcFactorStan

<details>

* Version: 1.5.1
* Source code: https://github.com/cran/pcFactorStan
* URL: https://github.com/jpritikin/pcFactorStan
* BugReports: https://github.com/jpritikin/pcFactorStan/issues
* Date/Publication: 2020-04-25 17:40:02 UTC
* Number of recursive dependencies: 134

Run `revdep_details(,"pcFactorStan")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 13.2Mb
      sub-directories of 1Mb or more:
        doc    2.9Mb
        libs   9.7Mb
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# pepStat

<details>

* Version: 1.22.0
* Source code: https://github.com/cran/pepStat
* URL: https://github.com/RGLab/pepStat
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 149

Run `revdep_details(,"pepStat")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘pepStat’ for: ‘end’, ‘start’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    coerce,peptideSet-ExpressionSet: no visible global function definition
      for ‘annotation’
    end,peptideSet: no visible global function definition for ‘end’
    position,peptideSet: no visible global function definition for ‘start’
    position,peptideSet: no visible global function definition for ‘end’
    start,peptideSet: no visible global function definition for ‘start’
    write.pSet,peptideSet: no visible global function definition for
      ‘start’
    write.pSet,peptideSet: no visible global function definition for ‘end’
    write.pSet,peptideSet: no visible global function definition for
      ‘write.csv’
    Undefined global functions or variables:
      annotation dev.flush dev.hold dev.interactive devAskNewPage end
      lm.fit lm.wfit mcols mcols<- median read.csv sd start write.csv
    Consider adding
      importFrom("grDevices", "dev.flush", "dev.hold", "dev.interactive",
                 "devAskNewPage")
      importFrom("stats", "end", "lm.fit", "lm.wfit", "median", "sd",
                 "start")
      importFrom("utils", "read.csv", "write.csv")
    to your NAMESPACE file.
    ```

# phenocamr

<details>

* Version: 1.1.4
* Source code: https://github.com/cran/phenocamr
* Date/Publication: 2020-03-22 17:40:05 UTC
* Number of recursive dependencies: 106

Run `revdep_details(,"phenocamr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘jsonlite’ ‘memoise’
      All declared Imports should be used.
    ```

# piano

<details>

* Version: 2.4.0
* Source code: https://github.com/cran/piano
* URL: http://www.sysbio.se/piano
* BugReports: https://github.com/varemo/piano/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 143

Run `revdep_details(,"piano")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rsbml’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘PGSEA’, ‘samr’, ‘GSA’
    Unknown package ‘HTSanalyzeR’ in Rd xrefs
    ```

# pitchRx

<details>

* Version: 1.8.2
* Source code: https://github.com/cran/pitchRx
* URL: http://cpsievert.github.com/pitchRx
* BugReports: http://github.com/cpsievert/pitchRx/issues
* Date/Publication: 2015-12-09 12:57:25
* Number of recursive dependencies: 94

Run `revdep_details(,"pitchRx")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ggsubplot’
    ```

# pivottabler

<details>

* Version: 1.5.0
* Source code: https://github.com/cran/pivottabler
* URL: http://www.pivottabler.org.uk/, https://github.com/cbailiss/pivottabler
* BugReports: https://github.com/cbailiss/pivottabler/issues
* Date/Publication: 2020-06-16 16:20:03 UTC
* Number of recursive dependencies: 78

Run `revdep_details(,"pivottabler")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.0Mb
    ```

# plethem

<details>

* Version: 0.1.7
* Source code: https://github.com/cran/plethem
* Date/Publication: 2019-01-02 15:50:03 UTC
* Number of recursive dependencies: 146

Run `revdep_details(,"plethem")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      You are now using PLETHEM.
                              If you have installed PLETHEM for the first time or have just updated it, please run setUserDb() to select an existing database or createUserDb() to create a new one.
      > 
      > test_check("plethem")
      ── 1. Error: Tissue volumes are less than 85% of body weight (@test_physio.R#19)
      invalid 'type' (list) of argument
      Backtrace:
       1. plethem::getLifecourseTissueVolumes(...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 8 | SKIPPED: 0 | WARNINGS: 1 | FAILED: 1 ]
      1. Error: Tissue volumes are less than 85% of body weight (@test_physio.R#19) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘V8’ ‘devtools’ ‘formatR’ ‘gdata’ ‘rhandsontable’ ‘shinythemes’
      ‘sqldf’
      All declared Imports should be used.
    ```

# plotGrouper

<details>

* Version: 1.6.0
* Source code: https://github.com/cran/plotGrouper
* URL: https://jdgagnon.github.io/plotGrouper/
* BugReports: https://github.com/jdgagnon/plotGrouper/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 136

Run `revdep_details(,"plotGrouper")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    gplot: no visible binding for global variable ‘max_value’
    gplot: no visible binding for global variable ‘max_error’
    Undefined global functions or variables:
      max_error max_value
    ```

# plotly

<details>

* Version: 4.9.2.1
* Source code: https://github.com/cran/plotly
* URL: https://plotly-r.com, https://github.com/ropensci/plotly#readme, https://plot.ly/r
* BugReports: https://github.com/ropensci/plotly/issues
* Date/Publication: 2020-04-04 19:50:02 UTC
* Number of recursive dependencies: 150

Run `revdep_details(,"plotly")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        htmlwidgets   3.7Mb
    ```

# pmd

<details>

* Version: 0.1.9
* Source code: https://github.com/cran/pmd
* URL: https://yufree.github.io/pmd
* BugReports: https://github.com/yufree/pmd/issues
* Date/Publication: 2020-06-04 17:50:02 UTC
* Number of recursive dependencies: 110

Run `revdep_details(,"pmd")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 62 marked UTF-8 strings
    ```

# pogos

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/pogos
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 97

Run `revdep_details(,"pogos")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    compoundsByCell: no visible binding for global variable ‘cell_lines_v1’
    compoundsByCell: no visible binding for global variable ‘compounds_v1’
    compoundsByCell: no visible binding for global variable ‘datasets_v1’
    lkc: no visible binding for global variable ‘cell_lines_v1’
    lkc: no visible binding for global variable ‘compounds_v1’
    lkc: no visible binding for global variable ‘datasets_v1’
    plot,DRProfSet-missing: no visible binding for global variable ‘dose’
    plot,DRProfSet-missing: no visible binding for global variable
      ‘response’
    plot,DRProfSet-missing: no visible binding for global variable ‘drug’
    plot,DRTraceSet-missing: no visible binding for global variable ‘dose’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘response’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘cell_line’
    Undefined global functions or variables:
      cell_line cell_lines_v1 compounds_v1 datasets_v1 dose drug response
    ```

# polaroid

<details>

* Version: 0.0.1
* Source code: https://github.com/cran/polaroid
* URL: https://github.com/jhk0530/polaroid
* BugReports: https://github.com/jhk0530/polaroid/issues
* Date/Publication: 2020-03-30 16:00:02 UTC
* Number of recursive dependencies: 79

Run `revdep_details(,"polaroid")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘argonDash’ ‘argonR’ ‘colourpicker’ ‘ggplot2’ ‘grid’ ‘hexSticker’
      ‘png’ ‘shinyWidgets’
      All declared Imports should be used.
    ```

# polmineR

<details>

* Version: 0.8.0
* Source code: https://github.com/cran/polmineR
* URL: https://www.github.com/PolMine/polmineR
* BugReports: https://github.com/PolMine/polmineR/issues
* Date/Publication: 2019-12-17 12:50:06 UTC
* Number of recursive dependencies: 99

Run `revdep_details(,"polmineR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        extdata   1.9Mb
    ```

# precisely

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/precisely
* URL: https://github.com/malcolmbarrett/precisely
* BugReports: https://github.com/malcolmbarrett/precisely/issues
* Date/Publication: 2020-06-02 08:50:09 UTC
* Number of recursive dependencies: 115

Run `revdep_details(,"precisely")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘shinycssloaders’ ‘shinythemes’
      All declared Imports should be used.
    ```

# PrecisionTrialDrawer

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/PrecisionTrialDrawer
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 128

Run `revdep_details(,"PrecisionTrialDrawer")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘LowMACAAnnotation’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# predictoR

<details>

* Version: 1.1.0
* Source code: https://github.com/cran/predictoR
* URL: http://www.promidat.com
* Date/Publication: 2019-06-24 22:50:03 UTC
* Number of recursive dependencies: 142

Run `revdep_details(,"predictoR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘ROCR’ ‘ada’ ‘colourpicker’ ‘corrplot’ ‘e1071’ ‘flexdashboard’
      ‘glmnet’ ‘kknn’ ‘neuralnet’ ‘randomForest’ ‘rattle’ ‘rpart’
      ‘shinyAce’ ‘shinyWidgets’ ‘shinydashboardPlus’ ‘shinyjs’ ‘tidyverse’
      ‘xgboost’ ‘zip’
      All declared Imports should be used.
    ```

# PREPShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/PREPShiny
* Date/Publication: 2019-04-25 12:30:03 UTC
* Number of recursive dependencies: 78

Run `revdep_details(,"PREPShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘datasets’ ‘dplyr’ ‘psycho’ ‘shiny’
      All declared Imports should be used.
    ```

# primirTSS

<details>

* Version: 1.6.1
* Source code: https://github.com/cran/primirTSS
* URL: https://github.com/ipumin/primirTSS
* BugReports: http://github.com/ipumin/primirTSS/issues
* Date/Publication: 2020-06-09
* Number of recursive dependencies: 176

Run `revdep_details(,"primirTSS")` for more info

</details>

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: 'magrittr'
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_primiRNA_track: no visible binding for global variable
      ‘symbol_name’
    plot_primiRNA_track: no visible binding for global variable
      ‘stem_loop_p1’
    plot_primiRNA_track: no visible binding for global variable
      ‘stem_loop_p2’
    plot_primiRNA_track: no visible binding for global variable ‘gene_id’
    plot_primiRNA_track: no visible binding for global variable ‘gene_p1’
    plot_primiRNA_track: no visible binding for global variable ‘gene_p2’
    plot_primiRNA_track: no visible binding for global variable ‘tss_p1’
    plot_primiRNA_track: no visible binding for global variable ‘tss_p2’
    require_fa: no visible binding for global variable ‘arrow’
    tss_filter: no visible binding for global variable ‘gene_id’
    tss_filter: no visible binding for global variable ‘new_info’
    tss_filter: no visible binding for global variable ‘predicted_tss’
    tss_filter: no visible binding for global variable ‘tss_type’
    Undefined global functions or variables:
      TF TF_class arrow can_tss dhs_p1 dhs_p2 e_p_rank end1 eponine_rank
      gene_id gene_p1 gene_p2 histone_p1_flank histone_p2_flank loci
      mir_name new_info phast_rank predicted_tss previous seqname start1
      stem_loop_p1 stem_loop_p2 symbol_name tss_p1 tss_p2 tss_type
    ```

# prioritizr

<details>

* Version: 5.0.1
* Source code: https://github.com/cran/prioritizr
* URL: https://prioritizr.net, https://github.com/prioritizr/prioritizr
* BugReports: https://github.com/prioritizr/prioritizr/issues
* Date/Publication: 2020-05-15 09:50:02 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"prioritizr")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'gurobi', 'Rsymphony'
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘exactextractr’
      All declared Imports should be used.
    ```

# PROBShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/PROBShiny
* Date/Publication: 2019-04-23 08:30:03 UTC
* Number of recursive dependencies: 34

Run `revdep_details(,"PROBShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘LaplacesDemon’ ‘epitools’ ‘rpivotTable’ ‘shiny’ ‘shinyMatrix’
      All declared Imports should be used.
    ```

# processanimateR

<details>

* Version: 1.0.3
* Source code: https://github.com/cran/processanimateR
* URL: https://github.com/bupaverse/processanimateR/
* BugReports: https://github.com/bupaverse/processanimateR/issues
* Date/Publication: 2020-03-13 21:30:02 UTC
* Number of recursive dependencies: 103

Run `revdep_details(,"processanimateR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.5Mb
      sub-directories of 1Mb or more:
        doc           8.8Mb
        htmlwidgets   2.5Mb
    ```

# psichomics

<details>

* Version: 1.14.1
* Source code: https://github.com/cran/psichomics
* URL: https://nuno-agostinho.github.io/psichomics/
* BugReports: https://github.com/nuno-agostinho/psichomics/issues
* Date/Publication: 2020-06-16
* Number of recursive dependencies: 236

Run `revdep_details(,"psichomics")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   4.2Mb
    ```

# pwrEWAS

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/pwrEWAS
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 120

Run `revdep_details(,"pwrEWAS")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    > ### ** Examples
    > 
    > outDelta <- pwrEWAS(minTotSampleSize = 10,
    +     maxTotSampleSize = 20,
    +     SampleSizeSteps = 10,
    +     NcntPer = 0.5,
    +     targetDelta = c(0.2, 0.5),
    +     J = 1000,
    +     targetDmCpGs = 10,
    +     tissueType = "Adult (PBMC)",
    +     detectionLimit = 0.01,
    +     DMmethod = "limma",
    +     FDRcritVal = 0.05,
    +     core = 2,
    +     sims = 30)
    Error: Corrupt Cache: sqlite file
      See vignette section on corrupt cache
      cache: /Users/barret/Library/Caches/ExperimentHub
      filename: experimenthub.sqlite3
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      pwrEWAS RUnit Tests - 1 test function, 1 error, 0 failures
      ERROR in test_pwrEWAS: Error : Corrupt Cache: sqlite file
        See vignette section on corrupt cache
        cache: /Users/barret/Library/Caches/ExperimentHub
        filename: experimenthub.sqlite3
      
      Test files with failing tests
      
         test_pwrEWAS.R 
           test_pwrEWAS 
      
      
      Error in BiocGenerics:::testPackage("pwrEWAS") : 
        unit tests failed for package pwrEWAS
      Execution halted
    ```

# QCA

<details>

* Version: 3.8.2
* Source code: https://github.com/cran/QCA
* Date/Publication: 2020-06-03 06:30:09 UTC
* Number of recursive dependencies: 20

Run `revdep_details(,"QCA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.3Mb
      sub-directories of 1Mb or more:
        gui          1.6Mb
        staticdocs   3.3Mb
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# qlcData

<details>

* Version: 0.2.1
* Source code: https://github.com/cran/qlcData
* Date/Publication: 2018-01-06 00:00:34 UTC
* Number of recursive dependencies: 56

Run `revdep_details(,"qlcData")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1459 marked UTF-8 strings
    ```

# qqplotr

<details>

* Version: 0.0.4
* Source code: https://github.com/cran/qqplotr
* URL: https://github.com/aloy/qqplotr
* BugReports: https://github.com/aloy/qqplotr/issues
* Date/Publication: 2020-02-04 07:40:02 UTC
* Number of recursive dependencies: 102

Run `revdep_details(,"qqplotr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# questionr

<details>

* Version: 0.7.1
* Source code: https://github.com/cran/questionr
* URL: https://juba.github.io/questionr/
* BugReports: https://github.com/juba/questionr/issues
* Date/Publication: 2020-05-26 11:30:05 UTC
* Number of recursive dependencies: 80

Run `revdep_details(,"questionr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘haven’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4145 marked UTF-8 strings
    ```

# R3CPET

<details>

* Version: 1.20.0
* Source code: https://github.com/cran/R3CPET
* BugReports: https://github.com/sirusb/R3CPET/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 172

Run `revdep_details(,"R3CPET")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘clues’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# RagGrid

<details>

* Version: 0.2.0
* Source code: https://github.com/cran/RagGrid
* URL: https://github.com/no-types/RagGrid/
* BugReports: https://github.com/no-types/RagGrid/issues
* Date/Publication: 2018-08-12 09:30:03 UTC
* Number of recursive dependencies: 32

Run `revdep_details(,"RagGrid")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# rAmCharts

<details>

* Version: 2.1.13
* Source code: https://github.com/cran/rAmCharts
* URL: http://datastorm-open.github.io/introduction_ramcharts/
* BugReports: https://github.com/datastorm-open/rAmCharts/issues/
* Date/Publication: 2019-12-06 15:50:05 UTC
* Number of recursive dependencies: 52

Run `revdep_details(,"rAmCharts")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.8Mb
      sub-directories of 1Mb or more:
        api           3.1Mb
        htmlwidgets   7.0Mb
    ```

# rangeModelMetadata

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/rangeModelMetadata
* Date/Publication: 2020-03-14 16:50:03 UTC
* Number of recursive dependencies: 209

Run `revdep_details(,"rangeModelMetadata")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘dplyr’ ‘jsonlite’ ‘rgbif’ ‘rgdal’ ‘rgeos’ ‘sp’ ‘spThin’
      ‘spatstat’
      All declared Imports should be used.
    ```

# rapbase

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/rapbase
* URL: http://github.com/Rapporteket/rapbase
* BugReports: http://github.com/Rapporteket/rapbase/issues
* Date/Publication: 2019-08-07 07:30:02 UTC
* Number of recursive dependencies: 97

Run `revdep_details(,"rapbase")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RJDBC’
      All declared Imports should be used.
    ```

# Rariant

<details>

* Version: 1.24.0
* Source code: https://github.com/cran/Rariant
* URL: https://github.com/juliangehring/Rariant
* BugReports: https://support.bioconductor.org
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 168

Run `revdep_details(,"Rariant")` for more info

</details>

## In both

*   checking whether package ‘Rariant’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/Rariant/new/Rariant.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘Rariant’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: object 'rbind_all' is not exported by 'namespace:dplyr'
Execution halted
ERROR: lazy loading failed for package ‘Rariant’
* removing ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/Rariant/new/Rariant.Rcheck/Rariant’

```
### CRAN

```
* installing *source* package ‘Rariant’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: object 'rbind_all' is not exported by 'namespace:dplyr'
Execution halted
ERROR: lazy loading failed for package ‘Rariant’
* removing ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/Rariant/old/Rariant.Rcheck/Rariant’

```
# rcellminer

<details>

* Version: 2.10.0
* Source code: https://github.com/cran/rcellminer
* URL: http://discover.nci.nih.gov/cellminer/
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 104

Run `revdep_details(,"rcellminer")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... WARNING
    ```
      Error loading dataset 'fingerprintList':
       Error in .requirePackage(package) : 
        unable to find required package 'fingerprint'
      
      The dataset(s) may use package(s) not declared in the DESCRIPTION file.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.9Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    5.9Mb
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘rcellminer’ for: ‘initialize’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotCellMiner: no visible global function definition for ‘par’
    plotCellMiner: no visible global function definition for ‘layout’
    plotCellMiner: no visible global function definition for ‘lcm’
    plotCellMiner: no visible global function definition for ‘axis’
    plotDrugSets: no visible global function definition for ‘par’
    plotDrugSets: no visible global function definition for ‘axis’
    plotDrugSets: no visible global function definition for ‘segments’
    Undefined global functions or variables:
      axis layout lcm par segments
    Consider adding
      importFrom("graphics", "axis", "layout", "lcm", "par", "segments")
    to your NAMESPACE file.
    ```

# rCGH

<details>

* Version: 1.18.0
* Source code: https://github.com/cran/rCGH
* URL: https://github.com/fredcommo/rCGH
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 136

Run `revdep_details(,"rCGH")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data      2.8Mb
        extdata   1.2Mb
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# Rcwl

<details>

* Version: 1.4.2
* Source code: https://github.com/cran/Rcwl
* Date/Publication: 2020-05-12
* Number of recursive dependencies: 106

Run `revdep_details(,"Rcwl")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(yaml)
      > test_check("Rcwl")
      ── 1. Error: extensions (@test_all.R#94)  ──────────────────────────────────────
      invalid 'description' argument
      Backtrace:
       1. testthat::expect_true(all(ext %in% read_yaml(cwlfile)))
       5. yaml::read_yaml(cwlfile)
       6. base::file(file, "rt", encoding = fileEncoding)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 16 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 1 ]
      1. Error: extensions (@test_all.R#94) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking S3 generic/method consistency ... WARNING
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking replacement functions ... WARNING
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    The argument of a replacement function which corresponds to the right
    hand side must be named ‘value’.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    ```

*   checking foreign function calls ... NOTE
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    ```

*   checking Rd \usage sections ... NOTE
    ```
    /usr/local/lib/python2.7/site-packages/cwltool/__init__.py:17: CWLToolDeprecationWarning: 
    DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020.
    Please upgrade your Python as the Python 2.7 version of cwltool won't be
    maintained after that date.
    
      """,  category=CWLToolDeprecationWarning)
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# rddapp

<details>

* Version: 1.2.1
* Source code: https://github.com/cran/rddapp
* Date/Publication: 2020-02-20 06:30:10 UTC
* Number of recursive dependencies: 131

Run `revdep_details(,"rddapp")` for more info

</details>

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
      Attaching package: 'zoo'
      
      The following objects are masked from 'package:base':
      
          as.Date, as.Date.numeric
      
      Loading required package: AER
      Loading required package: car
      Loading required package: carData
      Loading required package: survival
      Loading required package: Formula
      > library(rddtools)
      Error in library(rddtools) : there is no package called 'rddtools'
      Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rddtools’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Unknown package ‘rddtools’ in Rd xrefs
    ```

# recmap

<details>

* Version: 1.0.7
* Source code: https://github.com/cran/recmap
* BugReports: https://github.com/cpanse/recmap/issues
* Date/Publication: 2020-02-20 06:30:02 UTC
* Number of recursive dependencies: 55

Run `revdep_details(,"recmap")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1036 marked UTF-8 strings
    ```

# ReDaMoR

<details>

* Version: 0.4.2
* Source code: https://github.com/cran/ReDaMoR
* URL: https://github.com/patzaw/ReDaMoR
* BugReports: https://github.com/patzaw/ReDaMoR/issues
* Date/Publication: 2020-03-31 12:50:08 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"ReDaMoR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘igraph’
      All declared Imports should be used.
    ```

# RefNet

<details>

* Version: 1.24.0
* Source code: https://github.com/cran/RefNet
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 84

Run `revdep_details(,"RefNet")` for more info

</details>

## In both

*   checking whether package ‘RefNet’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: Package 'RefNet' is deprecated and will be removed from Bioconductor
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/RefNet/new/RefNet.Rcheck/00install.out’ for details.
    ```

# regressoR

<details>

* Version: 1.1.8
* Source code: https://github.com/cran/regressoR
* URL: http://www.promidat.com
* Date/Publication: 2019-09-19 04:30:03 UTC
* Number of recursive dependencies: 121

Run `revdep_details(,"regressoR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ROCR’ ‘colourpicker’ ‘corrplot’ ‘e1071’ ‘flexdashboard’ ‘gbm’
      ‘glmnet’ ‘kknn’ ‘neuralnet’ ‘rattle’ ‘rpart’ ‘shinyWidgets’
      ‘shinydashboardPlus’ ‘shinyjs’ ‘xgboost’ ‘zip’
      All declared Imports should be used.
    ```

# ReportingTools

<details>

* Version: 2.28.0
* Source code: https://github.com/cran/ReportingTools
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 175

Run `revdep_details(,"ReportingTools")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      'fData'
    .marrayLM.to.html: no visible global function definition for
      'featureNames'
    .marrayLM.to.html: no visible global function definition for 'fData'
    check.ids: no visible binding for global variable 'org.Hs.eg.db'
    check.ids: no visible global function definition for 'keys'
    custHeaderPanel : <anonymous>: no visible binding for global variable
      'tags'
    custHeaderPanel : <anonymous>: no visible global function definition
      for 'HTML'
    custHeaderPanel: no visible global function definition for 'tagList'
    custHeaderPanel: no visible global function definition for 'tag'
    custHeaderPanel: no visible global function definition for 'div'
    custHeaderPanel: no visible global function definition for 'h1'
    publish,trellis-HTMLReport: no visible binding for global variable
      'htmlRep'
    toReportDF,DESeqDataSet: no visible global function definition for
      'mcols'
    Undefined global functions or variables:
      HTML columns description div exprs fData featureNames h1 htmlRep keys
      keytype mcols org.Hs.eg.db tag tagList tags
    ```

# reverseR

<details>

* Version: 0.1
* Source code: https://github.com/cran/reverseR
* Date/Publication: 2019-04-24 15:40:03 UTC
* Number of recursive dependencies: 31

Run `revdep_details(,"reverseR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# RforProteomics

<details>

* Version: 1.26.0
* Source code: https://github.com/cran/RforProteomics
* URL: http://lgatto.github.com/RforProteomics/
* Date/Publication: 2020-05-07
* Number of recursive dependencies: 268

Run `revdep_details(,"RforProteomics")` for more info

</details>

## Newly broken

*   checking R code for possible problems ... NOTE
    ```
    shinyMA: possible error in plotOutput("plotma", clickId =
      "plotma_click", width = 400, height = 400): unused argument (clickId
      = "plotma_click")
    ```

## In both

*   checking whether package ‘RforProteomics’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'plotOutput("plotma", ': unused argument (clickId = "plotma_click") 
      Warning: replacing previous import ‘MSnbase::plot’ by ‘graphics::plot’ when loading ‘RforProteomics’
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/RforProteomics/new/RforProteomics.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'xcms', 'IPPD'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.7Mb
      sub-directories of 1Mb or more:
        doc  10.8Mb
    ```

# Rfssa

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/Rfssa
* URL: https://github.com/haghbinh/Rfssa.git
* Date/Publication: 2019-09-12 22:40:20 UTC
* Number of recursive dependencies: 97

Run `revdep_details(,"Rfssa")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘markdown’
      All declared Imports should be used.
    ```

# rgl

<details>

* Version: 0.100.54
* Source code: https://github.com/cran/rgl
* URL: https://r-forge.r-project.org/projects/rgl/
* BugReports: https://r-forge.r-project.org/projects/rgl/
* Date/Publication: 2020-04-14 14:40:02 UTC
* Number of recursive dependencies: 59

Run `revdep_details(,"rgl")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.7Mb
      sub-directories of 1Mb or more:
        doc     3.6Mb
        fonts   1.5Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘heplots’
    ```

# rhandsontable

<details>

* Version: 0.3.7
* Source code: https://github.com/cran/rhandsontable
* URL: http://jrowen.github.io/rhandsontable/
* BugReports: https://github.com/jrowen/rhandsontable/issues
* Date/Publication: 2018-11-20 05:50:03 UTC
* Number of recursive dependencies: 32

Run `revdep_details(,"rhandsontable")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.5Mb
      sub-directories of 1Mb or more:
        doc          10.2Mb
        htmlwidgets   1.2Mb
    ```

# RLumShiny

<details>

* Version: 0.2.2
* Source code: https://github.com/cran/RLumShiny
* URL: https://tzerk.github.io/RLumShiny/
* BugReports: https://github.com/tzerk/RLumShiny/issues
* Date/Publication: 2019-01-11 11:30:03 UTC
* Number of recursive dependencies: 81

Run `revdep_details(,"RLumShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RCarb’ ‘rmarkdown’ ‘shinyjs’
      All declared Imports should be used.
    ```

# rmarkdown

<details>

* Version: 2.3
* Source code: https://github.com/cran/rmarkdown
* URL: https://github.com/rstudio/rmarkdown
* BugReports: https://github.com/rstudio/rmarkdown/issues
* Date/Publication: 2020-06-18 14:50:02 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"rmarkdown")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.1Mb
      sub-directories of 1Mb or more:
        rmd   8.2Mb
    ```

# rmd

<details>

* Version: 0.1.5
* Source code: https://github.com/cran/rmd
* URL: https://github.com/pzhaonet/rmd
* BugReports: https://github.com/pzhaonet/rmd/issues
* Date/Publication: 2020-03-01 15:40:02 UTC
* Number of recursive dependencies: 108

Run `revdep_details(,"rmd")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘blogdown’ ‘bookdown’ ‘bookdownplus’ ‘citr’ ‘pagedown’ ‘rticles’
      ‘tinytex’ ‘xaringan’
      All declared Imports should be used.
    ```

# RnBeads

<details>

* Version: 2.6.0
* Source code: https://github.com/cran/RnBeads
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 245

Run `revdep_details(,"RnBeads")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘IlluminaHumanMethylation450kmanifest’
    
    Depends: includes the non-default packages:
      'BiocGenerics', 'S4Vectors', 'GenomicRanges', 'MASS', 'cluster',
      'ff', 'fields', 'ggplot2', 'gplots', 'gridExtra', 'limma',
      'matrixStats', 'illuminaio', 'methylumi', 'plyr'
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.1Mb
      sub-directories of 1Mb or more:
        R     4.1Mb
        doc   3.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘qvalue’
    A package should be listed in only one of these fields.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
      'grDevices:::.smoothScatterCalcDensity'
      'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
      'minfi:::.normalizeFunnorm450k'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      Measure Mmusculus PairsBootRefFreeEwasModel Predicted Probe
      RGChannelSet Rd2HTML RefFreeEwasModel SNP Sample Segment Slide Target
      Term UcscTrack Value addSex as.profileCGH assayDataElement
      assayDataElementNames barcode browserSession bv calculateFDRs chrom
      combinedRank combinedRank.var comma covgMedian covgPercLow covgPercUp
      cv.glmnet daglad diffmeth diffmeth.p.adj.fdr diffmeth.p.val
      dinucleotideFrequency eps expectedCounts featureData featureData<-
      featureNames featureNames<- foreach geneCounts genome<- getCN
      getDoParWorkers getGreen getManifest getMeth getRed getSex getTable
      getUnmeth getVarCov glmnet grid.draw grid.newpage group group1 group2
      i impute.knn is.subsegmentation k letterFrequency lme loadRegionDB
      log10FDR log10P mapToGenome mclapply mean.diff mean.quot.log2 melt
      mergeRegionDBs muted n.sites num.sites numSites numeric.names
      oddsRatios p.vals.t.na.adj pData percent_format phenoData phenoData<-
      plotAlphaDistributionOneChr plotFinalSegmentation plotTracks
      preprocessSWAN pvalues qvalue readMethylome readSNPTable refText
      reg.type region.size registerDoParallel relative.coord removeSNPs
      report runLOLA samples segmentPMDs segmentUMRsLMRs seqlengths
      seqlevels<- sigCategories sites2ignore size solve.QP stopCluster sva
      target texthere tsne type types ucscTableQuery universeCounts useMart
      v var.diff varFit varLabels x x2 xmlValue y y2 yint
    ```

# RobStatTM

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/RobStatTM
* Date/Publication: 2020-03-03 00:30:02 UTC
* Number of recursive dependencies: 87

Run `revdep_details(,"RobStatTM")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DEoptimR’ ‘DT’ ‘PerformanceAnalytics’ ‘ggplot2’ ‘graphics’
      ‘gridExtra’ ‘methods’ ‘shinyjs’ ‘utils’ ‘xts’
      All declared Imports should be used.
    ```

# rosr

<details>

* Version: 0.0.10
* Source code: https://github.com/cran/rosr
* URL: https://github.com/pzhaonet/rosr
* BugReports: https://github.com/pzhaonet/rosr/issues
* Date/Publication: 2020-05-11 11:10:02 UTC
* Number of recursive dependencies: 91

Run `revdep_details(,"rosr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘bookdown’ ‘rstudioapi’ ‘tinytex’
      All declared Imports should be used.
    ```

# rpostgisLT

<details>

* Version: 0.6.0
* Source code: https://github.com/cran/rpostgisLT
* URL: https://github.com/mablab/rpostgisLT
* BugReports: https://github.com/mablab/rpostgisLT/issues
* Date/Publication: 2018-03-02 18:02:11 UTC
* Number of recursive dependencies: 117

Run `revdep_details(,"rpostgisLT")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# Rqc

<details>

* Version: 1.22.0
* Source code: https://github.com/cran/Rqc
* URL: https://github.com/labbcb/Rqc
* BugReports: https://github.com/labbcb/Rqc/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 150

Run `revdep_details(,"Rqc")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'IRanges' 'Rcpp'
      All declared Imports should be used.
    ```

# RQuantLib

<details>

* Version: 0.4.12
* Source code: https://github.com/cran/RQuantLib
* URL: http://dirk.eddelbuettel.com/code/rquantlib.html
* BugReports: https://github.com/eddelbuettel/rquantlib/issues
* Date/Publication: 2020-04-02 11:50:06 UTC
* Number of recursive dependencies: 41

Run `revdep_details(,"RQuantLib")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    +                fixFreq="Annual",
    +                floatFreq="Semiannual")
    + 
    + setEvaluationDate(as.Date("2016-2-16"))               
    + times<-times <- seq(0,14.75,.25)
    + dcurve <- DiscountCurve(params, tsQuotes, times=times,legparams)
    + 
    + # Price the Bermudan swaption
    + pricing <- AffineSwaption(params, dcurve,swaptionMaturities, swapTenors, volMatrix,legparams)
    + summary(pricing)
    + 
    + }    
    
     *** caught segfault ***
    address 0x0, cause 'memory not mapped'
    
    Traceback:
     1: affineWithRebuiltCurveEngine(params, matchlegs, c(ts$table$date),     ts$table$zeroRates, expiry, tenor, vol)
     2: AffineSwaption.default(params, dcurve, swaptionMaturities, swapTenors,     volMatrix, legparams)
     3: AffineSwaption(params, dcurve, swaptionMaturities, swapTenors,     volMatrix, legparams)
    An irrecoverable exception occurred. R is aborting now ...
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/RQuantlib.R’ failed.
    Last 13 lines of output:
      +                             settleDate=as.Date('2002-2-15'),
      +                             dt=0.25,
      +                             interpWhat='discount', interpHow='loglinear')
      > discountCurve <- DiscountCurve(discountCurve.param, list(flat=0.05))
      > 
      > ZeroCouponBond(bond, discountCurve, dateparams)
      
       *** caught segfault ***
      address 0x0, cause 'memory not mapped'
      
      Traceback:
       1: ZeroBondWithRebuiltCurve(bond, c(discountCurve$table$date), discountCurve$table$zeroRates,     dateparams)
       2: ZeroCouponBond.default(bond, discountCurve, dateparams)
       3: ZeroCouponBond(bond, discountCurve, dateparams)
      An irrecoverable exception occurred. R is aborting now ...
    ```

*   checking whether package ‘RQuantLib’ can be installed ... WARNING
    ```
    ...
      /usr/local/include/ql/models/marketmodels/products/multistep/multistepcoinitialswaps.hpp:48:14: warning: 'auto_ptr<QuantLib::MarketModelMultiProduct>' is deprecated [-Wdeprecated-declarations]#    define _LIBCPP_DEPRECATED __attribute__ ((deprecated))
      55:14: warning: 'auto_ptr<QuantLib::MarketModelMultiProduct>' is deprecated [-Wdeprecated-declarations]
      :49:14: warning: 'auto_ptr<QuantLib::MarketModelMultiProduct>' is deprecated [-Wdeprecated-declarations]In file included from calendars.cpp:20:
      :2080:28:136:22: warning: 'auto_ptr<QuantLib::MarketModelPathwiseMultiProduct>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/models/marketmodels/products/multistep/multistepratchet.hpp#    define _LIBCPP_DEPRECATED __attribute__ ((deprecated)):51:14: warning: note'auto_ptr<QuantLib::MarketModelMultiProduct>' is deprecated [-Wdeprecated-declarations]: expanded from macro '_LIBCPP_DEPRECATED'
      /usr/local/include/ql/models/marketmodels/products/pathwise/pathwiseproductcaplet.hpp:136:22: warning: 2080:28: 'auto_ptr<QuantLib::MarketModelPathwiseMultiProduct>' is deprecated [-Wdeprecated-declarations]
      In file included from 48: warning: 'auto_ptr<QuantLib::MarketModelPathwiseMultiProduct>' is deprecated [-Wdeprecated-declarations]
      :195:22: warning: 'auto_ptr<QuantLib::MarketModelPathwiseMultiProduct>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/utilities/clone.hpp:49:20: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/termstructures/yield/fittedbonddiscountcurve.hpp:201:22: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/termstructures/yield/nonlinearfittingmethods.hpp:60:14: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/termstructures/yield/nonlinearfittingmethods.hpp:89:14: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/termstructures/yield/nonlinearfittingmethods.hpp:120:14: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/termstructures/yield/nonlinearfittingmethods.hpp:165:14: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/termstructures/yield/nonlinearfittingmethods.hpp:203:14: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      /usr/local/include/ql/termstructures/yield/nonlinearfittingmethods.hpp:223:14: warning: 'auto_ptr<QuantLib::FittedBondDiscountCurve::FittingMethod>' is deprecated [-Wdeprecated-declarations]
      affine.cpp:33:53: warning: 'CalibrationHelper' is deprecated [-Wdeprecated-declarations]
      bermudan.cpp:33:53: warning: 'CalibrationHelper' is deprecated [-Wdeprecated-declarations]
      hullwhite.cpp:43:54: warning: 'CalibrationHelper' is deprecated [-Wdeprecated-declarations]
      hullwhite.cpp:111:54: warning: 'CalibrationHelper' is deprecated [-Wdeprecated-declarations]
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/RQuantLib/new/RQuantLib.Rcheck/00install.out’ for details.
    ```

# rrvgo

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/rrvgo
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 187

Run `revdep_details(,"rrvgo")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘GOSemSim:::getAncestors’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    scatterPlot: no visible binding for global variable ‘V1’
    scatterPlot: no visible binding for global variable ‘V2’
    scatterPlot: no visible binding for global variable ‘parentTerm’
    scatterPlot: no visible binding for global variable ‘parent’
    Undefined global functions or variables:
      V1 V2 parent parentTerm
    ```

# rtimicropem

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/rtimicropem
* URL: https://github.com/ropensci/rtimicropem, https://docs.ropensci.org/rtimicropem/
* BugReports: https://github.com/ropensci/rtimicropem/issues
* Date/Publication: 2019-05-15 09:40:03 UTC
* Number of recursive dependencies: 94

Run `revdep_details(,"rtimicropem")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# RtutoR

<details>

* Version: 1.2
* Source code: https://github.com/cran/RtutoR
* Date/Publication: 2018-09-14 07:50:07 UTC
* Number of recursive dependencies: 120

Run `revdep_details(,"RtutoR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘rmarkdown’
      All declared Imports should be used.
    ```

# RxODE

<details>

* Version: 0.9.2-0
* Source code: https://github.com/cran/RxODE
* URL: https://nlmixrdevelopment.github.io/RxODE/
* BugReports: https://github.com/nlmixrdevelopment/RxODE/issues
* Date/Publication: 2020-03-13 07:10:14 UTC
* Number of recursive dependencies: 132

Run `revdep_details(,"RxODE")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'SnakeCharmR', 'installr'
    ```

# safetyGraphics

<details>

* Version: 1.1.0
* Source code: https://github.com/cran/safetyGraphics
* URL: https://github.com/SafetyGraphics/safetyGraphics
* BugReports: https://github.com/SafetyGraphics/safetyGraphics/issues
* Date/Publication: 2020-01-15 22:50:05 UTC
* Number of recursive dependencies: 101

Run `revdep_details(,"safetyGraphics")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinybusy’
      All declared Imports should be used.
    ```

# sangerseqR

<details>

* Version: 1.24.0
* Source code: https://github.com/cran/sangerseqR
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 39

Run `revdep_details(,"sangerseqR")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    chromatogram,sangerseq: no visible global function definition for
      ‘quantile’
    chromatogram,sangerseq: no visible global function definition for ‘IQR’
    chromatogram,sangerseq: no visible global function definition for ‘pdf’
    chromatogram,sangerseq: no visible global function definition for
      ‘rect’
    chromatogram,sangerseq: no visible global function definition for
      ‘lines’
    chromatogram,sangerseq: no visible global function definition for
      ‘mtext’
    chromatogram,sangerseq: no visible global function definition for
      ‘axis’
    chromatogram,sangerseq: no visible global function definition for
      ‘dev.off’
    Undefined global functions or variables:
      IQR axis dev.off lines mtext par pdf quantile rect
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("graphics", "axis", "lines", "mtext", "par", "rect")
      importFrom("stats", "IQR", "quantile")
    to your NAMESPACE file.
    ```

# santaR

<details>

* Version: 1.0
* Source code: https://github.com/cran/santaR
* URL: https://github.com/adwolfer
* Date/Publication: 2018-01-24 18:27:08 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"santaR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gridExtra’
      All declared Imports should be used.
    ```

# SC3

<details>

* Version: 1.16.0
* Source code: https://github.com/cran/SC3
* URL: https://github.com/hemberg-lab/SC3
* BugReports: https://support.bioconductor.org/t/sc3/
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 129

Run `revdep_details(,"SC3")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .github
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   2.7Mb
        doc    2.4Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

# scone

<details>

* Version: 1.12.0
* Source code: https://github.com/cran/scone
* BugReports: https://github.com/YosefLab/scone/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 221

Run `revdep_details(,"scone")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    sconeReport : server: no visible global function definition for ‘theme’
    sconeReport : server: no visible global function definition for
      ‘element_blank’
    sconeReport : server: no visible global function definition for
      ‘ggplotly’
    sconeReport : server: no visible global function definition for
      ‘geom_violin’
    sconeReport : server: no visible global function definition for
      ‘coord_cartesian’
    sconeReport : server: no visible global function definition for
      ‘scale_fill_manual’
    sconeReport : server: no visible global function definition for
      ‘geom_point’
    sconeReport : server: no visible global function definition for
      ‘guides’
    Undefined global functions or variables:
      %>% aes coord_cartesian element_blank geom_bar geom_point geom_violin
      ggplot ggplotly guides labs plot_ly plotlyOutput renderVisNetwork
      scale_fill_manual theme visEdges visGroups visHierarchicalLayout
      visLegend visNetwork visNetworkOutput visNetworkProxy visOptions
      visSelectNodes ylim
    ```

# SDEFSR

<details>

* Version: 0.7.21
* Source code: https://github.com/cran/SDEFSR
* URL: https://github.com/aklxao2/SDR
* Date/Publication: 2020-01-28 12:40:10 UTC
* Number of recursive dependencies: 65

Run `revdep_details(,"SDEFSR")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# SEA

<details>

* Version: 1.0
* Source code: https://github.com/cran/SEA
* Date/Publication: 2018-05-24 12:35:11 UTC
* Number of recursive dependencies: 27

Run `revdep_details(,"SEA")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.8Mb
      sub-directories of 1Mb or more:
        R   8.0Mb
    ```

# semdrw

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/semdrw
* Date/Publication: 2018-11-14 11:00:02 UTC
* Number of recursive dependencies: 146

Run `revdep_details(,"semdrw")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘lavaan’ ‘psych’ ‘semPlot’ ‘semTools’ ‘shinyAce’
      All declared Imports should be used.
    ```

# SemNeT

<details>

* Version: 1.3.0
* Source code: https://github.com/cran/SemNeT
* URL: https://github.com/AlexChristensen/SemNeT
* BugReports: https://github.com/AlexChristensen/SemNeT/issues
* Date/Publication: 2020-06-09 14:10:03 UTC
* Number of recursive dependencies: 140

Run `revdep_details(,"SemNeT")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘grid’ ‘purrr’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘SemNetCleaner’
    ```

# seqplots

<details>

* Version: 1.26.0
* Source code: https://github.com/cran/seqplots
* URL: http://github.com/przemol/seqplots
* BugReports: http://github.com/przemol/seqplots/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 125

Run `revdep_details(,"seqplots")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 15.0Mb
      sub-directories of 1Mb or more:
        doc        3.7Mb
        seqplots  10.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiocManager’
      All declared Imports should be used.
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("BWGFile_summary", ..., PACKAGE = "rtracklayer")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotHeatmap,list: no visible global function definition for ‘hclust’
    plotHeatmap,list: no visible global function definition for ‘dist’
    plotHeatmap,list: no visible global function definition for ‘cutree’
    plotHeatmap,list: no visible global function definition for
      ‘as.dendrogram’
    plotHeatmap,list: no visible global function definition for ‘title’
    Undefined global functions or variables:
      Var1 Var2 abline adjustcolor approx as.dendrogram axis box
      capture.output colorRampPalette cutree dist doFileOperations hclust
      image kmeans layout lines mtext par plot.new qt rainbow rect rgb text
      title value
    Consider adding
      importFrom("grDevices", "adjustcolor", "colorRampPalette", "rainbow",
                 "rgb")
      importFrom("graphics", "abline", "axis", "box", "image", "layout",
                 "lines", "mtext", "par", "plot.new", "rect", "text",
                 "title")
      importFrom("stats", "approx", "as.dendrogram", "cutree", "dist",
                 "hclust", "kmeans", "qt")
      importFrom("utils", "capture.output")
    to your NAMESPACE file.
    ```

# sglr

<details>

* Version: 0.7
* Source code: https://github.com/cran/sglr
* Date/Publication: 2014-05-03 00:54:29
* Number of recursive dependencies: 58

Run `revdep_details(,"sglr")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .computePStar: no visible global function definition for ‘uniroot’
    plotBoundary: no visible global function definition for
      ‘scale_y_continuous’
    Undefined global functions or variables:
      scale_y_continuous uniroot
    Consider adding
      importFrom("stats", "uniroot")
    to your NAMESPACE file.
    ```

# shiny.router

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/shiny.router
* Date/Publication: 2018-09-18 08:50:07 UTC
* Number of recursive dependencies: 37

Run `revdep_details(,"shiny.router")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# shiny.semantic

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/shiny.semantic
* BugReports: https://github.com/Appsilon/shiny.semantic/issues
* Date/Publication: 2020-03-13 07:10:02 UTC
* Number of recursive dependencies: 70

Run `revdep_details(,"shiny.semantic")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.2Mb
      sub-directories of 1Mb or more:
        www   8.0Mb
    ```

# shinyAce

<details>

* Version: 0.4.1
* Source code: https://github.com/cran/shinyAce
* BugReports: https://github.com/trestletech/shinyAce/issues
* Date/Publication: 2019-09-24 10:10:07 UTC
* Number of recursive dependencies: 47

Run `revdep_details(,"shinyAce")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.1Mb
      sub-directories of 1Mb or more:
        www   7.7Mb
    ```

# shinyaframe

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/shinyaframe
* Date/Publication: 2017-11-26 15:29:43 UTC
* Number of recursive dependencies: 51

Run `revdep_details(,"shinyaframe")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# shinybrms

<details>

* Version: 1.1.0
* Source code: https://github.com/cran/shinybrms
* URL: https://github.com/fweber144/shinybrms
* BugReports: https://github.com/fweber144/shinybrms/issues
* Date/Publication: 2020-06-09 19:20:02 UTC
* Number of recursive dependencies: 120

Run `revdep_details(,"shinybrms")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘brms’
      All declared Imports should be used.
    ```

# shinyBS

<details>

* Version: 0.61
* Source code: https://github.com/cran/shinyBS
* URL: https://ebailey78.github.io/shinyBS
* BugReports: https://github.com/ebailey78/shinyBS/issues
* Date/Publication: 2015-03-31 07:52:38
* Number of recursive dependencies: 18

Run `revdep_details(,"shinyBS")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    popify: no visible global function definition for ‘runif’
    tipify: no visible global function definition for ‘runif’
    Undefined global functions or variables:
      runif
    Consider adding
      importFrom("stats", "runif")
    to your NAMESPACE file.
    ```

# shinyEffects

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/shinyEffects
* URL: https://github.com/DivadNojnarg/shinyEffects, https://divadnojnarg.github.io/shinyEffects/
* BugReports: https://github.com/DivadNojnarg/shinyEffects/issues
* Date/Publication: 2018-11-18 18:40:03 UTC
* Number of recursive dependencies: 30

Run `revdep_details(,"shinyEffects")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinydashboard’
      All declared Imports should be used.
    ```

# shinyhelper

<details>

* Version: 0.3.2
* Source code: https://github.com/cran/shinyhelper
* BugReports: https://github.com/cwthom/shinyhelper/issues
* Date/Publication: 2019-11-09 12:30:02 UTC
* Number of recursive dependencies: 20

Run `revdep_details(,"shinyhelper")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘markdown’
      All declared Imports should be used.
    ```

# ShinyItemAnalysis

<details>

* Version: 1.3.3
* Source code: https://github.com/cran/ShinyItemAnalysis
* URL: http://www.ShinyItemAnalysis.org, https://CRAN.R-project.org/package=ShinyItemAnalysis
* BugReports: https://github.com/patriciamar/ShinyItemAnalysis/issues
* Date/Publication: 2020-05-04 15:20:02 UTC
* Number of recursive dependencies: 128

Run `revdep_details(,"ShinyItemAnalysis")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘VGAM’ ‘data.table’ ‘ggdendro’ ‘gridExtra’ ‘knitr’
      ‘latticeExtra’ ‘msm’ ‘plotly’ ‘shinyBS’ ‘shinydashboard’ ‘xtable’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘WrightMap’
    ```

# shinyjs

<details>

* Version: 1.1
* Source code: https://github.com/cran/shinyjs
* URL: https://deanattali.com/shinyjs
* BugReports: https://github.com/daattali/shinyjs/issues
* Date/Publication: 2020-01-13 06:40:03 UTC
* Number of recursive dependencies: 49

Run `revdep_details(,"shinyjs")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘htmltools’
      All declared Imports should be used.
    ```

# shinyKGode

<details>

* Version: 1.0.5
* Source code: https://github.com/cran/shinyKGode
* URL: https://github.com/joewandy/shinyKGode
* Date/Publication: 2018-02-12 12:57:59 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"shinyKGode")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘KGode’ ‘XML’ ‘ggplot2’ ‘gridExtra’ ‘mvtnorm’ ‘reshape2’ ‘shinyjs’
      All declared Imports should be used.
    ```

# shinyMethyl

<details>

* Version: 1.24.0
* Source code: https://github.com/cran/shinyMethyl
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 137

Run `revdep_details(,"shinyMethyl")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘IlluminaHumanMethylation450kmanifest’
    
    Package suggested but not available for checking: ‘minfiData’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# shinypanels

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/shinypanels
* URL: http://github.com/datasketch/shinypanels
* Date/Publication: 2020-01-26 10:30:08 UTC
* Number of recursive dependencies: 38

Run `revdep_details(,"shinypanels")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘shiny’ ‘shinyjs’
      All declared Imports should be used.
    ```

# shinyr

<details>

* Version: 0.2.6
* Source code: https://github.com/cran/shinyr
* Date/Publication: 2020-03-19 15:00:02 UTC
* Number of recursive dependencies: 124

Run `revdep_details(,"shinyr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘e1071’ ‘knitr’ ‘plotly’
      All declared Imports should be used.
    ```

# shinyrecap

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/shinyrecap
* URL: https://fellstat.github.io/shinyrecap/
* Date/Publication: 2019-01-19 23:40:03 UTC
* Number of recursive dependencies: 86

Run `revdep_details(,"shinyrecap")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘CARE1’ ‘LCMCR’ ‘coda’ ‘conting’ ‘dga’ ‘future’ ‘ggplot2’ ‘ipc’
      ‘promises’ ‘reshape’ ‘shinycssloaders’ ‘testthat’
      All declared Imports should be used.
    ```

# shinyTANDEM

<details>

* Version: 1.26.0
* Source code: https://github.com/cran/shinyTANDEM
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 28

Run `revdep_details(,"shinyTANDEM")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rTANDEM’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# sigmajs

<details>

* Version: 0.1.5
* Source code: https://github.com/cran/sigmajs
* URL: http://sigmajs.john-coene.com/
* BugReports: https://github.com/JohnCoene/sigmajs/issues
* Date/Publication: 2020-06-18 18:10:02 UTC
* Number of recursive dependencies: 61

Run `revdep_details(,"sigmajs")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 28 marked UTF-8 strings
    ```

# SimDesign

<details>

* Version: 2.0.1
* Source code: https://github.com/cran/SimDesign
* URL: https://github.com/philchalmers/SimDesign, https://github.com/philchalmers/SimDesign/wiki
* Date/Publication: 2020-01-20 19:00:02 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"SimDesign")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘doMPI’
    ```

# simplevis

<details>

* Version: 1.3.0
* Source code: https://github.com/cran/simplevis
* URL: https://statisticsnz.github.io/simplevis, https://github.com/statisticsnz/simplevis
* BugReports: https://github.com/statisticsNZ/simplevis/issues
* Date/Publication: 2020-06-15 12:20:02 UTC
* Number of recursive dependencies: 117

Run `revdep_details(,"simplevis")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘readr’ ‘rmarkdown’ ‘shinycssloaders’
      All declared Imports should be used.
    ```

# singleCellTK

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/singleCellTK
* URL: https://compbiomed.github.io/sctk_docs/
* BugReports: https://github.com/compbiomed/singleCellTK/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 249

Run `revdep_details(,"singleCellTK")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   1.0Mb
        doc    3.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘celda’ ‘shinyBS’ ‘shinythemes’
      All declared Imports should be used.
    Missing or unexported object: ‘SingleCellExperiment::isSpike’
    ```

# skeleSim

<details>

* Version: 0.9.8
* Source code: https://github.com/cran/skeleSim
* URL: https://github.com/christianparobek/skeleSim
* BugReports: https://github.com/christianparobek/skeleSim/issues
* Date/Publication: 2017-11-27 22:37:36 UTC
* Number of recursive dependencies: 157

Run `revdep_details(,"skeleSim")` for more info

</details>

## In both

*   checking whether package ‘skeleSim’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/skeleSim/new/skeleSim.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘skeleSim’ ...
** package ‘skeleSim’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: object ‘strata<-’ is not exported by 'namespace:strataG'
Execution halted
ERROR: lazy loading failed for package ‘skeleSim’
* removing ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/skeleSim/new/skeleSim.Rcheck/skeleSim’

```
### CRAN

```
* installing *source* package ‘skeleSim’ ...
** package ‘skeleSim’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Error: object ‘strata<-’ is not exported by 'namespace:strataG'
Execution halted
ERROR: lazy loading failed for package ‘skeleSim’
* removing ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/skeleSim/old/skeleSim.Rcheck/skeleSim’

```
# skpr

<details>

* Version: 0.64.2
* Source code: https://github.com/cran/skpr
* URL: https://github.com/tylermorganwall/skpr
* BugReports: https://github.com/tylermorganwall/skpr/issues
* Date/Publication: 2020-03-04 20:20:02 UTC
* Number of recursive dependencies: 127

Run `revdep_details(,"skpr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        libs   5.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘nlme’
      All declared Imports should be used.
    ```

# soc.ca

<details>

* Version: 0.7.3
* Source code: https://github.com/cran/soc.ca
* URL: https://github.com/Rsoc/soc.ca
* Date/Publication: 2016-02-09 15:53:12
* Number of recursive dependencies: 65

Run `revdep_details(,"soc.ca")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 833 marked UTF-8 strings
    ```

# soilcarbon

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/soilcarbon
* URL: https://powellcenter-soilcarbon.github.io/soilcarbon/
* BugReports: https://github.com/powellcenter-soilcarbon/soilcarbon/issues
* Date/Publication: 2017-08-04 03:17:31 UTC
* Number of recursive dependencies: 100

Run `revdep_details(,"soilcarbon")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    compileDatabase: no visible global function definition for ‘use_data’
    Undefined global functions or variables:
      use_data
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 145 marked UTF-8 strings
    ```

# sojourner

<details>

* Version: 1.2.1
* Source code: https://github.com/cran/sojourner
* URL: https://github.com/sheng-liu/sojourner
* BugReports: https://github.com/sheng-liu/sojourner/issues
* Date/Publication: 2020-05-30
* Number of recursive dependencies: 116

Run `revdep_details(,"sojourner")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rtiff’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# soundgen

<details>

* Version: 1.7.0
* Source code: https://github.com/cran/soundgen
* URL: http://cogsci.se/soundgen.html
* Date/Publication: 2020-05-24 14:20:02 UTC
* Number of recursive dependencies: 43

Run `revdep_details(,"soundgen")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc   4.3Mb
    ```

# SpaDES.addins

<details>

* Version: 0.1.2
* Source code: https://github.com/cran/SpaDES.addins
* URL: http://spades-addins.predictiveecology.org/, https://github.com/PredictiveEcology/SpaDES.addins
* BugReports: https://github.com/PredictiveEcology/SpaDES.addins/issues
* Date/Publication: 2019-02-03 20:13:20 UTC
* Number of recursive dependencies: 139

Run `revdep_details(,"SpaDES.addins")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘rstudioapi’
      All declared Imports should be used.
    ```

# spant

<details>

* Version: 1.6.0
* Source code: https://github.com/cran/spant
* BugReports: https://github.com/martin3141/spant/issues
* Date/Publication: 2020-06-05 15:40:06 UTC
* Number of recursive dependencies: 100

Run `revdep_details(,"spant")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘foreach’
      All declared Imports should be used.
    ```

# sparklyr

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/sparklyr
* URL: http://spark.rstudio.com
* BugReports: https://github.com/sparklyr/sparklyr/issues
* Date/Publication: 2020-04-20 19:30:02 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"sparklyr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R      2.1Mb
        java   2.4Mb
    ```

# sparkTable

<details>

* Version: 1.3.0
* Source code: https://github.com/cran/sparkTable
* URL: https://github.com/alexkowa/sparkTable
* BugReports: https://github.com/alexkowa/sparkTable/issues
* Date/Publication: 2016-12-13 19:55:10
* Number of recursive dependencies: 85

Run `revdep_details(,"sparkTable")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    pixmapGrob: no visible global function definition for ‘imageGrob’
    Undefined global functions or variables:
      imageGrob
    ```

# SpatialEpiApp

<details>

* Version: 0.3
* Source code: https://github.com/cran/SpatialEpiApp
* URL: https://github.com/Paula-Moraga/SpatialEpiApp
* BugReports: https://github.com/Paula-Moraga/SpatialEpiApp/issues
* Date/Publication: 2017-08-28 22:10:33 UTC
* Number of recursive dependencies: 116

Run `revdep_details(,"SpatialEpiApp")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘SpatialEpi’ ‘dplyr’ ‘dygraphs’ ‘ggplot2’
      ‘htmlwidgets’ ‘knitr’ ‘leaflet’ ‘mapproj’ ‘maptools’ ‘rgdal’ ‘rgeos’
      ‘rmarkdown’ ‘shinyjs’ ‘spdep’ ‘xts’
      All declared Imports should be used.
    ```

# spatialLIBD

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/spatialLIBD
* URL: https://github.com/LieberInstitute/spatialLIBD
* BugReports: https://support.bioconductor.org/t/spatialLIBD
* Date/Publication: 2020-05-07
* Number of recursive dependencies: 181

Run `revdep_details(,"spatialLIBD")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Check input image_path
    > ### Aliases: check_image_path
    > 
    > ### ** Examples
    > 
    > 
    > if (enough_ram()) {
    +     ## Obtain the necessary data
    +     if (!exists("sce")) sce <- fetch_data("sce")
    + 
    +     ## Get the path to the images
    +     img_path <- system.file("app", "www", "data", package = "spatialLIBD")
    + 
    +     ## Check the object
    +     check_image_path(img_path, sce)
    + }
    Error: Corrupt Cache: sqlite file
      See vignette section on corrupt cache
      cache: /Users/barret/Library/Caches/ExperimentHub
      filename: experimenthub.sqlite3
    Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.3Mb
      sub-directories of 1Mb or more:
        app    5.6Mb
        doc    5.9Mb
        help   2.3Mb
    ```

# spectrolab

<details>

* Version: 0.0.9
* Source code: https://github.com/cran/spectrolab
* URL: https://CRAN.R-project.org/package=spectrolab
* Date/Publication: 2020-06-09 13:20:03 UTC
* Number of recursive dependencies: 90

Run `revdep_details(,"spectrolab")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘usethis’
      All declared Imports should be used.
    ```

# SSDM

<details>

* Version: 0.2.8
* Source code: https://github.com/cran/SSDM
* URL: https://github.com/sylvainschmitt/SSDM
* BugReports: https://github.com/sylvainschmitt/SSDM/issues
* Date/Publication: 2020-02-28 13:00:02 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"SSDM")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinyFiles’
      All declared Imports should be used.
    ```

# ssrch

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/ssrch
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 48

Run `revdep_details(,"ssrch")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    docset_searchapp : server : <anonymous>: no visible global function
      definition for ‘metadata<-’
    docset_searchapp : server : <anonymous>: no visible global function
      definition for ‘metadata’
    Undefined global functions or variables:
      metadata metadata<-
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4 marked UTF-8 strings
    ```

# sstModel

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/sstModel
* Date/Publication: 2018-05-03 22:21:08 UTC
* Number of recursive dependencies: 68

Run `revdep_details(,"sstModel")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shinydashboard’
      All declared Imports should be used.
    ```

# STAT

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/STAT
* Date/Publication: 2019-04-01 16:40:10 UTC
* Number of recursive dependencies: 115

Run `revdep_details(,"STAT")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Hmisc’ ‘corrgram’ ‘datasets’ ‘dplyr’ ‘psych’ ‘psycho’ ‘rpivotTable’
      ‘shiny’
      All declared Imports should be used.
    ```

# steemr

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/steemr
* URL: https://github.com/pzhaonet/steemr
* BugReports: https://github.com/pzhaonet/steemr/issues
* Date/Publication: 2019-01-16 09:20:03 UTC
* Number of recursive dependencies: 115

Run `revdep_details(,"steemr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rlist’ ‘tm’ ‘zoo’
      All declared Imports should be used.
    ```

# stminsights

<details>

* Version: 0.3.0
* Source code: https://github.com/cran/stminsights
* URL: https://github.com/cschwem2er/stminsights
* BugReports: https://github.com/cschwem2er/stminsights/issues
* Date/Publication: 2018-11-24 13:10:03 UTC
* Number of recursive dependencies: 130

Run `revdep_details(,"stminsights")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘huge’ ‘readr’ ‘scales’ ‘shinyjs’
      All declared Imports should be used.
    ```

# STMotif

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/STMotif
* Date/Publication: 2020-01-23 13:20:05 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"STMotif")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# strand

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/strand
* URL: https://github.com/strand-tech/strand
* BugReports: https://github.com/strand-tech/strand/issues
* Date/Publication: 2020-05-26 10:10:02 UTC
* Number of recursive dependencies: 82

Run `revdep_details(,"strand")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘Rsymphony’
    ```

# subscreen

<details>

* Version: 2.0.1
* Source code: https://github.com/cran/subscreen
* Date/Publication: 2019-01-07 16:40:07 UTC
* Number of recursive dependencies: 86

Run `revdep_details(,"subscreen")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘V8’ ‘bsplus’ ‘colourpicker’ ‘dplyr’ ‘grDevices’ ‘graphics’
      ‘jsonlite’ ‘shinyjs’
      All declared Imports should be used.
    ```

# tableHTML

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/tableHTML
* URL: https://github.com/LyzandeR/tableHTML
* BugReports: https://github.com/LyzandeR/tableHTML/issues
* Date/Publication: 2019-03-16 17:30:02 UTC
* Number of recursive dependencies: 50

Run `revdep_details(,"tableHTML")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   5.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# tablerDash

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/tablerDash
* URL: https://rinterface.github.io/tablerDash/, https://github.com/RinteRface/tablerDash/
* BugReports: https://github.com/RinteRface/tablerDash/issues
* Date/Publication: 2019-03-08 16:00:03 UTC
* Number of recursive dependencies: 65

Run `revdep_details(,"tablerDash")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# TAShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/TAShiny
* Date/Publication: 2018-11-27 17:30:03 UTC
* Number of recursive dependencies: 44

Run `revdep_details(,"TAShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘SnowballC’ ‘dplyr’ ‘igraph’ ‘tm’ ‘wordcloud2’
      All declared Imports should be used.
    ```

# TBSignatureProfiler

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/TBSignatureProfiler
* URL: https://github.com/compbiomed/TBSignatureProfiler
* BugReports: https://github.com/compbiomed/TBSignatureProfiler/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 193

Run `revdep_details(,"TBSignatureProfiler")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    1.8Mb
    ```

# TCGAbiolinksGUI

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/TCGAbiolinksGUI
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 321

Run `revdep_details(,"TCGAbiolinksGUI")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 33.7Mb
      sub-directories of 1Mb or more:
        app   1.1Mb
        doc  32.4Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'DT'
      All declared Imports should be used.
    Package in Depends field not imported from: 'TCGAbiolinksGUI.data'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# TeachBayes

<details>

* Version: 1.0
* Source code: https://github.com/cran/TeachBayes
* Date/Publication: 2017-03-25 10:58:44 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"TeachBayes")` for more info

</details>

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Documented arguments not in \usage in documentation object 'beta_draw':
      ‘...’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# tenXplore

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/tenXplore
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 115

Run `revdep_details(,"tenXplore")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    apd: warning in dir(system.file("app", package = "tenXplore"), full =
      TRUE): partial argument match of 'full' to 'full.names'
    ```

# TestDesign

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/TestDesign
* URL: https://github.com/choi-phd/TestDesign
* BugReports: https://github.com/choi-phd/TestDesign/issues
* Date/Publication: 2020-02-06 00:50:02 UTC
* Number of recursive dependencies: 94

Run `revdep_details(,"TestDesign")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: 'Rsymphony', 'gurobi'
    ```

# testextra

<details>

* Version: 0.1.0.1
* Source code: https://github.com/cran/testextra
* URL: https://github.com/RDocTaskForce/testextra
* BugReports: https://github.com/RDocTaskForce/testextra/issues
* Date/Publication: 2019-12-18 09:15:29 UTC
* Number of recursive dependencies: 84

Run `revdep_details(,"testextra")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘stringi’ ‘utils’
      All declared Imports should be used.
    ```

# texPreview

<details>

* Version: 1.4.4
* Source code: https://github.com/cran/texPreview
* URL: https://github.com/metrumresearchgroup/texPreview
* BugReports: https://github.com/metrumresearchgroup/texPreview/issues
* Date/Publication: 2020-02-16 06:10:03 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"texPreview")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘details’
      All declared Imports should be used.
    ```

# TFutils

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/TFutils
* Date/Publication: 2020-04-30
* Number of recursive dependencies: 167

Run `revdep_details(,"TFutils")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘TFutils-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: tffamCirc.plot
    > ### Title: use a radial plot (by default) for motif stack
    > ### Aliases: tffamCirc.plot
    > 
    > ### ** Examples
    > 
    > p1 = tffamCirc.prep( )
    Loading required namespace: motifStack
    Failed with error:  'there is no package called 'MotIV''
    Error in FUN(X[[i]], ...) : 
      please install package motifStack to use this function.
    Calls: tffamCirc.prep -> sapply -> lapply -> FUN
    Execution halted
    ```

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' imports not declared from:
      'BiocFileCache' 'DT' 'httr' 'readxl' 'rjson'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.1Mb
      sub-directories of 1Mb or more:
        data      4.0Mb
        doc       2.2Mb
        lambert   1.4Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Rsamtools’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    fimo_granges : <anonymous>: no visible global function definition for
      ‘seqinfo<-’
    retrieve_lambert_main: no visible global function definition for
      ‘bfcadd’
    setupHIZE: no visible global function definition for ‘read.delim’
    tffamCirc.prep: no visible binding for global variable ‘TF family’
    tffamCirc.prep: no visible binding for global variable ‘TFfamily’
    tffamCirc.prep: no visible binding for global variable ‘Transcription
      factor’
    topTraitsOfTargets: no visible global function definition for ‘mcols’
    topTraitsOfTargets: no visible binding for global variable
      ‘DISEASE.TRAIT’
    Undefined global functions or variables:
      DISEASE.TRAIT Family_Name HGNC MAPPED_GENE TF family TFfamily
      Transcription factor bfcadd cisbpTFcat mcols read.csv read.delim
      reduceByRange rowRanges<- seqinfo<-
    Consider adding
      importFrom("base", "factor")
      importFrom("stats", "family")
      importFrom("utils", "read.csv", "read.delim")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 62 marked UTF-8 strings
    ```

# tidycells

<details>

* Version: 0.2.2
* Source code: https://github.com/cran/tidycells
* URL: https://r-rudra.github.io/tidycells/, https://github.com/r-rudra/tidycells
* BugReports: https://github.com/r-rudra/tidycells/issues
* Date/Publication: 2020-01-09 19:10:09 UTC
* Number of recursive dependencies: 115

Run `revdep_details(,"tidycells")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    > cd <- as_cell_df(d) %>% numeric_values_classifier()
    Error: `x` must be a vector, not a `tbl_df/tbl/data.frame/unpivotr` object.
    Backtrace:
         █
      1. ├─as_cell_df(d) %>% numeric_values_classifier()
      2. │ └─base::eval(lhs, parent, parent)
      3. │   └─base::eval(lhs, parent, parent)
      4. ├─tidycells::as_cell_df(d)
      5. └─tidycells:::as_cell_df.data.frame(d)
      6.   └─d %>% attach_intermediate_class() %>% as_cell_df_internal(...)
      7.     ├─base::withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
      8.     └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
      9.       └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
     10.         └─tidycells:::`_fseq`(`_lhs`)
     11.           └─magrittr::freduce(value, `_function_list`)
     12.             ├─base::withVisible(function_list[[k]](value))
     13.             └─function_list[[k]](value)
     14.               ├─tidycells:::as_cell_df_internal(., ...)
     15.               └─tidycells:::as_cell_df_internal.default(., ...)
     16.                 └─`%
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 34 | SKIPPED: 2 | WARNINGS: 8 | FAILED: 29 ]
      1. Error: numeric_values_classifier works (@test-VA_classifier.R#5) 
      2. Error: sample_based_classifier works (@test-VA_classifier.R#45) 
      3. Error: sample_based_classifier works (@test-VA_classifier.R#41) 
      4. Error: (unknown) (@test-VA_classifier.R#41) 
      5. Error: analyze_cell works: base 
      6. Error: analyze_cell works: tidyxl (@test-analyze_cells.R#28) 
      7. Error: analyze_cell works: tidyxl (@test-analyze_cells.R#25) 
      8. Error: (unknown) (@test-analyze_cells.R#25) 
      9. Error: as_cell_df() works on tidyxl::xlsx_cells on single sheet (@test-as_cell_df.R#6) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

# timeline

<details>

* Version: 0.9
* Source code: https://github.com/cran/timeline
* URL: http://jason.bryer.org/timeline, http://github.com/jbryer/timeline
* BugReports: https://github.com/jbryer/timeline/issues
* Date/Publication: 2013-05-09 17:03:14
* Number of recursive dependencies: 58

Run `revdep_details(,"timeline")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘shiny’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    timeline: no visible global function definition for ‘ggplot’
    timeline: no visible global function definition for ‘geom_segment’
    timeline: no visible global function definition for ‘aes_string’
    timeline: no visible global function definition for ‘geom_rect’
    timeline: no visible global function definition for ‘geom_text’
    timeline: no visible global function definition for ‘theme’
    timeline: no visible global function definition for ‘element_blank’
    timeline: no visible global function definition for ‘xlab’
    timeline: no visible global function definition for ‘ylab’
    timeline: no visible global function definition for ‘xlim’
    timeline: no visible global function definition for
      ‘scale_y_continuous’
    timeline: no visible global function definition for ‘geom_point’
    timeline: no visible global function definition for ‘scale_color_grey’
    timeline: no visible global function definition for ‘geom_hline’
    Undefined global functions or variables:
      aes_string element_blank geom_hline geom_point geom_rect geom_segment
      geom_text ggplot scale_color_grey scale_y_continuous theme xlab xlim
      ylab
    ```

# TimerQuant

<details>

* Version: 1.18.0
* Source code: https://github.com/cran/TimerQuant
* Date/Publication: 2020-05-07
* Number of recursive dependencies: 80

Run `revdep_details(,"TimerQuant")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
    genTimeSteadyStateHeatmap: no visible global function definition for
      ‘rainbow’
    plotPrimordiumProfile: no visible binding for global variable ‘median’
    plotPrimordiumProfile: no visible binding for global variable ‘mad’
    plotPrimordiumProfile: no visible global function definition for ‘par’
    plotPrimordiumProfile: no visible global function definition for ‘axis’
    plotPrimordiumProfile: no visible global function definition for
      ‘points’
    plotPrimordiumProfile: no visible global function definition for
      ‘polygon’
    plotPrimordiumProfile: no visible global function definition for ‘rgb’
    simulatedRatio: no visible global function definition for ‘rnorm’
    Undefined global functions or variables:
      approxfun axis mad median optimize par points polygon predict rainbow
      rgb rnorm
    Consider adding
      importFrom("grDevices", "rainbow", "rgb")
      importFrom("graphics", "axis", "par", "points", "polygon")
      importFrom("stats", "approxfun", "mad", "median", "optimize",
                 "predict", "rnorm")
    to your NAMESPACE file.
    ```

# timeseriesdb

<details>

* Version: 0.4.1
* Source code: https://github.com/cran/timeseriesdb
* URL: https://github.com/mbannert/timeseriesdb
* BugReports: https://github.com/mbannert/timeseriesdb/issues
* Date/Publication: 2018-08-06 11:20:03 UTC
* Number of recursive dependencies: 52

Run `revdep_details(,"timeseriesdb")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# timevis

<details>

* Version: 0.5
* Source code: https://github.com/cran/timevis
* URL: https://github.com/daattali/timevis, http://daattali.com/shiny/timevis-demo/
* BugReports: https://github.com/daattali/timevis/issues
* Date/Publication: 2019-01-16 09:00:03 UTC
* Number of recursive dependencies: 55

Run `revdep_details(,"timevis")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lubridate’
      All declared Imports should be used.
    ```

# tmaptools

<details>

* Version: 3.0
* Source code: https://github.com/cran/tmaptools
* URL: https://github.com/mtennekes/tmaptools
* BugReports: https://github.com/mtennekes/tmaptools/issues
* Date/Publication: 2020-03-30 15:00:02 UTC
* Number of recursive dependencies: 129

Run `revdep_details(,"tmaptools")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘KernSmooth’
      All declared Imports should be used.
    ```

# TR8

<details>

* Version: 0.9.21
* Source code: https://github.com/cran/TR8
* URL: https://github.com/GioBo/TR8
* BugReports: https://github.com/GioBo/TR8/issues
* Date/Publication: 2020-05-17 13:50:06 UTC
* Number of recursive dependencies: 85

Run `revdep_details(,"TR8")` for more info

</details>

## In both

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 153 marked Latin-1 strings
      Note: found 108 marked UTF-8 strings
    ```

# trackeRapp

<details>

* Version: 1.1
* Source code: https://github.com/cran/trackeRapp
* URL: https://github.com/trackerproject/trackeRapp
* BugReports: https://github.com/trackerproject/trackeRapp/issues
* Date/Publication: 2020-05-03 17:50:03 UTC
* Number of recursive dependencies: 128

Run `revdep_details(,"trackeRapp")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘V8’ ‘colorspace’
      All declared Imports should be used.
    ```

# treespace

<details>

* Version: 1.1.3.2
* Source code: https://github.com/cran/treespace
* URL: https://cran.r-project.org/package=treespace, https://github.com/thibautjombart/treespace
* BugReports: https://github.com/thibautjombart/treespace/issues
* Date/Publication: 2019-12-08 22:40:02 UTC
* Number of recursive dependencies: 174

Run `revdep_details(,"treespace")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        doc   4.3Mb
    ```

# TSCAN

<details>

* Version: 1.26.0
* Source code: https://github.com/cran/TSCAN
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 76

Run `revdep_details(,"TSCAN")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘lm’
    exprmclust: no visible global function definition for ‘dist’
    plotmclust: no visible binding for global variable ‘pca_dim_1’
    plotmclust: no visible binding for global variable ‘pca_dim_2’
    plotmclust: no visible binding for global variable ‘sample_name’
    preprocess: no visible binding for global variable ‘sd’
    preprocess: no visible global function definition for ‘hclust’
    preprocess: no visible global function definition for ‘dist’
    preprocess: no visible global function definition for ‘cutree’
    preprocess: no visible global function definition for ‘aggregate’
    singlegeneplot: no visible global function definition for
      ‘fitted.values’
    singlegeneplot: no visible binding for global variable ‘predict’
    Undefined global functions or variables:
      aggregate cutree dist fitted.values hclust lm p.adjust pca_dim_1
      pca_dim_2 pchisq prcomp predict sample_name sd
    Consider adding
      importFrom("stats", "aggregate", "cutree", "dist", "fitted.values",
                 "hclust", "lm", "p.adjust", "pchisq", "prcomp", "predict",
                 "sd")
    to your NAMESPACE file.
    ```

# ttestshiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/ttestshiny
* Date/Publication: 2018-10-30 16:30:03 UTC
* Number of recursive dependencies: 35

Run `revdep_details(,"ttestshiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘shinyAce’ ‘shinyjs’
      All declared Imports should be used.
    ```

# TVTB

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/TVTB
* URL: https://github.com/kevinrue/TVTB
* BugReports: https://github.com/kevinrue/TVTB/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 158

Run `revdep_details(,"TVTB")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   2.7Mb
    ```

# twoddpcr

<details>

* Version: 1.12.0
* Source code: https://github.com/cran/twoddpcr
* URL: http://github.com/CRUKMI-ComputationalBiology/twoddpcr/
* BugReports: http://github.com/CRUKMI-ComputationalBiology/twoddpcr/issues/
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 107

Run `revdep_details(,"twoddpcr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data   1.2Mb
        doc    4.1Mb
    ```

# TxRegInfra

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/TxRegInfra
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 158

Run `revdep_details(,"TxRegInfra")` for more info

</details>

## In both

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘rjson’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to 'shiny' in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    dnmetaApp: no visible global function definition for ‘fluidPage’
    dnmetaApp: no visible global function definition for ‘sidebarLayout’
    dnmetaApp: no visible global function definition for ‘sidebarPanel’
    dnmetaApp: no visible global function definition for ‘helpText’
    dnmetaApp: no visible global function definition for ‘radioButtons’
    dnmetaApp: no visible global function definition for
      ‘checkboxGroupInput’
    dnmetaApp: no visible global function definition for ‘uiOutput’
    dnmetaApp: no visible global function definition for ‘mainPanel’
    dnmetaApp: no visible global function definition for ‘dataTableOutput’
    dnmetaApp : server: no visible global function definition for
      ‘renderUI’
    dnmetaApp : server: no visible global function definition for
      ‘selectInput’
    dnmetaApp : server: no visible global function definition for
      ‘renderDataTable’
    dnmetaApp: no visible global function definition for ‘runApp’
    Undefined global functions or variables:
      checkboxGroupInput dataTableOutput fluidPage genes helpText mainPanel
      radioButtons renderDataTable renderUI runApp selectInput
      sidebarLayout sidebarPanel uiOutput
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: addsyms.Rd:23-25: Dropping empty section \examples
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 14 marked UTF-8 strings
    ```

# ubiquity

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/ubiquity
* URL: https://r.ubiquity.tools/
* BugReports: https://github.com/john-harrold/ubiquity/issues
* Date/Publication: 2020-01-21 07:50:02 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"ubiquity")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        doc     6.4Mb
        ubinc   2.0Mb
    ```

# UCSCXenaShiny

<details>

* Version: 0.5.0
* Source code: https://github.com/cran/UCSCXenaShiny
* URL: https://github.com/openbiox/XenaShiny
* BugReports: https://github.com/openbiox/XenaShiny/issues
* Date/Publication: 2020-06-03 13:40:02 UTC
* Number of recursive dependencies: 126

Run `revdep_details(,"UCSCXenaShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘RColorBrewer’ ‘plotly’ ‘shinyBS’ ‘shinyWidgets’ ‘shinyjs’
      ‘shinythemes’ ‘zip’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 162 marked UTF-8 strings
    ```

# Ularcirc

<details>

* Version: 1.6.0
* Source code: https://github.com/cran/Ularcirc
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 145

Run `revdep_details(,"Ularcirc")` for more info

</details>

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘mirbase.db’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ursa

<details>

* Version: 3.8.20
* Source code: https://github.com/cran/ursa
* URL: https://github.com/nplatonov/ursa
* BugReports: https://github.com/nplatonov/ursa/issues
* Date/Publication: 2020-06-12 09:10:06 UTC
* Number of recursive dependencies: 116

Run `revdep_details(,"ursa")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘rje’
    ```

# VariantFiltering

<details>

* Version: 1.24.0
* Source code: https://github.com/cran/VariantFiltering
* URL: https://github.com/rcastelo/VariantFiltering
* BugReports: https://github.com/rcastelo/VariantFiltering/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 169

Run `revdep_details(,"VariantFiltering")` for more info

</details>

## In both

*   checking running R code from vignettes ...
    ```
      ‘usingVariantFiltering.Rnw’... failed
     ERROR
    Errors in running code in vignettes:
    when running code in ‘usingVariantFiltering.Rnw’
      ...
    > vfpar <- VariantFilteringParam(CEUvcf)
    Loading BSgenome annotation package BSgenome.Hsapiens.1000genomes.hs37d5
    Loading OrgDb annotation package org.Hs.eg.db
    Loading TxDb annotation package TxDb.Hsapiens.UCSC.hg19.knownGene
    Loading SNPlocs annotation package SNPlocs.Hsapiens.dbSNP144.GRCh37
    Loading annotation package MafDb.1Kgenomes.phase1.hs37d5
    
      When sourcing 'usingVariantFiltering.R':
    Error: please install the Bioconductor package PolyPhen.Hsapiens.dbSNP131.
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      'PolyPhen.Hsapiens.dbSNP131', 'SIFT.Hsapiens.dbSNP137'
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.3Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        extdata   3.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'S4Vectors:::labeledLine' 'VariantAnnotation:::.checkArgs'
      'VariantAnnotation:::.consolidateHits'
      'VariantAnnotation:::.returnEmpty'
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      '.adjustForStrandSense' '.ancestorsSO' '.findSOIDs'
    ```

# VarSelLCM

<details>

* Version: 2.1.3
* Source code: https://github.com/cran/VarSelLCM
* URL: http://varsellcm.r-forge.r-project.org/
* Date/Publication: 2018-08-29 10:24:28 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"VarSelLCM")` for more info

</details>

## In both

*   checking line endings in C/C++/Fortran sources/headers ... NOTE
    ```
    ...
      inst/include/AlgorithmCategorical.h
      inst/include/AlgorithmContinuous.h
      inst/include/AlgorithmInteger.h
      inst/include/AlgorithmMixed.h
      inst/include/Data.h
      inst/include/DataCategorical.h
      inst/include/DataContinuous.h
      inst/include/DataInteger.h
      inst/include/DataMixed.h
      inst/include/Param.h
      inst/include/ParamCategorical.h
      inst/include/ParamContinuous.h
      inst/include/ParamInteger.h
      inst/include/ParamMixed.h
      inst/include/XEM.h
      inst/include/XEMCategorical.h
      inst/include/XEMContinuous.h
      inst/include/XEMInteger.h
      inst/include/XEMMixed.h
      inst/include/XEMPen.h
    Some compilers warn on such files.
    ```

# vdiffr

<details>

* Version: 0.3.2
* Source code: https://github.com/cran/vdiffr
* URL: https://github.com/r-lib/vdiffr
* BugReports: https://github.com/r-lib/vdiffr/issues
* Date/Publication: 2020-06-06 13:00:02 UTC
* Number of recursive dependencies: 103

Run `revdep_details(,"vdiffr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘freetypeharfbuzz’
      All declared Imports should be used.
    ```

# viromeBrowser

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/viromeBrowser
* Date/Publication: 2019-08-21 08:10:02 UTC
* Number of recursive dependencies: 111

Run `revdep_details(,"viromeBrowser")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘plyr’ ‘reshape’
      All declared Imports should be used.
    ```

# visit

<details>

* Version: 2.1
* Source code: https://github.com/cran/visit
* Date/Publication: 2019-08-26 09:30:02 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"visit")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘sqldf’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# visNetwork

<details>

* Version: 2.0.9
* Source code: https://github.com/cran/visNetwork
* URL: http://datastorm-open.github.io/visNetwork/
* BugReports: https://github.com/datastorm-open/visNetwork/issues
* Date/Publication: 2019-12-06 08:50:02 UTC
* Number of recursive dependencies: 93

Run `revdep_details(,"visNetwork")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        doc           4.3Mb
        docjs         1.4Mb
        htmlwidgets   3.9Mb
    ```

# VTShiny

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/VTShiny
* Date/Publication: 2019-07-18 09:08:11 UTC
* Number of recursive dependencies: 69

Run `revdep_details(,"VTShiny")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘EnvStats’ ‘shiny’ ‘stats’
      All declared Imports should be used.
    ```

# waiter

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/waiter
* Date/Publication: 2020-01-14 15:50:02 UTC
* Number of recursive dependencies: 18

Run `revdep_details(,"waiter")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# wallace

<details>

* Version: 1.0.6.2
* Source code: https://github.com/cran/wallace
* URL: https://wallaceEcoMod.github.io
* Date/Publication: 2020-02-13 22:30:02 UTC
* Number of recursive dependencies: 130

Run `revdep_details(,"wallace")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘ENMeval’ ‘RColorBrewer’ ‘XML’ ‘dismo’ ‘dplyr’ ‘leaflet.extras’
      ‘maptools’ ‘raster’ ‘rgdal’ ‘rgeos’ ‘rmarkdown’ ‘shinyjs’
      ‘shinythemes’ ‘spThin’ ‘spocc’ ‘testthat’ ‘zip’
      All declared Imports should be used.
    ```

# webr

<details>

* Version: 0.1.5
* Source code: https://github.com/cran/webr
* URL: https://github.com/cardiomoon/webr
* BugReports: https://github.com/cardiomoon/webr/issues
* Date/Publication: 2020-01-26 14:20:02 UTC
* Number of recursive dependencies: 147

Run `revdep_details(,"webr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘shiny’ ‘tidyselect’
      All declared Imports should be used.
    ```

# wiseR

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/wiseR
* URL: https://github.com/SAFE-ICU/wiseR
* BugReports: https://github.com/SAFE-ICU/wiseR/issues
* Date/Publication: 2018-11-29 16:10:03 UTC
* Number of recursive dependencies: 137

Run `revdep_details(,"wiseR")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        bn   7.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘DescTools’ ‘HydeNet’ ‘RBGL’ ‘Rgraphviz’ ‘arules’ ‘bnlearn’
      ‘dplyr’ ‘graph’ ‘igraph’ ‘linkcomm’ ‘missRanger’ ‘parallel’ ‘psych’
      ‘rhandsontable’ ‘rintrojs’ ‘shinyBS’ ‘shinyWidgets’ ‘shinyalert’
      ‘shinycssloaders’ ‘shinydashboard’ ‘tools’ ‘visNetwork’
      All declared Imports should be used.
    ```

# wTO

<details>

* Version: 1.6.3
* Source code: https://github.com/cran/wTO
* Date/Publication: 2018-11-17 11:50:03 UTC
* Number of recursive dependencies: 39

Run `revdep_details(,"wTO")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘shiny’
      All declared Imports should be used.
    ```

# xROI

<details>

* Version: 0.9.13
* Source code: https://github.com/cran/xROI
* BugReports: https://github.com/bnasr/xROI/issues
* Date/Publication: 2019-02-13 22:00:03 UTC
* Number of recursive dependencies: 103

Run `revdep_details(,"xROI")` for more info

</details>

## In both

*   checking whether package ‘xROI’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘lubridate::intersect’ by ‘raster::intersect’ when loading ‘xROI’
      Warning: replacing previous import ‘lubridate::union’ by ‘raster::union’ when loading ‘xROI’
    See ‘/Users/barret/Documents/git/rstudio/shiny/shiny/revdep/checks.noindex.nosync/xROI/new/xROI.Rcheck/00install.out’ for details.
    ```

# ymlthis

<details>

* Version: 0.1.2
* Source code: https://github.com/cran/ymlthis
* URL: https://ymlthis.r-lib.org, https://github.com/r-lib/ymlthis
* BugReports: https://github.com/r-lib/ymlthis/issues
* Date/Publication: 2020-02-03 23:00:02 UTC
* Number of recursive dependencies: 87

Run `revdep_details(,"ymlthis")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘miniUI’ ‘shinyBS’
      All declared Imports should be used.
    ```

# yuimaGUI

<details>

* Version: 1.3.0
* Source code: https://github.com/cran/yuimaGUI
* Date/Publication: 2018-11-25 12:50:03 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"yuimaGUI")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ghyp’
      All declared Imports should be used.
    ```

