diff --git a/src/index.js b/src/index.js
index 954e3c729694ec60a178400305a09607ca8f989b..55d554f75d435d31389cf6ef1723c946b50e6ff4 100644
--- a/src/index.js
+++ b/src/index.js
@@ -16,6 +16,7 @@ const pluginBabel = (options = {}) => ({
 					supportsStaticESM: true
 				}
 			});
+			if (!babelOptions) return { contents }

 			if (babelOptions.sourceMaps) {
 				const filename = path.relative(process.cwd(), args.path);
