diff --git a/misc.d.ts b/misc.d.ts
index 126d374477db459e1a251f5af548e88b46f43cdd..5c0e8ce70da01784decfd8bf457ee4cf83cbc63e 100644
--- a/misc.d.ts
+++ b/misc.d.ts
@@ -6617,9 +6617,6 @@ $( "#checkMetaKey" ).click(function( event ) {
     type _Falsy = false | null | undefined | 0 | '' | typeof document.all;
 }

-declare const jQuery: JQueryStatic;
-declare const $: JQueryStatic;
-
 type _Event = Event;
 type _UIEvent = UIEvent;
 type _MouseEvent = MouseEvent;
