diff --git a/misc.d.ts b/misc.d.ts
index 126d374477db459e1a251f5af548e88b46f43cdd..948cfb287b0bfa6057f3ccbcd013579aef07a29a 100644
--- a/misc.d.ts
+++ b/misc.d.ts
@@ -6618,7 +6618,7 @@ $( "#checkMetaKey" ).click(function( event ) {
 }

 declare const jQuery: JQueryStatic;
-declare const $: JQueryStatic;
+// declare const $: JQueryStatic;

 type _Event = Event;
 type _UIEvent = UIEvent;
@@ -6643,6 +6643,6 @@ interface SymbolConstructor {
     readonly toStringTag: symbol;
 }

-declare var Symbol: SymbolConstructor;
+// declare var Symbol: SymbolConstructor;

 // #endregion
